

<!DOCTYPE html>

<html lang="en-US">
<head>
  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=Edge">

  

  <link rel="shortcut icon" href="/mitkb/favicon.ico" type="image/x-icon">

  <link rel="stylesheet" href="/mitkb/assets/css/just-the-docs-default.css">

  
    <script async src="https://www.googletagmanager.com/gtag/js?id=G-84ETYTKDBN"></script>
    <script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'G-84ETYTKDBN', { 'anonymize_ip': true });
    </script>

  

  
    <script type="text/javascript" src="/mitkb/assets/js/vendor/lunr.min.js"></script>
  
  <script type="text/javascript" src="/mitkb/assets/js/just-the-docs.js"></script>

  <meta name="viewport" content="width=device-width, initial-scale=1">

  <!-- Begin Jekyll SEO tag v2.7.1 -->
<title>4 Biological experiments | mitkb</title>
<meta name="generator" content="Jekyll v4.2.0" />
<meta property="og:title" content="4 Biological experiments" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="High-throughput biological experiments" />
<meta property="og:description" content="High-throughput biological experiments" />
<link rel="canonical" href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/4-high-throughput-biological-experiments/" />
<meta property="og:url" content="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/4-high-throughput-biological-experiments/" />
<meta property="og:site_name" content="mitkb" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="4 Biological experiments" />
<script type="application/ld+json">
{"description":"High-throughput biological experiments","url":"https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/4-high-throughput-biological-experiments/","@type":"WebPage","headline":"4 Biological experiments","publisher":{"@type":"Organization","logo":{"@type":"ImageObject","url":"https://ncrnas.github.io/mitkb/assets/images/logo.png"}},"@context":"https://schema.org"}</script>
<!-- End Jekyll SEO tag -->


  
<meta name="google-site-verification" content="vstkqjcoJe9vQcOOpsCKrxv6WfqZ2SKRNWSv7XoHFeQ" />

<link rel="apple-touch-icon" sizes="180x180" href="/mitkb/assets/images/favicon/apple-touch-icon.png">
<link rel="icon" type="image/png" sizes="32x32" href="/mitkb/assets/images/favicon/favicon-32x32.png">
<link rel="icon" type="image/png" sizes="16x16" href="/mitkb/assets/images/favicon/favicon-16x16.png">
<link rel="manifest" href="/mitkb/assets/images/favicon/site.webmanifest">
<link rel="mask-icon" href="/mitkb/assets/images/favicon/safari-pinned-tab.svg" color="#5bbad5">
<meta name="msapplication-TileColor" content="#2b5797">
<meta name="theme-color" content="#ffffff">

<!-- for mathjax support -->



  <script src="https://unpkg.com/vanilla-back-to-top@7.2.1/dist/vanilla-back-to-top.min.js"></script>


<script
  data-cookie-notice='{ "learnMoreLinkEnabled": true, "learnMoreLinkHref": "/mitkb/terms/" }'
  src="https://unpkg.com/cookie-notice@^1/dist/cookie.notice.min.js"
></script>


</head>

<body>
  <svg xmlns="http://www.w3.org/2000/svg" style="display: none;">
    <symbol id="svg-link" viewBox="0 0 24 24">
      <title>Link</title>
      <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-link">
        <path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path>
      </svg>
    </symbol>
    <symbol id="svg-search" viewBox="0 0 24 24">
      <title>Search</title>
      <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-search">
        <circle cx="11" cy="11" r="8"></circle><line x1="21" y1="21" x2="16.65" y2="16.65"></line>
      </svg>
    </symbol>
    <symbol id="svg-menu" viewBox="0 0 24 24">
      <title>Menu</title>
      <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-menu">
        <line x1="3" y1="12" x2="21" y2="12"></line><line x1="3" y1="6" x2="21" y2="6"></line><line x1="3" y1="18" x2="21" y2="18"></line>
      </svg>
    </symbol>
    <symbol id="svg-arrow-right" viewBox="0 0 24 24">
      <title>Expand</title>
      <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-chevron-right">
        <polyline points="9 18 15 12 9 6"></polyline>
      </svg>
    </symbol>
    <symbol id="svg-doc" viewBox="0 0 24 24">
      <title>Document</title>
      <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-file">
        <path d="M13 2H6a2 2 0 0 0-2 2v16a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V9z"></path><polyline points="13 2 13 9 20 9"></polyline>
      </svg>
    </symbol>
  </svg>

  <div class="side-bar">
    <div class="site-header">
      <a href="https://ncrnas.github.io/mitkb/" class="site-title lh-tight">
  <div class="site-logo"></div>

</a>
      <a href="#" id="menu-button" class="site-button">
        <svg viewBox="0 0 24 24" class="icon"><use xlink:href="#svg-menu"></use></svg>
      </a>
    </div>
    <nav role="navigation" aria-label="Main" id="site-nav" class="site-nav">
      
        <ul class="nav-list"><li class="nav-list-item"><a href="https://ncrnas.github.io/mitkb/" class="nav-list-link">Home</a></li><li class="nav-list-item active"><a href="#" class="nav-list-expander"><svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg></a><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/" class="nav-list-link">Overview</a><ul class="nav-list "><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/0-front-matter/" class="nav-list-link">0 Front matter</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/1-introduction/" class="nav-list-link">1 Introduction</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/2-papers-and-their-corresponding-sub-goals/" class="nav-list-link">2 Papers and sub-goals</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/3-micrornas-and-other-non-coding-rnas/" class="nav-list-link">3 miRNAs and ncRNAs</a></li><li class="nav-list-item  active"><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/4-high-throughput-biological-experiments/" class="nav-list-link active">4 Biological experiments</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/5-statistical-tests-and-methods/" class="nav-list-link">5 Statistical methods</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/6-machine-learning-theory-and-support-vector-machine/" class="nav-list-link">6 Support vector machine</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/7-computational-implementation/" class="nav-list-link">7 Implementation</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/8-future-perspectives/" class="nav-list-link">8 Future perspectives</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander"><svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg></a><a href="https://ncrnas.github.io/mitkb/mirna-target-prediction/" class="nav-list-link">Target prediction</a><ul class="nav-list "><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/mirna-target-prediction/micrornas-targeting-and-target-prediction/" class="nav-list-link">Review: miRNA target</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/mirna-target-prediction/two-step-svm-model/" class="nav-list-link">Research: Two-step SVM</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/mirna-target-prediction/two-step-svm-model-supplementary-information/" class="nav-list-link">Supplementary information</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander"><svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg></a><a href="https://ncrnas.github.io/mitkb/mirna-high-throughput-experiments/" class="nav-list-link">Biological experiments</a><ul class="nav-list "><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/mirna-high-throughput-experiments/confounding-factors-in-mirna-experiments/" class="nav-list-link">Research: Confounding</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/mirna-high-throughput-experiments/confounding-factors-in-mirna-experiments-supplementary-information/" class="nav-list-link">Supplementary information</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander"><svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg></a><a href="https://ncrnas.github.io/mitkb/mirna-and-other-ncrnas/" class="nav-list-link">Other ncRNAs</a><ul class="nav-list "></ul></li><li class="nav-list-item"><a href="https://ncrnas.github.io/mitkb/support-articles/" class="nav-list-link">Support articles</a></li><li class="nav-list-item"><a href="https://ncrnas.github.io/mitkb/contact" class="nav-list-link">Contact us</a></li></ul>

      
    </nav>
    <footer class="site-footer">
      <a href="https://ncrnas.github.io/mitkb/terms">Terms and Privacy Policy</a> <br /><br />
      This site uses <a href="https://github.com/pmarsceill/just-the-docs">Just the Docs</a>, a documentation theme for Jekyll.
    </footer>
  </div>
  <div class="main" id="top">
    <div id="main-header" class="main-header">
      
        <div class="search">
          <div class="search-input-wrap">
            <input type="text" id="search-input" class="search-input" tabindex="0" placeholder="Search mitkb" aria-label="Search mitkb" autocomplete="off">
            <label for="search-input" class="search-label"><svg viewBox="0 0 24 24" class="search-icon"><use xlink:href="#svg-search"></use></svg></label>
          </div>
          <div id="search-results" class="search-results"></div>
        </div>
      
      
      
        <nav aria-label="Auxiliary" class="aux-nav">
          <ul class="aux-nav-list">
            
              <li class="aux-nav-list-item">
                <a href="https://github.com/ncrnas/mitkb" class="site-button"
                  
                >
                  mitkb on github
                </a>
              </li>
            
          </ul>
        </nav>
      
    </div>
    <div id="main-content-wrap" class="main-content-wrap">
      
        <nav aria-label="Breadcrumb" class="breadcrumb-nav">
            <ol class="breadcrumb-nav-list">
              
                <li class="breadcrumb-nav-list-item"><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/">Overview</a></li>
              
              <li class="breadcrumb-nav-list-item"><span>4 Biological experiments</span></li>
            </ol>
          </nav>
        
      
      <div id="main-content" class="main-content" role="main">
        
          <h1 class="no_toc" id="4-high-throughput-biological-experiments">
        
        
          <a href="#4-high-throughput-biological-experiments" class="anchor-heading" aria-labelledby="4-high-throughput-biological-experiments"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 4 High-throughput biological experiments
        
        
      </h1>
    <hr />

<details open="">
  <summary class="text-delta">
    Table of contents
  </summary>
<ol id="markdown-toc">
  <li><a href="#S:4.1" id="markdown-toc-S:4.1">4.1 One microarray experiment can detect thousands of gene expressions simultaneously</a></li>
  <li><a href="#S:4.2" id="markdown-toc-S:4.2">4.2 The next generation sequencing methods are faster and more cost-effective than Sanger sequencing</a></li>
  <li><a href="#S:4.3" id="markdown-toc-S:4.3">4.3 The second generation sequencing technologies can cover a wide range of applications</a></li>
  <li><a href="#S:4.4" id="markdown-toc-S:4.4">4.4 Liquid chromatography-tandem mass spectrometry is a powerful tool to analyze quantitative proteomics</a></li>
  <li><a href="#S:4.5" id="markdown-toc-S:4.5">4.5 Most preprocessed and raw data sets from high-throughput experiments are publicly available</a></li>
  <li><a href="#S:4.6" id="markdown-toc-S:4.6">References</a></li>
</ol>

</details><hr />

<p>Emerging high-throughput technologies have enabled genome-wide analyses
of various biological data, such as different cell lines, tissues, and
species, under different conditions. This chapter explains several
high-throughput technologies for transcriptomic and proteomic analyses
used in our research. We used both microarray and quantitative
proteomics data to achieve all three sub-goals, but we used next
generation sequence data only for the second and third sub-goals:
<em><a href="/mitkb/mirna-high-throughput-experiments/">miRNA high-throughput experiments</a></em>,
and <em><a href="/mitkb/mirna-and-other-ncrnas/">miRNA and other ncRNAs</a></em>.</p>
      <h2 id="S:4.1">
        
        
          <a href="#S:4.1" class="anchor-heading" aria-labelledby="S:4.1"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 4.1 One microarray experiment can detect thousands of gene expressions simultaneously
        
        
      </h2>
    

<p>Microarrays can capture the expression levels of many genes in a single
set of experiments. The microarray technology enables a high-throughput
transcriptome-wide analysis, which is hard to achieve with other
techniques for gene expression analyses, such as Northern blot
<a class="citation" href="#Alwine1977">[1]</a> or reverse transcription quantitative polymerase chain
reaction (RT-qPCR) <a class="citation" href="#Becker-Andre1989">[2,3]</a>. The Northern blot
is a qualitative but low-throughput technique that requires the use of
electrophoresis and large amounts of the input RNA <a class="citation" href="#Morozova2009">[4]</a>.
RT-qPCR can achieve higher throughput levels with less amount of the
input RNA than Northern blot, but the throughput remains on the order of
hundreds of known transcripts at a time
<a class="citation" href="#Morozova2009">[4,5]</a>.</p>

<p>Although several types of microarray technologies exist, the DNA
microarray is usually used for the transcriptome analysis. One
microarray chip usually consists of thousands of spots, and each spot
contains DNA oligonucleotides of a specific sequence
<a class="citation" href="#Quackenbush2006d">[6]</a>. These DNA oligonucleotides are suitable for
hybridization with DNA or RNA isolated from cells. Isolated RNAs are
chemically labeled before hybridization. These labels, such as
fluorescence dyes, are used to detect signal intensities of spots and
determine the relative mRNA abundance among samples. One of the major
DNA microarray applications is to measure the relative difference of
mRNA abundance between two samples. For example, Figure
<a href="#F:4.1">4.1</a> shows the procedure of a two-color microarray system
<a class="citation" href="#Quackenbush2006d">[6]</a>. Another popular microarray design is one-color
microarray system, and data quality is essentially equivalent between
one- and two-color approaches <a class="citation" href="#Patterson2006">[7]</a>.</p>
      <h3 id="F:4.1" class="no_toc">
        
        
          <a href="#F:4.1" class="anchor-heading" aria-labelledby="F:4.1"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a>  
        
        
      </h3>
    
<p><img src="/mitkb/assets/images/thesis/two-color-microarray.webp" alt="Figure 4.1" width="484" /></p>
      <h4 class="no_toc text-caption" id="figure-41-microarray-procedure">
        
        
          <a href="#figure-41-microarray-procedure" class="anchor-heading" aria-labelledby="figure-41-microarray-procedure"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> <strong>Figure 4.1</strong>. Microarray procedure.
        
        
      </h4>
    
<p class="fs-2">Two samples are compared by a two-color DNA
microarray. Sample RNA represents a sample of interest, whereas
Reference RNA represents a control sample for comparison. Sample RNA is
dyed red by Cy5, and Reference RNA is dyed green by Cy3. These two
samples are combined and hybridized with DNA oligonucleotides on the
chip. Color intensity is measured to estimate relative RNA expression
levels of the two samples.</p><hr />

<p>After measuring the intensity levels, the raw data usually go through
normalization and transformation <a class="citation" href="#Quackenbush2002">[8]</a>, and this whole
process is usually called the pre-processing of microarray data. The
pre-processing of the raw data is very important to reduce the noise in
each sample at the local level and among multiple samples at the global
level. In addition to detecting relative RNA expression levels, the DNA
microarray can be used for other applications, such as the detection of
single-nucleotide polymorphisms (SNPs), and alternative splicing events
<a class="citation" href="#Mockler2005">[9,4]</a>.</p>

<p>Two major drawbacks of microarray technologies are the limited ability
to detect novel transcripts and noisy data even after pre-processing
<a class="citation" href="#Morozova2009">[4]</a>. Nonetheless, the microarray technology is still widely
used for many transcriptomics analyses because of the ability to measure
the expression of thousands of genes simultaneously at relatively low
cost.</p>
      <h2 id="S:4.2">
        
        
          <a href="#S:4.2" class="anchor-heading" aria-labelledby="S:4.2"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 4.2 The next generation sequencing methods are faster and more cost-effective than Sanger sequencing
        
        
      </h2>
    

<p>As an alternative approach to the microarray technology, DNA sequencing
approaches are also widely used for transcriptome analysis. The
advantage of these sequencing methods is the ability to identify new
transcripts and measure the abundance of transcripts directly
<a class="citation" href="#Morozova2009">[4]</a>. First generation sequencing methods relied on the
Sanger method <a class="citation" href="#Sanger1977">[10]</a>. Since the original Sanger method uses the
full-length complementary DNA (FLcDNA), it involves a complex <em>in vivo</em>
cloning step that usually results in very high cost. Therefore, the
original Sanger method is normally limited only to novel transcript
discovery and annotation <a class="citation" href="#Morozova2009">[4]</a>.</p>

<p>Two examples of Sanger method applications are expressed sequence tag
(EST) <a class="citation" href="#Adams1991">[11]</a> and Serial Analysis of Gene Expression (SAGE)
<a class="citation" href="#Velculescu1995">[12]</a>. Both approaches use shorter tags, which are short
sub-sequences of the cDNA sequence, rather than the FLcDNAs. ESTs are
short tags generated from either 3’ or 5’ end of a cDNA
clone. Even though the sequencing cost is reduced by ESTs compared with
FLcDNAs, it is still too expensive for the whole transcriptome analysis
<a class="citation" href="#Morozova2009">[4]</a>. SAGE is a method that uses short tags generated from
3’ ends of mRNA transcripts. SAGE is suitable for estimating
transcript abundance due to high redundancy of sequencing reads
<a class="citation" href="#Morozova2009">[4]</a>. However, SAGE is still costly for the transcriptome
analysis because it still relies on labor intensive <em>in vivo</em> cloning
procedures <a class="citation" href="#Morozova2009">[4]</a>.</p>

<p>The next generation, or the second generation, sequencing methods have
substantially improved upon the Sanger method. They produce millions of
short reads in a relatively short period of time <a class="citation" href="#Voelkerding2010">[13]</a>
depending on several criteria, such as read length, sequence coverage,
and the size of the genome of interest (Table <a href="#T:4.1">4.1</a>).
The reads are assembled computationally afterwards if necessary. The
major contribution to this enhancement is to parallelize the sequencing
process <a class="citation" href="#Hall2007">[14]</a>, though other features, such as the usage of
PCR-based amplification instead of costly and labor intensive <em>in vivo</em>
cloning, also contribute <a class="citation" href="#Morozova2009">[4]</a>. Three popular commercially
available next generation sequencing technologies are, Roche 454
(<a href="https://en.wikipedia.org/wiki/454_Life_Sciences">http://454.com</a>),
Illumina (<a href="https://www.illumina.com">https://www.illumina.com</a>), and Applied
Biosystems SOLiD (<a href="https://www.appliedbiosystems.com">https://www.appliedbiosystems.com</a>) (Table <a href="#T:4.1">4.1</a>).
The second generation sequencing can be used for all the applications
that are based on the Sanger method, including EST and SAGE. Some
applications that are based on the second generation sequencing are
explained in the next section.</p>

<p><br /></p>
      <h3 id="T:4.1" class="no_toc text-caption">
        
        
          <a href="#T:4.1" class="anchor-heading" aria-labelledby="T:4.1"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> <strong>Table 4.1</strong>. Next generation sequencing technologies.
        
        
      </h3>
    
<p class="fs-2">The table shows three examples of commercially available next generation sequencing
technologies with specifications obtained from their corresponding web
sites as of March 2011. “bp” and “Gb” represent base pairs and giga base
pairs respectively. The product type with the best specification is
selected for each technology. Both 2 × n and n × m
represent the read length of the pair end approach where n and m are
read lengths in base pairs.</p>

<div class="table-wrapper"><table>
  <thead>
    <tr>
      <th> </th>
      <th><strong>Roche 454</strong></th>
      <th><strong>Illumina</strong></th>
      <th><strong>SOLiD</strong></th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td><strong>Product type</strong></td>
      <td>GS FLX</td>
      <td>HiSeq 2000</td>
      <td>5500xl</td>
    </tr>
    <tr>
      <td><strong>Read length</strong></td>
      <td>400 bp</td>
      <td>1 × 35 bp</td>
      <td>75 bp</td>
    </tr>
    <tr>
      <td> </td>
      <td> </td>
      <td>2 × 50 bp</td>
      <td>75 bp × 35 bp</td>
    </tr>
    <tr>
      <td> </td>
      <td> </td>
      <td>2 × 100 bp</td>
      <td>60 bp × 60 bp</td>
    </tr>
    <tr>
      <td><strong>Throughput per day</strong></td>
      <td>1 Gb</td>
      <td>25 Gb</td>
      <td>20-30 Gb</td>
    </tr>
  </tbody>
</table></div><hr />

<p>Third generation sequencing techniques will be available in the near
future. The main feature of the third generation is the ability to
sequence the whole single molecule instead of breaking down the molecule
into short reads, therefore the read lengths should be much longer than
those of the second generation sequencing technologies. Several strong
candidates that may lead the third generation sequencing are Pacific
Bioscience SMRT Sequencing (<a href="https://www.pacificbiosciences.com">https://www.pacificbiosciences.com</a>), Oxford
Nanopore technologies (<a href="https://www.nanoporetech.com/">https://www.nanoporetech.com/</a>), and Life
technologies Single Molecule Sequencing
(<a href="https://www.lifetechnologies.com">https://www.lifetechnologies.com</a>).</p>
      <h2 id="S:4.3">
        
        
          <a href="#S:4.3" class="anchor-heading" aria-labelledby="S:4.3"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 4.3 The second generation sequencing technologies can cover a wide range of applications
        
        
      </h2>
    

<p>The second generation sequencing can be used in many different
applications because of its high-throughput and cost effectiveness. For
example, the applications can be transcript rearrangement discovery,
single-nucleotide variation profiling, and non-coding RNA discovery
<a class="citation" href="#Morozova2009">[4]</a>. Two such applications, RNA-Seq and CLIP (Cross-Linking
and ImmunoPrecipitation)-Seq, are very powerful and useful for
transcriptomic analyses.</p>

<p>RNA-Seq or the whole transcriptome shotgun sequencing (WTSS) is a
technique that uses the second generation sequencing technology to
produce sequence reads at the whole transcriptome level
<a class="citation" href="#Morin2008">[15,16]</a>. Since the second generation sequencing
technology can yield sufficient sequencing depth, which represents the
total number of sequence reads generated from a sequencing library
<a class="citation" href="#Morozova2009">[4]</a>, RNA-Seq can be used for gene expression profiling with
high accuracy.</p>

<p>CLIP-Seq <a class="citation" href="#Sanford2009">[17]</a>, also called HITS-CLIP (High throughput
sequencing CLIP) <a class="citation" href="#Licatalosi2008">[18]</a>, is a technique that employs three
important steps, cross-linking, immunoprecipitation, and next generation
sequencing. It can be used to tag and pull-down RNA-interacting proteins
of interest and infer the interactions between RNAs and RNA-binding
proteins. Firstly, RNA binding proteins and their target RNA regions are
cross-linked by ultraviolet (UV) light, and the antibodies for the
proteins are used for immunoprecipitation <a class="citation" href="#Jensen2008">[19]</a>. Subsequently,
the RNA transcripts pulled down with the proteins go through the second
generation sequencing procedure <a class="citation" href="#Sanford2009">[17]</a>. RIP (Ribonucleoprotein
ImmunoPrecipitation)-Seq is similar to CLIP-Seq <a class="citation" href="#Wong2009b">[20]</a>, but it
uses chemical cross-linkers such as formaldehyde instead of UV
cross-linking <a class="citation" href="#Niranjanakumari2002">[21,22]</a>. This cross-linking is
reversible, and it is subject to potential reassociation between RNAs
and RNA-binding proteins after cell lysis in some cases <a class="citation" href="#Mili2004">[23]</a>.</p>
      <h2 id="S:4.4">
        
        
          <a href="#S:4.4" class="anchor-heading" aria-labelledby="S:4.4"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 4.4 Liquid chromatography-tandem mass spectrometry is a powerful tool to analyze quantitative proteomics
        
        
      </h2>
    

<p>In recent years, several new technologies for the identification and
quantification of proteins have emerged. Most of them are based on
mass-spectrometry (MS), which is a technique that ionizes molecules and
measures the mass-to-charge ratio by detecting them in an
electromagnetic field <a class="citation" href="#Boggess2001">[24]</a>. Although there are many variants
of MS-based technologies <a class="citation" href="#Mallick2010">[25]</a>, the Liquid
chromatography-tandem mass spectrometry (LC-MS/MS) with stable isotope
labeling with amino acids in cell culture (SILAC) <a class="citation" href="#Ong2002">[26]</a> approach is
widely used in detecting protein expression profiles.</p>

<p>LC-MS/MS uses high-performance liquid chromatography that can separate a
mixture of molecules with very small particles and a high pressure
before the MS/MS phase. MS/MS, or tandem mass spectrometry, involves two
steps of MS selections. The first MS can be used for the quantification
of peptides, and the second MS can be used for the identification of the
peptides <a class="citation" href="#Mallick2010">[25]</a>.</p>

<p>LC-MS/MS is usually combined with either labeling or labeling-free
methods for a quantification approach. For example, SILAC tends to be
used for small changes (10%-50%), and isotope tags for relative and
absolute quantification (iTRAQ), which is another labeling method, tends
to be used for moderate changes (50%-200%) <a class="citation" href="#Mallick2010">[25]</a>. Moreover, a
labeling-free method using spectrum counts can be used for large
changes (&gt;100%) <a class="citation" href="#Mallick2010">[25]</a>. Among them, SILAC is a simple
but very powerful method for quantitative proteomics <a class="citation" href="#Ong2005">[27]</a>. The
SILAC procedure uses two different stable amino acid isotopes, as
“light” and “heavy” labels. The relative abundance of proteins can be
detected by comparing the intensities of isotope clusters
<a class="citation" href="#Mallick2010">[25,27]</a>.</p>

<p>The coverage of protein identification in the genome is usually less
than 10% for higher organisms <a class="citation" href="#Bantscheff2007a">[28]</a> due to enormous
molecular complexity and the dynamic nature of proteins, such as
post-translational modifications and protein stability <a class="citation" href="#Mallick2010">[25]</a>.
However, the protein coverage of quantification is even lower than the
protein identification. One possible explanation for this low coverage
is that protein quantification requires much higher data quality, in
terms of information content, than protein identification
<a class="citation" href="#Bantscheff2007a">[28]</a>.</p>
      <h2 id="S:4.5">
        
        
          <a href="#S:4.5" class="anchor-heading" aria-labelledby="S:4.5"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 4.5 Most preprocessed and raw data sets from high-throughput experiments are publicly available
        
        
      </h2>
    

<p>Two major repositories for microarray experiment data are ArrayExpress
(<a href="https://www.ebi.ac.uk/arrayexpress">https://www.ebi.ac.uk/arrayexpress</a>) <a class="citation" href="#Parkinson2011">[29]</a> at European
Bioinformatics Institute (EBI), and Gene Expression Omnibus (GEO)
(<a href="https://www.ncbi.nlm.nih.gov/geo">https://www.ncbi.nlm.nih.gov/geo</a>) <a class="citation" href="#Barrett2006">[30]</a> at National Center
for Biotechnology Information (NCBI). Both repositories encourage
submitters to supply Minimum Information About a Microarray Experiment
(MIAME) <a class="citation" href="#Brazma2001">[31]</a> compliant data. MIAME is a standard for the
microarray data formats.
caption
The major repository for the next generation sequencing data is the
Sequence Read Archive (SRA), which is operated by the International
Nucleotide Sequence Database Collaboration (INSDC) <a class="citation" href="#Leinonen2011">[32]</a>.
However, due to a rapid growth of next generation sequencing data and
budget constrains, NCBI, which is the main member of INSDS, currently
accepts limited types and forms of the next generation sequencing data.</p>

<p>There are no central repositories for quantitative proteomics data, but
many small and medium scale public repositories are available instead.
Some of the examples of such repositories <a class="citation" href="#Mead2009">[33]</a> are Proteomics
IDEntifications database (PRIDE) <a class="citation" href="#Jones2006">[34]</a>, the Global Proteome
Machine database (GPMDB) <a class="citation" href="#Craig2004">[35]</a>, and PeptideAtlas <a class="citation" href="#Deutsch2010">[36]</a>.</p>
      <h2 id="S:4.6">
        
        
          <a href="#S:4.6" class="anchor-heading" aria-labelledby="S:4.6"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> References
        
        
      </h2>
    
<ol class="bibliography"><li><span id="Alwine1977">Alwine JC, Kemp DJ, Stark GR. Method for detection of specific RNAs in agarose gels by transfer to diazobenzyloxymethyl-paper and hybridization with DNA probes. Proceedings of the National Academy of Sciences 1977;74:5350–4. <a href="https://doi.org/10.1073/pnas.74.12.5350">https://doi.org/10.1073/pnas.74.12.5350.</a></span>
</li>
<li><span id="Becker-Andre1989">Becker-André M, Hahlbrock K. Absolute mRNA quantification using the polymerase chain reaction (PCR). A novel approach by aPCR aidedtranscipttitration assay (PATTY). Nucleic Acids Research 1989;17:9437–46. <a href="https://doi.org/10.1093/nar/17.22.9437">https://doi.org/10.1093/nar/17.22.9437.</a></span>
</li>
<li><span id="Noonan1990">Noonan KE, Beck C, Holzmayer TA, Chin JE, Wunder JS, Andrulis IL, et al. Quantitative analysis of MDR1 (multidrug resistance) gene expression in human tumors by polymerase chain reaction. Proceedings of the National Academy of Sciences 1990;87:7160–4. <a href="https://doi.org/10.1073/pnas.87.18.7160">https://doi.org/10.1073/pnas.87.18.7160.</a></span>
</li>
<li><span id="Morozova2009">Morozova O, Hirst M, Marra MA. Applications of new sequencing technologies for transcriptome analysis. Annual Review of Genomics and Human Genetics 2009;10:135–51. <a href="https://doi.org/10.1146/annurev-genom-082908-145957">https://doi.org/10.1146/annurev-genom-082908-145957.</a></span>
</li>
<li><span id="VanGuilder2008">VanGuilder HD, Vrana KE, Freeman WM. Twenty-five years of quantitative PCR for gene expression analysis. BioTechniques 2008;44:619–26. <a href="https://doi.org/10.2144/000112776">https://doi.org/10.2144/000112776.</a></span>
</li>
<li><span id="Quackenbush2006d">Quackenbush J. Microarray analysis and tumor classification. New England Journal of Medicine 2006;354:2463–72. <a href="https://doi.org/10.1056/nejmra042342">https://doi.org/10.1056/nejmra042342.</a></span>
</li>
<li><span id="Patterson2006">Patterson TA, Lobenhofer EK, Fulmer-Smentek SB, Collins PJ, Chu T-M, Bao W, et al. Performance comparison of one-color and two-color platforms within the Microarray Quality Control (MAQC) project. Nature Biotechnology 2006;24:1140–50. <a href="https://doi.org/10.1038/nbt1242">https://doi.org/10.1038/nbt1242.</a></span>
</li>
<li><span id="Quackenbush2002">Quackenbush J. Microarray data normalization and transformation. Nature Genetics 2002;32:496–501. <a href="https://doi.org/10.1038/ng1032">https://doi.org/10.1038/ng1032.</a></span>
</li>
<li><span id="Mockler2005">Mockler TC, Ecker JR. Applications of DNA tiling arrays for whole-genome analysis. Genomics 2005;85:1–15. <a href="https://doi.org/10.1016/j.ygeno.2004.10.005">https://doi.org/10.1016/j.ygeno.2004.10.005.</a></span>
</li>
<li><span id="Sanger1977">Sanger F, Nicklen S, Coulson AR. DNA sequencing with chain-terminating inhibitors. Proceedings of the National Academy of Sciences 1977;74:5463–7. <a href="https://doi.org/10.1073/pnas.74.12.5463">https://doi.org/10.1073/pnas.74.12.5463.</a></span>
</li>
<li><span id="Adams1991">Adams M, Kelley J, Gocayne J, Dubnick M, Polymeropoulos M, Xiao H, et al. Complementary DNA sequencing: expressed sequence tags and human genome project. Science 1991;252:1651–6. <a href="https://doi.org/10.1126/science.2047873">https://doi.org/10.1126/science.2047873.</a></span>
</li>
<li><span id="Velculescu1995">Velculescu VE, Zhang L, Vogelstein B, Kinzler KW. Serial analysis of gene expression. Science 1995;270:484–7. <a href="https://doi.org/10.1126/science.270.5235.484">https://doi.org/10.1126/science.270.5235.484.</a></span>
</li>
<li><span id="Voelkerding2010">Voelkerding KV, Dames S, Durtschi JD. Next generation sequencing for clinical diagnostics-principles and application to targeted resequencing for hypertrophic cardiomyopathy. The Journal of Molecular Diagnostics 2010;12:539–51. <a href="https://doi.org/10.2353/jmoldx.2010.100043">https://doi.org/10.2353/jmoldx.2010.100043.</a></span>
</li>
<li><span id="Hall2007">Hall N. Advanced sequencing technologies and their wider impact in microbiology. Journal of Experimental Biology 2007;210:1518–25. <a href="https://doi.org/10.1242/jeb.001370">https://doi.org/10.1242/jeb.001370.</a></span>
</li>
<li><span id="Morin2008">Morin RD, Bainbridge M, Fejes A, Hirst M, Krzywinski M, Pugh TJ, et al. Profiling the HeLa S3 transcriptome using randomly primed cDNA and massively parallel short-read sequencing. BioTechniques 2008;45:81–94. <a href="https://doi.org/10.2144/000112900">https://doi.org/10.2144/000112900.</a></span>
</li>
<li><span id="Nagalakshmi2008">Nagalakshmi U, Wang Z, Waern K, Shou C, Raha D, Gerstein M, et al. The transcriptional landscape of the yeast genome defined by RNA sequencing. Science 2008;320:1344–9. <a href="https://doi.org/10.1126/science.1158441">https://doi.org/10.1126/science.1158441.</a></span>
</li>
<li><span id="Sanford2009">Sanford JR, Wang X, Mort M, VanDuyn N, Cooper DN, Mooney SD, et al. Splicing factor SFRS1 recognizes a functionally diverse landscape of RNA transcripts. Genome Research 2008;19:381–94. <a href="https://doi.org/10.1101/gr.082503.108">https://doi.org/10.1101/gr.082503.108.</a></span>
</li>
<li><span id="Licatalosi2008">Licatalosi DD, Mele A, Fak JJ, Ule J, Kayikci M, Chi SW, et al. HITS-CLIP yields genome-wide insights into brain alternative RNA processing. Nature 2008;456:464–9. <a href="https://doi.org/10.1038/nature07488">https://doi.org/10.1038/nature07488.</a></span>
</li>
<li><span id="Jensen2008">Jensen KB, Darnell RB. CLIP: crosslinking and ImmunoPrecipitation of in vivo RNA targets of RNA-binding proteins. Methods in Molecular Biology, vol. 488, Humana Press; 2008, p. 85–98. <a href="https://doi.org/10.1007/978-1-60327-475-3_6">https://doi.org/10.1007/978-1-60327-475-3_6.</a></span>
</li>
<li><span id="Wong2009b">Wong E, Wei C-L. ChIP’ing the mammalian genome: technical advances and insights into functional elements. Genome Medicine 2009;1:89. <a href="https://doi.org/10.1186/gm89">https://doi.org/10.1186/gm89.</a></span>
</li>
<li><span id="Niranjanakumari2002">Niranjanakumari S, Lasda E, Brazas R, Garcia-Blanco MA. Reversible cross-linking combined with immunoprecipitation to study RNA-protein interactions in vivo. Methods 2002;26:182–90. <a href="https://doi.org/10.1016/s1046-2023(02)00021-x">https://doi.org/10.1016/s1046-2023(02)00021-x.</a></span>
</li>
<li><span id="Conrad2008">Conrad NK. Co-immunoprecipitation techniques for assessing RNA-protein interactions in vivo. Methods in Enzymology, vol. 449, Elsevier; 2008, p. 317–42. <a href="https://doi.org/10.1016/s0076-6879(08)02415-4">https://doi.org/10.1016/s0076-6879(08)02415-4.</a></span>
</li>
<li><span id="Mili2004">Mili S, Steitz JA. Evidence for reassociation of RNA-binding proteins after cell lysis: Implications for the interpretation of immunoprecipitation analyses. RNA 2004;10:1692–4. <a href="https://doi.org/10.1261/rna.7151404">https://doi.org/10.1261/rna.7151404.</a></span>
</li>
<li><span id="Boggess2001">Boggess B. Mass spectrometry desk reference (Sparkman, O. David). Journal of Chemical Education 2001;78:168. <a href="https://doi.org/10.1021/ed078p168.2">https://doi.org/10.1021/ed078p168.2.</a></span>
</li>
<li><span id="Mallick2010">Mallick P, Kuster B. Proteomics: a pragmatic perspective. Nature Biotechnology 2010;28:695–709. <a href="https://doi.org/10.1038/nbt.1658">https://doi.org/10.1038/nbt.1658.</a></span>
</li>
<li><span id="Ong2002">Ong S-E, Blagoev B, Kratchmarova I, Kristensen DB, Steen H, Pandey A, et al. Stable isotope labeling by amino acids in cell culture, SILAC, as a simple and accurate approach to expression proteomics. Molecular &amp; Cellular Proteomics 2002;1:376–86. <a href="https://doi.org/10.1074/mcp.m200025-mcp200">https://doi.org/10.1074/mcp.m200025-mcp200.</a></span>
</li>
<li><span id="Ong2005">Ong S-E, Mann M. Mass spectrometry-based proteomics turns quantitative. Nature Chemical Biology 2005;1:252–62. <a href="https://doi.org/10.1038/nchembio736">https://doi.org/10.1038/nchembio736.</a></span>
</li>
<li><span id="Bantscheff2007a">Bantscheff M, Schirle M, Sweetman G, Rick J, Kuster B. Quantitative mass spectrometry in proteomics: a critical review. Analytical and Bioanalytical Chemistry 2007;389:1017–31. <a href="https://doi.org/10.1007/s00216-007-1486-6">https://doi.org/10.1007/s00216-007-1486-6.</a></span>
</li>
<li><span id="Parkinson2011">Parkinson H, Sarkans U, Kolesnikov N, Abeygunawardena N, Burdett T, Dylag M, et al. ArrayExpress update–an archive of microarray and high-throughput sequencing-based functional genomics experiments. Nucleic Acids Research 2010;39:D1002–D1004. <a href="https://doi.org/10.1093/nar/gkq1040">https://doi.org/10.1093/nar/gkq1040.</a></span>
</li>
<li><span id="Barrett2006">Barrett T, Edgar R. [19] Gene expression omnibus: microarray data storage, submission, retrieval, and analysis. Methods in Enzymology, vol. 411, Elsevier; 2006, p. 352–69. <a href="https://doi.org/10.1016/s0076-6879(06)11019-8">https://doi.org/10.1016/s0076-6879(06)11019-8.</a></span>
</li>
<li><span id="Brazma2001">Brazma A, Hingamp P, Quackenbush J, Sherlock G, Spellman P, Stoeckert C, et al. Minimum information about a microarray experiment (MIAME)—toward standards for microarray data. Nature Genetics 2001;29:365–71. <a href="https://doi.org/10.1038/ng1201-365">https://doi.org/10.1038/ng1201-365.</a></span>
</li>
<li><span id="Leinonen2011">Leinonen R, Sugawara H, Shumway M. The sequence read archive. Nucleic Acids Research 2010;39:D19–D21. <a href="https://doi.org/10.1093/nar/gkq1019">https://doi.org/10.1093/nar/gkq1019.</a></span>
</li>
<li><span id="Mead2009">Mead JA, Bianco L, Bessant C. Recent developments in public proteomic MS repositories and pipelines. PROTEOMICS 2009;9:861–81. <a href="https://doi.org/10.1002/pmic.200800553">https://doi.org/10.1002/pmic.200800553.</a></span>
</li>
<li><span id="Jones2006">Jones P, Côté RG, Martens L, Quinn AF, Taylor CF, Derache W, et al. PRIDE: a public repository of protein and peptide identifications for the proteomics community. Nucleic Acids Research 2006;34:D659–D663. <a href="https://doi.org/10.1093/nar/gkj138">https://doi.org/10.1093/nar/gkj138.</a></span>
</li>
<li><span id="Craig2004">Craig R, Cortens JP, Beavis RC. Open source system for analyzing, validating, and storing protein identification data. Journal of Proteome Research 2004;3:1234–42. <a href="https://doi.org/10.1021/pr049882h">https://doi.org/10.1021/pr049882h.</a></span>
</li>
<li><span id="Deutsch2010">Deutsch EW. The PeptideAtlas project. Methods in Molecular Biology, vol. 604, Humana Press; 2009, p. 285–96. <a href="https://doi.org/10.1007/978-1-60761-444-9_19">https://doi.org/10.1007/978-1-60761-444-9_19.</a></span>
</li></ol>

        

        

        
          <hr>
          <h2>Leave a comment</h2>
          <div id="disqus_thread"></div>
        

        
        
          <hr>
          <footer>
            
              <p><a href="#top" id="back-to-top">Back to top</a></p>
            

            <p class="text-small text-grey-dk-100 mb-0">Copyright &copy; 2021 Takaya Saito.</p>

            
              <div class="d-flex mt-2">
                
                
              </div>
            
          </footer>
        

      </div>
    </div>

    
      

      <div class="search-overlay"></div>
    
  </div>

  
     <script>addBackToTop({
       id: 'back-to-top-js-button'
     })</script>
  

  
    <script>
        /**
        *  RECOMMENDED CONFIGURATION VARIABLES: EDIT AND UNCOMMENT THE SECTION BELOW TO INSERT DYNAMIC VALUES FROM YOUR PLATFORM OR CMS.
        *  LEARN WHY DEFINING THESE VARIABLES IS IMPORTANT: https://disqus.com/admin/universalcode/#configuration-variables    */
        /*
        var disqus_config = function () {
        this.page.url = PAGE_URL;  // Replace PAGE_URL with your page's canonical URL variable
        this.page.identifier = PAGE_IDENTIFIER; // Replace PAGE_IDENTIFIER with your page's unique identifier variable
        };
        */
        (function() { // DON'T EDIT BELOW THIS LINE
        var d = document, s = d.createElement('script');
        s.src = 'https://mitkb.disqus.com/embed.js';
        s.setAttribute('data-timestamp', +new Date());
        (d.head || d.body).appendChild(s);
        })();
    </script>
    <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
  

</body>
</html>

