

<!DOCTYPE html>

<html lang="en-US">
<head>
  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=Edge">

  

  <link rel="shortcut icon" href="/mitkb/favicon.ico" type="image/x-icon">

  <link rel="stylesheet" href="/mitkb/assets/css/just-the-docs-default.css">

  
    <script async src="https://www.googletagmanager.com/gtag/js?id=G-84ETYTKDBN"></script>
    <script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'G-84ETYTKDBN', { 'anonymize_ip': true });
    </script>

  

  
    <script type="text/javascript" src="/mitkb/assets/js/vendor/lunr.min.js"></script>
  
  <script type="text/javascript" src="/mitkb/assets/js/just-the-docs.js"></script>

  <meta name="viewport" content="width=device-width, initial-scale=1">

  <!-- Begin Jekyll SEO tag v2.7.1 -->
<title>3 miRNAs and ncRNAs | mitkb</title>
<meta name="generator" content="Jekyll v4.2.0" />
<meta property="og:title" content="3 miRNAs and ncRNAs" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="MicroRNAs and other non-coding RNAs" />
<meta property="og:description" content="MicroRNAs and other non-coding RNAs" />
<link rel="canonical" href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/3-micrornas-and-other-non-coding-rnas/" />
<meta property="og:url" content="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/3-micrornas-and-other-non-coding-rnas/" />
<meta property="og:site_name" content="mitkb" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="3 miRNAs and ncRNAs" />
<script type="application/ld+json">
{"description":"MicroRNAs and other non-coding RNAs","url":"https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/3-micrornas-and-other-non-coding-rnas/","@type":"WebPage","headline":"3 miRNAs and ncRNAs","publisher":{"@type":"Organization","logo":{"@type":"ImageObject","url":"https://ncrnas.github.io/mitkb/assets/images/logo.png"}},"@context":"https://schema.org"}</script>
<!-- End Jekyll SEO tag -->


  
<meta name="google-site-verification" content="vstkqjcoJe9vQcOOpsCKrxv6WfqZ2SKRNWSv7XoHFeQ" />

<link rel="apple-touch-icon" sizes="180x180" href="/mitkb/assets/images/favicon/apple-touch-icon.png">
<link rel="icon" type="image/png" sizes="32x32" href="/mitkb/assets/images/favicon/favicon-32x32.png">
<link rel="icon" type="image/png" sizes="16x16" href="/mitkb/assets/images/favicon/favicon-16x16.png">
<link rel="manifest" href="/mitkb/assets/images/favicon/site.webmanifest">
<link rel="mask-icon" href="/mitkb/assets/images/favicon/safari-pinned-tab.svg" color="#5bbad5">
<meta name="msapplication-TileColor" content="#2b5797">
<meta name="theme-color" content="#ffffff">

<!-- for mathjax support -->



  <script src="https://unpkg.com/vanilla-back-to-top@7.2.1/dist/vanilla-back-to-top.min.js"></script>


<script
  data-cookie-notice='{ "learnMoreLinkEnabled": true, "learnMoreLinkHref": "/mitkb/terms/" }'
  src="https://unpkg.com/cookie-notice@^1/dist/cookie.notice.min.js"
></script>


</head>

<body>
  <svg xmlns="http://www.w3.org/2000/svg" style="display: none;">
    <symbol id="svg-link" viewBox="0 0 24 24">
      <title>Link</title>
      <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-link">
        <path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path>
      </svg>
    </symbol>
    <symbol id="svg-search" viewBox="0 0 24 24">
      <title>Search</title>
      <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-search">
        <circle cx="11" cy="11" r="8"></circle><line x1="21" y1="21" x2="16.65" y2="16.65"></line>
      </svg>
    </symbol>
    <symbol id="svg-menu" viewBox="0 0 24 24">
      <title>Menu</title>
      <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-menu">
        <line x1="3" y1="12" x2="21" y2="12"></line><line x1="3" y1="6" x2="21" y2="6"></line><line x1="3" y1="18" x2="21" y2="18"></line>
      </svg>
    </symbol>
    <symbol id="svg-arrow-right" viewBox="0 0 24 24">
      <title>Expand</title>
      <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-chevron-right">
        <polyline points="9 18 15 12 9 6"></polyline>
      </svg>
    </symbol>
    <symbol id="svg-doc" viewBox="0 0 24 24">
      <title>Document</title>
      <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-file">
        <path d="M13 2H6a2 2 0 0 0-2 2v16a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V9z"></path><polyline points="13 2 13 9 20 9"></polyline>
      </svg>
    </symbol>
  </svg>

  <div class="side-bar">
    <div class="site-header">
      <a href="https://ncrnas.github.io/mitkb/" class="site-title lh-tight">
  <div class="site-logo"></div>

</a>
      <a href="#" id="menu-button" class="site-button">
        <svg viewBox="0 0 24 24" class="icon"><use xlink:href="#svg-menu"></use></svg>
      </a>
    </div>
    <nav role="navigation" aria-label="Main" id="site-nav" class="site-nav">
      
        <ul class="nav-list"><li class="nav-list-item"><a href="https://ncrnas.github.io/mitkb/" class="nav-list-link">Home</a></li><li class="nav-list-item active"><a href="#" class="nav-list-expander"><svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg></a><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/" class="nav-list-link">Overview</a><ul class="nav-list "><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/0-front-matter/" class="nav-list-link">0 Front matter</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/1-introduction/" class="nav-list-link">1 Introduction</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/2-papers-and-their-corresponding-sub-goals/" class="nav-list-link">2 Papers and sub-goals</a></li><li class="nav-list-item  active"><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/3-micrornas-and-other-non-coding-rnas/" class="nav-list-link active">3 miRNAs and ncRNAs</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/4-high-throughput-biological-experiments/" class="nav-list-link">4 Biological experiments</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/5-statistical-tests-and-methods/" class="nav-list-link">5 Statistical methods</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/6-machine-learning-theory-and-support-vector-machine/" class="nav-list-link">6 Support vector machine</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/7-computational-implementation/" class="nav-list-link">7 Implementation</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/8-future-perspectives/" class="nav-list-link">8 Future perspectives</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander"><svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg></a><a href="https://ncrnas.github.io/mitkb/mirna-target-prediction/" class="nav-list-link">Target prediction</a><ul class="nav-list "><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/mirna-target-prediction/micrornas-targeting-and-target-prediction/" class="nav-list-link">Review: miRNA target</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/mirna-target-prediction/two-step-svm-model/" class="nav-list-link">Research: Two-step SVM</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/mirna-target-prediction/two-step-svm-model-supplementary-information/" class="nav-list-link">Supplementary information</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander"><svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg></a><a href="https://ncrnas.github.io/mitkb/mirna-high-throughput-experiments/" class="nav-list-link">Biological experiments</a><ul class="nav-list "><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/mirna-high-throughput-experiments/confounding-factors-in-mirna-experiments/" class="nav-list-link">Research: Confounding</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/mirna-high-throughput-experiments/confounding-factors-in-mirna-experiments-supplementary-information/" class="nav-list-link">Supplementary information</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander"><svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg></a><a href="https://ncrnas.github.io/mitkb/mirna-and-other-ncrnas/" class="nav-list-link">Other ncRNAs</a><ul class="nav-list "></ul></li><li class="nav-list-item"><a href="https://ncrnas.github.io/mitkb/support-articles/" class="nav-list-link">Support articles</a></li><li class="nav-list-item"><a href="https://ncrnas.github.io/mitkb/contact" class="nav-list-link">Contact us</a></li></ul>

      
    </nav>
    <footer class="site-footer">
      <a href="https://ncrnas.github.io/mitkb/terms">Terms and Privacy Policy</a> <br /><br />
      This site uses <a href="https://github.com/pmarsceill/just-the-docs">Just the Docs</a>, a documentation theme for Jekyll.
    </footer>
  </div>
  <div class="main" id="top">
    <div id="main-header" class="main-header">
      
        <div class="search">
          <div class="search-input-wrap">
            <input type="text" id="search-input" class="search-input" tabindex="0" placeholder="Search mitkb" aria-label="Search mitkb" autocomplete="off">
            <label for="search-input" class="search-label"><svg viewBox="0 0 24 24" class="search-icon"><use xlink:href="#svg-search"></use></svg></label>
          </div>
          <div id="search-results" class="search-results"></div>
        </div>
      
      
      
        <nav aria-label="Auxiliary" class="aux-nav">
          <ul class="aux-nav-list">
            
              <li class="aux-nav-list-item">
                <a href="https://github.com/ncrnas/mitkb" class="site-button"
                  
                >
                  mitkb on github
                </a>
              </li>
            
          </ul>
        </nav>
      
    </div>
    <div id="main-content-wrap" class="main-content-wrap">
      
        <nav aria-label="Breadcrumb" class="breadcrumb-nav">
            <ol class="breadcrumb-nav-list">
              
                <li class="breadcrumb-nav-list-item"><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/">Overview</a></li>
              
              <li class="breadcrumb-nav-list-item"><span>3 miRNAs and ncRNAs</span></li>
            </ol>
          </nav>
        
      
      <div id="main-content" class="main-content" role="main">
        
          <h1 class="no_toc" id="3-micrornas-and-other-non-coding-rnas">
        
        
          <a href="#3-micrornas-and-other-non-coding-rnas" class="anchor-heading" aria-labelledby="3-micrornas-and-other-non-coding-rnas"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 3 MicroRNAs and other non-coding RNAs
        
        
      </h1>
    <hr />

<details open="">
  <summary class="text-delta">
    Table of contents
  </summary>
<ol id="markdown-toc">
  <li><a href="#T:3.1" id="markdown-toc-T:3.1">3.1 Thousands of miRNAs have been identified since the first miRNA discovery of <em>lin-4</em> in 1993</a></li>
  <li><a href="#T:3.2" id="markdown-toc-T:3.2">3.2 MicroRNA biogenesis involves multiple steps</a></li>
  <li><a href="#T:3.3" id="markdown-toc-T:3.3">3.3 RNA interference is the central mechanism for gene regulation by miRNAs and small interfering RNA</a></li>
  <li><a href="#T:3.4" id="markdown-toc-T:3.4">3.4 MicroRNAs have various regulatory roles that are associated with important physiological and pathological processes</a></li>
  <li><a href="#T:3.5" id="markdown-toc-T:3.5">3.5 Multiple properties of miRNA target recognition may enhance target efficacy in animal</a></li>
  <li><a href="#T:3.6" id="markdown-toc-T:3.6">3.6 MicroRNA binding also occurs outside of the 3’ UTR</a></li>
  <li><a href="#T:3.7" id="markdown-toc-T:3.7">3.7 Some of coding and non-coding pairs of cis-NATs potentially may have regulatory interactions with miRNAs</a></li>
  <li><a href="#T:3.8" id="markdown-toc-T:3.8">3.8 Chromatin associated RNAs are potentially associated with the modification of chromatin structure</a></li>
  <li><a href="#S:3.9" id="markdown-toc-S:3.9">References</a></li>
</ol>

</details><hr />

<p>Since our main goal is to reveal the miRNA regulation and
characteristics, this chapter introduces several different aspects of
miRNAs, such as the history of miRNA discovery, miRNA biogenesis, and
mechanism of miRNA regulation. In addition to miRNAs, it also describes
several other classes of ncRNAs and their potential interactions with
miRNAs. Specifically, analysis on such ncRNAs is the main objective of
the third sub-goal: <em><a href="/mitkb/mirna-and-other-ncrnas/">miRNA and other ncRNAs</a></em>.</p>
      <h2 id="T:3.1">
        
        
          <a href="#T:3.1" class="anchor-heading" aria-labelledby="T:3.1"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 3.1 Thousands of miRNAs have been identified since the first miRNA discovery of <em>lin-4</em> in 1993
        
        
      </h2>
    

<p>In 1993, Lee et al. found that <em>lin-4</em>, a gene involved in development
timing in <em>C. elegans</em>, produces a small ncRNA instead of a messenger
RNA (mRNA) <a class="citation" href="#Lee1993">[1]</a>. <em>lin-4</em> was known to regulate <em>lin-14</em>, but the
protein product of <em>lin-4</em> had been undetected. The result of an
alignment analysis indicated that <em>lin-4</em> has multiple complementary
sites on the 3’ UTR of <em>lin-14</em> <a class="citation" href="#Lee1993">[1]</a>. Further experiment
revealed that this small ncRNA produced by <em>lin-4</em> can directly suppress
the expression of <em>lin-14</em> by base-paring on the 3’ UTR. This
was the first discovery of this functional ncRNA with about 22
nucleotides that regulates specific protein expression by base-paring on
the 3’ UTR of its target mRNA <a class="citation" href="#Bartel2004">[2]</a>. However, there were
no other <em>lin-4</em>-like small ncRNAs identified, and this peculiar
regulation of <em>lin-4</em> was recognized as a rare case <a class="citation" href="#Bartel2004">[2]</a>.</p>

<p>Meanwhile, Fire et al. reported that they observed a gene silencing
effect after injecting double-stranded RNAs (dsRNAs) into <em>C. elegans</em>
<a class="citation" href="#Fire1998">[3]</a> in 1998. They coined the term, RNA interference (RNAi), to
describe this gene-silencing mechanism. Soon thereafter, RNAi was found
to silence genes at the post-transcriptional level with small RNA
molecules with 20-25 nucleotides, called small interfering RNAs (siRNAs)
<a class="citation" href="#Hamilton1999">[4,5,6]</a>.</p>

<p>Then, nearly seven years after the discovery of <em>lin-4</em>, Reinhart et al.
identified a <em>lin-4</em>-like small ncRNA, <em>let-7</em>, in <em>C. elegans</em> in 2000
<a class="citation" href="#Reinhart2000">[7]</a>. They revealed that <em>let-7</em> is also an ncRNA with about
22 nucleotides that regulates specific protein expression by base-paring
on the 3’ UTR. Since <em>let-7</em> has homologs in various species,
this discovery led to identification of many other <em>let-7</em>- and
<em>lin-4</em>-like small ncRNAs in other animals, including human and
<em>Drosophila</em> <a class="citation" href="#Pasquinelli2000">[8]</a>. The term microRNA (miRNA) was coined to
refer to these small ncRNAs of about 22 nucleotide length
<a class="citation" href="#Lagos-Quintana2001">[9,10,11]</a>. Moreover, like siRNAs, miRNAs
appeared to use the RNAi pathway to regulated genes at the
post-transcriptional level <a class="citation" href="#Bartel2004">[2]</a>.</p>

<p>Today, miRNAs are recognized as a very common class of ncRNAs that can
regulate protein expression
<a class="citation" href="#Lagos-Quintana2001">[9,10,11,2]</a>. For instance,
miRBase
<a class="citation" href="#Griffiths-Jones2004">[12,13,14,15]</a>,
which is the main database for miRNA annotations, contains 15172 entries
in 142 species as of release 16, 2010. They are abundant and found
mostly in eukaryotes as well as in some viruses. MicroRNAs are known to
play many important regulatory roles in eukaryotes <a class="citation" href="#Bartel2004">[2]</a>,
whereas some viruses encode viral miRNA genes that potentially regulate
fundamental cellular processes both in the viruses and in their host
cells <a class="citation" href="#Nair2006">[16,17]</a>.</p>
      <h2 id="T:3.2">
        
        
          <a href="#T:3.2" class="anchor-heading" aria-labelledby="T:3.2"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 3.2 MicroRNA biogenesis involves multiple steps
        
        
      </h2>
    

<p>Although the precise mechanism of miRNA biogenesis is unknown, Figure
<a href="#F:3.1">3.1</a> shows the most widely accepted view of the miRNA biogenesis to date. The
biogenesis involves multiple processes both in the nucleus and the
cytoplasm.</p>
      <h3 id="F:3.1" class="no_toc">
        
        
          <a href="#F:3.1" class="anchor-heading" aria-labelledby="F:3.1"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a>  
        
        
      </h3>
    
<p><img src="/mitkb/assets/images/thesis/microrna-biogenesis.webp" alt="Figure 3.1" width="546" /></p>
      <h4 class="no_toc text-caption" id="figure-31-microrna-biogenesis">
        
        
          <a href="#figure-31-microrna-biogenesis" class="anchor-heading" aria-labelledby="figure-31-microrna-biogenesis"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> <strong>Figure 3.1</strong>. MicroRNA biogenesis.
        
        
      </h4>
    
<p class="fs-2">The figure shows the overview of miRNA
biogenesis. Pol II transcribes ssRNA from DNA. The ssRNA forms
pri-miRNA, that is further processed to a hairpin loop structure called
pre-miRNA by DGCR8 and Drosha. Exportin 5 together with Ran-GTP exports
pre-miRNA into the cytoplasm. Dicer cleaves pre-miRNA to form miRNA
duplex. Only one strand of the miRNA duplex is usually bound to the
Argonaute protein and loaded into the RISC
complex.</p><hr />

<p>First, RNA polymerase II (Pol II) transcribes a miRNA gene on the
chromosome from DNA to single-stranded RNA (ssRNA) with 5’ cap
and poly-A tail <a class="citation" href="#Lee2004">[18]</a>. This ssRNA called primary miRNA (pri-miRNA)
can be several hundreds or thousands nucleotides long, and it may
contain one or more hairpin loops <a class="citation" href="#Lee2002">[19]</a>. An enzyme called DiGeorge
Syndrome Critical Region 8 (DGCR8) recognizes the hairpin loop in
pri-miRNA, and a DGCR8 associated enzyme, called Drosha, cleaves the
hairpin from the pri-miRNA
<a class="citation" href="#Lee2003">[20,21,22,23,24]</a>. This
cleavage results in a hairpin structure with approximately 60
nucleotides called precursor miRNA (pre-miRNA)
<a class="citation" href="#Saetrom2006">[25,20,26,27]</a>. Exportin 5 together
with Ran-GTP exports pre-miRNAs from the nucleus to the cytoplasm
<a class="citation" href="#Lund2004">[28,29]</a>.</p>

<p>In the cytoplasm, a Ribonuclease (RNase) III enzyme, called Dicer,
cleaves the loop of pre-miRNA and produces a miRNA:miRNA* duplex with
approximately 22 nucleotides <a class="citation" href="#Lee2003">[20]</a>. Only one strand of this duplex
usually becomes a mature miRNA as a guide to the target mRNA, and the
other strand, defined as miRNA star-strand or miRNA* <a class="citation" href="#Lau2001">[10]</a>, is
eventually degraded <a class="citation" href="#Bartel2004">[2]</a>. The mechanism of the strand selection
is unclear, but a strand that is less thermodynamically stable at its
5’ end appears to be favored in some cases
<a class="citation" href="#Tomari2005">[30,31]</a>. Argonaute (Ago) proteins are key proteins for
miRNA targeting <a class="citation" href="#Fagard2000b">[32]</a>. Ago2, which is one of the Ago clade
proteins, is mainly associated with mature miRNAs in mammals
<a class="citation" href="#Carthew2009">[33]</a>. A protein complex called RNA-Induced Silencing Complex
(RISC) <a class="citation" href="#Hammond2000">[34]</a> that incorporates Ago2, uses the mature miRNA as a
guide to bind and then catalyze specific target mRNAs <a class="citation" href="#Carthew2009">[33]</a>.</p>

<p>Moreover, there are several known alternative pathways for miRNA
biogenesis. For instance, some intronic miRNAs bypass Drosha processing
by directly forming a pre-miRNA-like hairpin structure. These intronic
miRNAs are defined as mitrons <a class="citation" href="#Okamura2007">[35,36]</a> first identified
in <em>Drosophila</em> and <em>C. elegans</em> <a class="citation" href="#Ruby2007">[36]</a>, and also found in mammals
<a class="citation" href="#Berezikov2007">[37]</a>. Another example of alternative miRNA pathways is that
RNA polymerase III (Pol III) instead of Pol II transcribes some miRNAs
<a class="citation" href="#Bartel2004">[2]</a>, especially those residing upstream of Alu sequences
<a class="citation" href="#Borchert2006">[38]</a>. Alu sequences or Alu elements are abundant mobile
elements especially found in the primate genomes <a class="citation" href="#Batzer2002">[39]</a>.</p>
      <h2 id="T:3.3">
        
        
          <a href="#T:3.3" class="anchor-heading" aria-labelledby="T:3.3"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 3.3 RNA interference is the central mechanism for gene regulation by miRNAs and small interfering RNA
        
        
      </h2>
    

<p>MicroRNAs and siRNA regulate and control gene expression through RNAi
<a class="citation" href="#Mello2004">[6,33]</a>. Although siRNAs have biochemically
indistinguishable mature forms of ssRNAs from those of miRNAs, the siRNA
biogenesis pathway to its mature form is different from that of miRNA
<a class="citation" href="#Bartel2004">[2]</a>. Dicer processes siRNA precursors, such as long dsRNAs or
small hairpin RNAs (shRNAs), and cleaves them into a ~22 nt dsRNA with 2-nt 3’ overhangs
<a class="citation" href="#Bartel2004">[2,40,33]</a>. This dsRNA form of siRNA is
essentially equivalent with the miRNA:miRNA* duplex.</p>

<p>Major functionalities of siRNAs and miRNAs are that siRNAs are defenders
against foreign or invasive nucleic acid molecules such as viruses,
transposons, and transgenes <a class="citation" href="#Mello2004">[6]</a>, whereas miRNAs are regulators
of endogenous protein-coding genes <a class="citation" href="#Carthew2009">[33]</a>. Moreover, exogenous
siRNAs are widely used in gene knockdown experiments, and they are
potentially useful for gene therapy. Exogenous siRNAs are directly
introduced into the cytoplasm or taken up from the environment
<a class="citation" href="#Mello2004">[6,33]</a>.</p>

<p>The central mechanism of RNAi is that si/miRNAs loaded in RISC act like
guides to bind the sites of their target mRNAs. The RISCs loaded with
miRNAs are called miRISCs, whereas the RISCs with siRNAs are called
siRISCs <a class="citation" href="#Carthew2009">[33]</a>. RNAi has several different gene silencing modes
(Fig. <a href="#F:3.2">3.2</a>). The common gene silencing mode for siRNAs
and plant miRNAs is cleaving mRNAs that have near-perfect complementary
sites with the si/miRNAs (Fig. <a href="#F:3.2">3.2</a>)<a class="citation" href="#Bartel2004">[2]</a>.
Two known gene silencing modes for animal miRNAs are
transcriptional repression and mRNA degradation (Fig. <a href="#F:3.2">3.2</a>).
In either mode, miRNAs mainly target mRNAs
that have partial complementary sites on their 3’ UTRs
<a class="citation" href="#Bartel2004">[2]</a>. Although translational repression was initially thought
as the major regulatory model of animal miRNAs, a recent study used
ribosome profiling assay and reported that most target genes of animal
miRNAs were actually degraded <a class="citation" href="#Guo2010">[41]</a>. The precise mechanism of this
degradation is still unclear, but it is possibly associated with
deadenylation, decapping, and exonucleolytic digestion of the mRNA
<a class="citation" href="#Behm-Ansmant2006a">[42,43,44]</a>.</p>
      <h3 id="F:3.2" class="no_toc">
        
        
          <a href="#F:3.2" class="anchor-heading" aria-labelledby="F:3.2"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a>  
        
        
      </h3>
    
<p><img src="/mitkb/assets/images/thesis/types-of-microrna-targeting.webp" alt="Figure 3.2" width="473" /></p>
      <h4 class="no_toc text-caption" id="figure-32-mirna-target">
        
        
          <a href="#figure-32-mirna-target" class="anchor-heading" aria-labelledby="figure-32-mirna-target"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> <strong>Figure 3.2</strong>. miRNA target.
        
        
      </h4>
    
<p class="fs-2">The figure shows three examples of RNAi regulation
by miRISC and siRISC. The long curved line represents mRNA that is
separated into three regions, 5’ UTR, CDS, and 3’ UTR.
Ribosomes synthesize peptides while moving through the mRNA from
5’ to 3’ direction. One siRISC binds on the CDS and two
miRISCs bind on the 3’ UTR of mRNA, in this example. Plant miRNA
and siRNA have nearly perfect complementary which results in mRNA
cleavage. Animal miRNAs require only partial complementary, and they
either repress translation or contribute to mRNA
degradation.</p><hr />

<p>Moreover, exogenous siRNAs are known to act like miRNAs and
down-regulate numerous unintended mRNAs in the same way as miRNA gene
silencing. This unintended effect is called siRNA off-targeting
<a class="citation" href="#Jackson2003">[45]</a>. Considering this siRNA off-targeting is very important
to design effective exogenous siRNAs.</p>
      <h2 id="T:3.4">
        
        
          <a href="#T:3.4" class="anchor-heading" aria-labelledby="T:3.4"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 3.4 MicroRNAs have various regulatory roles that are associated with important physiological and pathological processes
        
        
      </h2>
    

<p>Many miRNAs play important regulatory roles by negatively controlling
the expression level of mRNAs <a class="citation" href="#Bartel2004">[2]</a>, and current estimates
indicate that at least 60% of human protein-coding genes are under some
influence of miRNAs <a class="citation" href="#Friedman2009">[46]</a>. Many miRNAs, like <em>lin-4</em> and
<em>let-7</em>, are involved in cell development processes <a class="citation" href="#Stefani2008">[47]</a>.
Other experimentally validated miRNA regulatory roles can be found in
many cellular processes, such as growth control, differentiation, stem
cell and germline proliferation, and apoptosis <a class="citation" href="#Alvarez-Garcia2005">[48]</a>.
However, annotations of many miRNA regulations are still poor,
therefore, predicting accurate miRNA target genes is important to infer
miRNA regulatory roles.</p>

<p>MicroRNAs are also associated with human diseases because of their wide
range of gene regulatory roles <a class="citation" href="#Alvarez-Garcia2005">[48]</a>. Several studies
revealed miRNA involved diseases, such as cancer <a class="citation" href="#He2005">[49,50]</a>,
heart disease <a class="citation" href="#Chen2008">[51,52]</a>, DiGeorge syndrome
<a class="citation" href="#Alvarez-Garcia2005">[48]</a>, Alzheimer’s disease, and central nerve system
(CNS) disorders <a class="citation" href="#Maes2009">[53]</a>. Moreover, some viruses encode viral miRNA
genes <a class="citation" href="#Nair2006">[16]</a>. These viral miRNAs potentially regulate fundamental
cellular processes both in the viruses and in their host cells
<a class="citation" href="#Boss2009">[17]</a>.</p>
      <h2 id="T:3.5">
        
        
          <a href="#T:3.5" class="anchor-heading" aria-labelledby="T:3.5"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 3.5 Multiple properties of miRNA target recognition may enhance target efficacy in animal
        
        
      </h2>
    

<p>Since most animal miRNAs have only partial complementary to their target
mRNAs <a class="citation" href="#Bartel2009">[54]</a>, miRNA targeting usually requires additional
features for better target recognition. The most important feature is
the seed type, which is the region for the partial complementary. The
seed site contains six nucleotides from position 2 to 7 of the miRNA
<a class="citation" href="#Lewis2003">[55,56,54]</a>, as the position begins with 1
at the 5’ end of the miRNA. Even though the definition of the
seed site is ubiquitous, the definition of the seed types is different
among different studies. Figure <a href="#F:3.3">3.3</a>
shows nine common seed types that are widely accepted in many studies.
Seed types consist of stringent and non-stringent groups. Three seed
types, 7mer-A1, 7mer-m8, and 8mer, belong to the stringent group (Fig.
<a href="#F:3.3">3.3</a>). These seed types have perfect Watson-Crick
paring in their seed sites, and they are usually stronger than those in
the non-stringent groups in terms of miRNA target recognition
<a class="citation" href="#Bartel2009">[54]</a>. 7mer-A1 has an adenine (A) at position 1 of the target
mRNA. An adenine at position 1 of the mRNA is known to enhance miRNA
target recognition <a class="citation" href="#Lewis2005">[57]</a>. 7mer-m8 has paring at position 8. 8mer
has an adenine at position 1 and paring at position 8. The non-stringent
group consists of 6mer, two G:U wobble, one loop, and two bulge types
(Fig. <a href="#F:3.3">3.3</a>) <a class="citation" href="#Gaidatzis2007">[58]</a>. They are less effective
than the stringent group, but they are still functional since miRISC can
tolerate some mismatches <a class="citation" href="#Ellwanger2011">[59]</a>. 6mer has perfect seed paring,
whereas the other types are equivalent to 8mers except one mismatch or
wobble paring. LP has a loop in the seed site. GUM has a G:U wobble site
on the miRNA whereas GUT has a wobble site on the target mRNA.
Similarly, BM has a bulge on the miRNA whereas BT has a bulge on the
target mRNA.</p>
      <h3 id="F:3.3" class="no_toc">
        
        
          <a href="#F:3.3" class="anchor-heading" aria-labelledby="F:3.3"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a>  
        
        
      </h3>
    
<p><img src="/mitkb/assets/images/thesis/types-of-microrna-seed-sequences.webp" alt="Figure 3.3" width="400" /></p>
      <h4 class="no_toc text-caption" id="figure-33-mirna-seed-types">
        
        
          <a href="#figure-33-mirna-seed-types" class="anchor-heading" aria-labelledby="figure-33-mirna-seed-types"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> <strong>Figure 3.3</strong>. miRNA seed types.
        
        
      </h4>
    
<p class="fs-2">Examples of three stringent seed types (8mer,
7mer-A1, and 7mer-m8), and six non-stringent seed types (6mer, GUM, GUT,
LP, BM, and MT). The strand on the top of each seed type represents
position 1-8/9 of miRNA, and the bottom strand represents target
mRNAs.</p><hr />

<p>Additional paring at the 3’ part of miRNAs can increase the
efficacy of miRNA repression, and it can also compensate for a seed
mismatch to create a functional site <a class="citation" href="#Bartel2009">[54]</a>. Three to four
matches at position 13-16 for stringent seeds, and four to five matches
at position 13-19 for non-stringent seeds are known as 3’
supplementary and 3’ compensatory paring, respectively
<a class="citation" href="#Grimson2007">[60,54]</a>.</p>

<p>Many target sites are well conserved among closely related species
<a class="citation" href="#Friedman2009">[46]</a>. However, there are many approaches to define “well
conserved” targets. For instance, some use perfect seed matches among
several species <a class="citation" href="#Krek2005">[61,62,63]</a>, whereas others use
pre-defined conservation scores calculated by global phylogenetic
analysis <a class="citation" href="#Betel2010">[64,65]</a>. Moreover, even though many targets
are well conserved, some targets are also species-specific. For
instance, one study predicted that about 30% of all experimentally
validated targets are poorly conserved <a class="citation" href="#Sethupathy2006">[66]</a>.</p>

<p>The site accessibility of miRISC can be measured by computing the
secondary structure of target sites through minimum free energy
approaches. The site accessibility is potentially a very strong feature
to predict true miRNA target sites because it is directly linked to
target recognition. However, the precise mechanism of miRISC access on
its targets is unknown, and developing a precise prediction model with
mRNA secondary structure calculation usually requires huge computational
power. Some models used elaborate two step approaches with the first
step as initial forming of miRNA:mRNA complex, and the second step as
hybrid elongation to form the complete miRNA:mRNA complex
<a class="citation" href="#Kertesz2007">[65,67]</a>. An alternative feature to site accessibility
is to measure the occurrence of AU rich elements both upstream and
downstream of the seed site <a class="citation" href="#Grimson2007">[60]</a>. Thermodynamic stabilities of
AU base pairs are much lower than that of GC base pairs, and it can be a
reason that sites surrounded by AU-rich context has better site
accessibility for miRISCs. Moreover, this AU rich approach is
computationally inexpensive with a good prediction performance
<a class="citation" href="#Grimson2007">[60]</a>. Scoring AU context is a more reliable measurement of
target accessibility than any other currently available models with mRNA
secondary structure calculation <a class="citation" href="#Bartel2009">[54]</a>.</p>

<p>Multiple target sites enhance miRNA target efficacy <a class="citation" href="#John2004">[68]</a>.
Although the general effect of multiple sites is additive, the effect
can be synergistic when two miRNA targets are within optimal distance.
One reported such optimal distance is defined as two seed sites
separated by between 13 and 35 nt <a class="citation" href="#Saetrom2007">[69]</a>. For example, a gene
with two 7mers within optimal distance is more down-regulated than a
gene with a single 8mer. However, the effect is not apparent when two
7mers are not within optimal distance. In this case, a gene with 8mer is
more down-regulated than a gene with two 7mers <a class="citation" href="#Bartel2009">[54]</a>.</p>

<p>To summarize miRNA target recognition and efficacy, most animal miRNAs
bind on the 3’ region of their target mRNAs by base paring. The
seed site, which is located at position 2 to 7 of the miRNA, is usually
used for this base paring, and the seed type tends to be one of the most
important features. There are also other additional features that can
contribute to target recognition and efficacy, such as 3’
additional paring, site conservation, site accessibility, and
cooperability of multiple target sites.</p>
      <h2 id="T:3.6">
        
        
          <a href="#T:3.6" class="anchor-heading" aria-labelledby="T:3.6"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 3.6 MicroRNA binding also occurs outside of the 3’ UTR
        
        
      </h2>
    

<p>Although the precise regulatory mechanism of miRISC’s binding on the
3’ UTR is unclear, it may cause deadenylation, decapping, and
exonucleolytic digestion of the mRNA
<a class="citation" href="#Behm-Ansmant2006a">[42,43,44]</a>. Moreover, miRISCs bind
other regions that reside outside of 3’ UTRs, such as CDS
<a class="citation" href="#Hafner2010">[70]</a>, 5’ UTR <a class="citation" href="#Lee2009">[71,70]</a>, and promoter
regions <a class="citation" href="#Kim2008">[72,73]</a>.</p>

<p>Many miRNAs bind CDS regions; however, most of the sites are likely
non-functional because ribosomes seem to detach miRISCs from their
binding site while moving along the CDS region for translation
<a class="citation" href="#Bartel2004">[2]</a>. Nonetheless, there are several evidences that some miRNA
target sites in CDS are functional. One example is that rare codons in
CDS regions tend to make ribosomes stalled, and miRNA target sites right
after these codons can be functional <a class="citation" href="#Gu2009">[74]</a>. Moreover, some target
sites experimentally validated in CDS tend to have one very strong site
<a class="citation" href="#Duursma2008">[75,76]</a>, or multiple sites within optimal distance
<a class="citation" href="#Forman2008">[77,78]</a>.</p>

<p>Some miRNAs also target 5’ UTR regions, though it is much less
common than 3’ UTR and CDS regions <a class="citation" href="#Bartel2009">[54]</a>. One intriguing
class of miRNA targets involved in 5’ UTR is miBridge targets
<a class="citation" href="#Lee2009">[71]</a>. The miBridge target is a miRNA target site that has a normal
seed site on the 3’ UTR and a 5’ portion paring on the
5’ UTR simultaneously. One possible explanation for the
regulation of miBridge is that miBridge is involved in the translational
initiation by preventing ribosome scanning through the 5’ UTR
<a class="citation" href="#Lee2009">[71]</a>.</p>

<p>MicroRNA regulation can also occur inside the nucleus despite that
miRNA’s major regulatory roles are in the cytoplasm
<a class="citation" href="#Hwang2007">[79,80]</a>. The miRNA regulation in the nucleus is likely
at the transcriptional level rather than the translational level.
Several studies reported that miRNAs cause transcriptional silencing by
targeting promoter regions <a class="citation" href="#Kim2008">[72,73]</a>.</p>
      <h2 id="T:3.7">
        
        
          <a href="#T:3.7" class="anchor-heading" aria-labelledby="T:3.7"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 3.7 Some of coding and non-coding pairs of cis-NATs potentially may have regulatory interactions with miRNAs
        
        
      </h2>
    

<p>A complex locus is a locus that contains several genes that interact
among each other <a class="citation" href="#Engstrom2006">[81]</a>. Two major classes of complex loci are
cis-natural antisense transcripts (cis-NATs) and bi-directionally
promoters (Fig. <a href="#F:3.4">3.4</a>) <a class="citation" href="#Engstrom2006">[81]</a>. Many complex loci have
important regulatory roles, even though the precise mechanism is unknown
<a class="citation" href="#Duhig1998">[82,83]</a>.</p>
      <h3 id="F:3.4" class="no_toc">
        
        
          <a href="#F:3.4" class="anchor-heading" aria-labelledby="F:3.4"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a>  
        
        
      </h3>
    
<p><img src="/mitkb/assets/images/thesis/complex-loci.webp" alt="Figure 3.4" width="297" /></p>
      <h4 class="no_toc text-caption" id="figure-34-complex-loci">
        
        
          <a href="#figure-34-complex-loci" class="anchor-heading" aria-labelledby="figure-34-complex-loci"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> <strong>Figure 3.4</strong>. Complex loci.
        
        
      </h4>
    
<p class="fs-2">Complex loci consist of multiple genes that
interact among each other. Two major classes of complex loci are
cis-NATs and bi-directional promoters. Cis-NATs can be divided into
three categories depending on the directions of the overlaps. “Head to
Head” is that sense and antisense transcripts are partially overlapped
on their 5’ ends. “Tail to Tail” is that sense and antisense
transcripts are partially overlapped on their 3’ ends. “Full
overlap” is that one transcript is fully overlapped with the other
transcript. Bi-directional promoters reside between two genes arranged
head-to-head on opposite strands with less than 1000 base pairs
separating their transcription start sites
<a class="citation" href="#Trinklein2004a">[84]</a>.</p><hr />

<p>Cis-NATs is a sense-antisense pair of transcripts that are partially
overlapping in the same locus <a class="citation" href="#Wang2005">[85]</a>. Cis-NATs are relatively
common in many species, and they are quite abundant in human
<a class="citation" href="#Chen2004">[86,81,87]</a>. Three major classes, or
orientations, of cis-NATs are “head to head”, “tail to tail”, and “full
overlap” (Fig. <a href="#F:3.4">3.4</a>) <a class="citation" href="#Osato2007">[88]</a>. Although the molecular mechanism
of cis-NAT regulation is poorly understood, three models may explain the
potential cis-NAT regulation <a class="citation" href="#Osato2007">[88]</a>. The first model is the
transcriptional collision model <a class="citation" href="#Osato2007">[88]</a>, which can be the main
mechanism for “head to head” cis-NATs. The second model is the dsRNA
formation model of sense and antisense transcripts <a class="citation" href="#Osato2007">[88]</a>. One
study showed that an endogenous ncRNA derived from a cis-NAT pair
regulates its anti-sense transcript of a protein coding gene in plants
<a class="citation" href="#Borsani2005">[89]</a>. The third model is that cis-NATs are involved in
epigenetic regulation. This model is based on the evidence that some
ncRNAs are involved in the modification of chromatin structure and DNA
methylation in the promoter region <a class="citation" href="#Tufarelli2003">[90,91]</a>.</p>

<p>Bi-directional promoters reside between two genes arranged head-to-head
on opposite strands with less than 1000 base pairs separating their
transcription start sites <a class="citation" href="#Trinklein2004a">[84]</a>. Many pairs from
bi-directional promoters are co-expressed, but some are antiregulated
<a class="citation" href="#Trinklein2004a">[84]</a>. Bi-directional promoters are abundant; for instance,
they represent approximately 10% of all the genes in human
<a class="citation" href="#Trinklein2004a">[84]</a>.</p>

<p>The main objective of the third sub-goal: <em><a href="/mitkb/mirna-and-other-ncrnas/">miRNA and other ncRNAs</a></em> is to
investigate potential miRNA interactions with other ncRNAs. Of these
classes of complex loci, some miRNAs potentially interact with
ncRNA:mRNA pairs of cis-NATs. In this case, miRNAs indirectly regulate
the expression of the mRNAs in cis-NATs through directly regulating
their paired ncRNAs (Fig. <a href="#F:3.5">3.5</a>). Bi-directional promoters may also have
ncRNA:mRNA pairs that potentially involve miRNAs regulation. Annotated
data of bi-directional promoters are available for mRNA:mRNA pairs
<a class="citation" href="#Engstrom2006">[81]</a>, but there are few reliable data for ncRNA:mRNA pairs of
bi-directional promoters. Therefore, we excluded bi-directional
promoters and focused on cis-NATs, especially on ncRNA:mRNA pairs of
cis-NATs, in the third sub-goal: <em><a href="/mitkb/mirna-and-other-ncrnas/">miRNA and other ncRNAs</a></em>.</p>
      <h3 id="F:3.5" class="no_toc">
        
        
          <a href="#F:3.5" class="anchor-heading" aria-labelledby="F:3.5"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a>  
        
        
      </h3>
    
<p><img src="/mitkb/assets/images/thesis/microrna-regulation-on-cis-nat.webp" alt="Figure 3.5" width="438" /></p>
      <h4 class="no_toc text-caption" id="figure-35-mirna-regulation-on-cis-nat">
        
        
          <a href="#figure-35-mirna-regulation-on-cis-nat" class="anchor-heading" aria-labelledby="figure-35-mirna-regulation-on-cis-nat"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> <strong>Figure 3.5</strong>. miRNA regulation on cis-NAT.
        
        
      </h4>
    
<p class="fs-2">The figure illustrates two different
potential modes of miRNA regulation on cis-NATs. Direct miRNA regulation
is a normal regulatory mode of miRNAs, in which miRNA binds on the
3’ UTR of mRNA. Indirect mode is that miRNA regulates the
protein coding mRNA in cis-NAT indirectly through binding the non-coding
transcript.</p><hr />
      <h2 id="T:3.8">
        
        
          <a href="#T:3.8" class="anchor-heading" aria-labelledby="T:3.8"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 3.8 Chromatin associated RNAs are potentially associated with the modification of chromatin structure
        
        
      </h2>
    

<p>Chromatin associated RNAs (CARs) are experimentally validated non-coding
RNAs that can bind a part of the chromatin directly <a class="citation" href="#Mondal2010">[92]</a>. Since
CARs affect their host and neighboring genes <a class="citation" href="#Mondal2010">[92]</a>, CARs can be
seen as a class of complex loci. CARs are likely involved in the
regulation of chromatin structure by recruiting chromatin-modifying
complexes (Fig. <a href="#F:3.6">3.6</a>). This scenario is supported by the evidence that
long ncRNAs regulate chromatin modification by guiding chromatin
remodeling complexes to specific genome loci
<a class="citation" href="#Mercer2009">[93,94]</a>. In addition to long ncRNAs, RNAi is also
known to have roles in the regulation of chromatin structure. For
instance, in fission yeast, CARs serve as assembly platform to the
RNA-induced initiation of transcriptional gene-silencing (RITS) complex.
In this case of fission yeast, siRNAs associate with AGO1 and guide the
RITS complex to CARs <a class="citation" href="#Beisel2011">[91]</a>.</p>

<p>Similar to ncRNA:mRNA pairs of cis-NATs, CARs potentially have
interactions with miRNAs, though there is currently no strong evidence
to support this. Therefore, we chose cis-NATs and CARs to investigate
their potential interactions with miRNAs in the third sub-goal: <em><a href="/mitkb/mirna-and-other-ncrnas/">miRNA and other ncRNAs</a></em>.</p>
      <h3 id="F:3.6" class="no_toc">
        
        
          <a href="#F:3.6" class="anchor-heading" aria-labelledby="F:3.6"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a>  
        
        
      </h3>
    
<p><img src="/mitkb/assets/images/thesis/chromatin-associated-rna.webp" alt="Figure 3.6" width="301" /></p>
      <h4 class="no_toc text-caption" id="figure-36-cars">
        
        
          <a href="#figure-36-cars" class="anchor-heading" aria-labelledby="figure-36-cars"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> <strong>Figure 3.6</strong>. CARs.
        
        
      </h4>
    
<p class="fs-2">Upper panel shows a protein complex, a CAR, and
nucleosomes. The complementary DNA region to the CAR is indicated in
red. Lower panel shows one example of the CAR regulation to the
chromatin. The CAR acts as a guide for the protein complex that can
modify the chromatin structure.</p><hr />
      <h2 id="S:3.9">
        
        
          <a href="#S:3.9" class="anchor-heading" aria-labelledby="S:3.9"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> References
        
        
      </h2>
    
<ol class="bibliography"><li><span id="Lee1993">Lee RC, Feinbaum RL, Ambros V. The C. elegans heterochronic gene lin-4 encodes small RNAs with antisense complementarity to lin-14. Cell 1993;75:843–54. <a href="https://doi.org/10.1016/0092-8674(93)90529-y">https://doi.org/10.1016/0092-8674(93)90529-y.</a></span>
</li>
<li><span id="Bartel2004">Bartel DP. MicroRNAs: genomics, biogenesis, mechanism, and function. Cell 2004;116:281–97. <a href="https://doi.org/10.1016/s0092-8674(04)00045-5">https://doi.org/10.1016/s0092-8674(04)00045-5.</a></span>
</li>
<li><span id="Fire1998">Fire A, Xu SQ, Montgomery MK, Kostas SA, Driver SE, Mello CC. Potent and specific genetic interference by double-stranded RNA in Caenorhabditis elegans. Nature 1998;391:806–11. <a href="https://doi.org/10.1038/35888">https://doi.org/10.1038/35888.</a></span>
</li>
<li><span id="Hamilton1999">Hamilton AJ, Baulcombe DC. A species of small antisense RNA in posttranscriptional gene silencing in plants. Science 1999;286:950–2. <a href="https://doi.org/10.1126/science.286.5441.950">https://doi.org/10.1126/science.286.5441.950.</a></span>
</li>
<li><span id="Elbashir2001">Elbashir SM, Harborth J, Lendeckel W, Yalcin A, Weber K, Tuschl T. Duplexes of 21-nucleotide RNAs mediate RNA interference in cultured mammalian cells. Nature 2001;411:494–8. <a href="https://doi.org/10.1038/35078107">https://doi.org/10.1038/35078107.</a></span>
</li>
<li><span id="Mello2004">Mello CC, Conte D. Revealing the world of RNA interference. Nature 2004;431:338–42. <a href="https://doi.org/10.1038/nature02872">https://doi.org/10.1038/nature02872.</a></span>
</li>
<li><span id="Reinhart2000">Reinhart BJ, Slack FJ, Basson M, Pasquinelli AE, Bettinger JC, Rougvie AE, et al. The 21-nucleotide let-7 RNA regulates developmental timing in Caenorhabditis elegans. Nature 2000;403:901–6. <a href="https://doi.org/10.1038/35002607">https://doi.org/10.1038/35002607.</a></span>
</li>
<li><span id="Pasquinelli2000">Pasquinelli AE, Reinhart BJ, Slack F, Martindale MQ, Kuroda MI, Maller B, et al. Conservation of the sequence and temporal expression of let-7 heterochronic regulatory RNA. Nature 2000;408:86–9. <a href="https://doi.org/10.1038/35040556">https://doi.org/10.1038/35040556.</a></span>
</li>
<li><span id="Lagos-Quintana2001">Lagos-Quintana M, Rauhut R, Lendeckel W, Tuschl T. Identification of novel genes coding for small expressed RNAs. Science 2001;294:853–8. <a href="https://doi.org/10.1126/science.1064921">https://doi.org/10.1126/science.1064921.</a></span>
</li>
<li><span id="Lau2001">Lau NC, Lim LP, Weinstein EG, Bartel DP. An abundant class of tiny RNAs with probable regulatory roles in caenorhabditis elegans. Science 2001;294:858–62. <a href="https://doi.org/10.1126/science.1065062">https://doi.org/10.1126/science.1065062.</a></span>
</li>
<li><span id="Lee2001">Lee RC, Ambros V. An extensive class of small RNAs in Caenorhabditis elegans. Science 2001;294:862–4. <a href="https://doi.org/10.1126/science.1065329">https://doi.org/10.1126/science.1065329.</a></span>
</li>
<li><span id="Griffiths-Jones2004">Griffiths-Jones S. The microRNA registry. Nucleic Acids Research 2004;32:109D–111. <a href="https://doi.org/10.1093/nar/gkh023">https://doi.org/10.1093/nar/gkh023.</a></span>
</li>
<li><span id="Griffiths-Jones2006">Griffiths-Jones S. miRBase: the MicroRNA sequence database. MicroRNA Protocols, vol. 342, Humana Press; 2006, p. 129–38. <a href="https://doi.org/10.1385/1-59745-123-1:129">https://doi.org/10.1385/1-59745-123-1:129.</a></span>
</li>
<li><span id="Griffiths-Jones2008">Griffiths-Jones S, Saini HK, Dongen S van, Enright AJ. miRBase: tools for microRNA genomics. Nucleic Acids Research 2007;36:D154–D158. <a href="https://doi.org/10.1093/nar/gkm952">https://doi.org/10.1093/nar/gkm952.</a></span>
</li>
<li><span id="Kozomara2011">Kozomara A, Griffiths-Jones S. miRBase: integrating microRNA annotation and deep-sequencing data. Nucleic Acids Research 2010;39:D152–D157. <a href="https://doi.org/10.1093/nar/gkq1027">https://doi.org/10.1093/nar/gkq1027.</a></span>
</li>
<li><span id="Nair2006">Nair V, Zavolan M. Virus-encoded microRNAs: novel regulators of gene expression. Trends in Microbiology 2006;14:169–75. <a href="https://doi.org/10.1016/j.tim.2006.02.007">https://doi.org/10.1016/j.tim.2006.02.007.</a></span>
</li>
<li><span id="Boss2009">Boss IW, Plaisance KB, Renne R. Role of virus-encoded microRNAs in herpesvirus biology. Trends in Microbiology 2009;17:544–53. <a href="https://doi.org/10.1016/j.tim.2009.09.002">https://doi.org/10.1016/j.tim.2009.09.002.</a></span>
</li>
<li><span id="Lee2004">Lee Y, Kim M, Han J, Yeom K-H, Lee S, Baek SH, et al. MicroRNA genes are transcribed by RNA polymerase II. The EMBO Journal 2004;23:4051–60. <a href="https://doi.org/10.1038/sj.emboj.7600385">https://doi.org/10.1038/sj.emboj.7600385.</a></span>
</li>
<li><span id="Lee2002">Lee Y, Jeon K, Lee J-T, Kim S, Kim VN. MicroRNA maturation: stepwise processing and subcellular localization. The EMBO Journal 2002;21:4663–70. <a href="https://doi.org/10.1093/emboj/cdf476">https://doi.org/10.1093/emboj/cdf476.</a></span>
</li>
<li><span id="Lee2003">Lee Y, Ahn C, Han J, Choi H, Kim J, Yim J, et al. The nuclear RNase III Drosha initiates microRNA processing. Nature 2003;425:415–9. <a href="https://doi.org/10.1038/nature01957">https://doi.org/10.1038/nature01957.</a></span>
</li>
<li><span id="Denli2004">Denli AM, Tops BBJ, Plasterk RHA, Ketting RF, Hannon GJ. Processing of primary microRNAs by the Microprocessor complex. Nature 2004;432:231–5. <a href="https://doi.org/10.1038/nature03049">https://doi.org/10.1038/nature03049.</a></span>
</li>
<li><span id="Gregory2004">Gregory RI, Yan K-ping, Amuthan G, Chendrimada T, Doratotaj B, Cooch N, et al. The Microprocessor complex mediates the genesis of microRNAs. Nature 2004;432:235–40. <a href="https://doi.org/10.1038/nature03120">https://doi.org/10.1038/nature03120.</a></span>
</li>
<li><span id="Han2004">Han J, Lee Y, Yeom K-H, Kim Y-K, Jin H, Kim VN. The Drosha-DGCR8 complex in primary microRNA processing. Genes &amp; Development 2004;18:3016–27. <a href="https://doi.org/10.1101/gad.1262504">https://doi.org/10.1101/gad.1262504.</a></span>
</li>
<li><span id="Landthaler2004">Landthaler M, Yalcin A, Tuschl T. The human DiGeorge syndrome critical region gene 8 and its d. Melanogaster homolog are required for miRNA biogenesis. Current Biology 2004;14:2162–7. <a href="https://doi.org/10.1016/j.cub.2004.11.001">https://doi.org/10.1016/j.cub.2004.11.001.</a></span>
</li>
<li><span id="Saetrom2006">Sætrom P, Snøve O, Nedland M, Grünfeld TB, Lin Y, Bass MB, et al. Conserved MicroRNA characteristics in mammals. Oligonucleotides 2006;16:115–44. <a href="https://doi.org/10.1089/oli.2006.16.115">https://doi.org/10.1089/oli.2006.16.115.</a></span>
</li>
<li><span id="Zeng2003">Zeng Y, Yi R, Cullen BR. MicroRNAs and small interfering RNAs can inhibit mRNA expression by similar mechanisms. Proceedings of the National Academy of Sciences 2003;100:9779–84. <a href="https://doi.org/10.1073/pnas.1630797100">https://doi.org/10.1073/pnas.1630797100.</a></span>
</li>
<li><span id="Batuwita2009">Batuwita R, Palade V. microPred: effective classification of pre-miRNAs for human miRNA gene prediction. Bioinformatics 2009;25:989–95. <a href="https://doi.org/10.1093/bioinformatics/btp107">https://doi.org/10.1093/bioinformatics/btp107.</a></span>
</li>
<li><span id="Lund2004">Lund E, Güttinger S, Calado A, Dahlberg JE, Kutay U. Nuclear export of MicroRNA precursors. Science 2004;303:95–8. <a href="https://doi.org/10.1126/science.1090599">https://doi.org/10.1126/science.1090599.</a></span>
</li>
<li><span id="Yi2003">Yi R, Qin Y, Macara IG, Cullen BR. Exportin-5 mediates the nuclear export of pre-microRNAs and short hairpin RNAs. Genes &amp; Development 2003;17:3011–6. <a href="https://doi.org/10.1101/gad.1158803">https://doi.org/10.1101/gad.1158803.</a></span>
</li>
<li><span id="Tomari2005">Tomari Y, Zamore PD. Perspective: machines for RNAi. Genes &amp; Development 2005;19:517–29. <a href="https://doi.org/10.1101/gad.1284105">https://doi.org/10.1101/gad.1284105.</a></span>
</li>
<li><span id="Kim2005">Kim VN. MicroRNA biogenesis: coordinated cropping and dicing. Nature Reviews Molecular Cell Biology 2005;6:376–85. <a href="https://doi.org/10.1038/nrm1644">https://doi.org/10.1038/nrm1644.</a></span>
</li>
<li><span id="Fagard2000b">Fagard M, Boutet S, Morel J-B, Bellini C, Vaucheret H. AGO1, QDE-2, and RDE-1 are related proteins required for post-transcriptional gene silencing in plants, quelling in fungi, and RNA interference in animals. Proceedings of the National Academy of Sciences 2000;97:11650–4. <a href="https://doi.org/10.1073/pnas.200217597">https://doi.org/10.1073/pnas.200217597.</a></span>
</li>
<li><span id="Carthew2009">Carthew RW, Sontheimer EJ. Origins and mechanisms of miRNAs and siRNAs. Cell 2009;136:642–55. <a href="https://doi.org/10.1016/j.cell.2009.01.035">https://doi.org/10.1016/j.cell.2009.01.035.</a></span>
</li>
<li><span id="Hammond2000">Hammond SM, Bernstein E, Beach D, Hannon GJ. An RNA-directed nuclease mediates post-transcriptional gene silencing in Drosophila cells. Nature 2000;404:293–6. <a href="https://doi.org/10.1038/35005107">https://doi.org/10.1038/35005107.</a></span>
</li>
<li><span id="Okamura2007">Okamura K, Hagen JW, Duan H, Tyler DM, Lai EC. The mirtron pathway generates microRNA-class regulatory RNAs in Drosophila. Cell 2007;130:89–100. <a href="https://doi.org/10.1016/j.cell.2007.06.028">https://doi.org/10.1016/j.cell.2007.06.028.</a></span>
</li>
<li><span id="Ruby2007">Ruby JG, Jan CH, Bartel DP. Intronic microRNA precursors that bypass Drosha processing. Nature 2007;448:83–6. <a href="https://doi.org/10.1038/nature05983">https://doi.org/10.1038/nature05983.</a></span>
</li>
<li><span id="Berezikov2007">Berezikov E, Chung W-J, Willis J, Cuppen E, Lai EC. Mammalian mirtron genes. Molecular Cell 2007;28:328–36. <a href="https://doi.org/10.1016/j.molcel.2007.09.028">https://doi.org/10.1016/j.molcel.2007.09.028.</a></span>
</li>
<li><span id="Borchert2006">Borchert GM, Lanier W, Davidson BL. RNA polymerase III transcribes human microRNAs. Nature Structural &amp; Molecular Biology 2006;13:1097–101. <a href="https://doi.org/10.1038/nsmb1167">https://doi.org/10.1038/nsmb1167.</a></span>
</li>
<li><span id="Batzer2002">Batzer MA, Deininger PL. Alu repeats and human genomic diversity. Nature Reviews Genetics 2002;3:370–9. <a href="https://doi.org/10.1038/nrg798">https://doi.org/10.1038/nrg798.</a></span>
</li>
<li><span id="Paddison2002">Paddison PJ, Caudy AA, Bernstein E, Hannon GJ, Conklin DS. Short hairpin RNAs (shRNAs) induce sequence-specific silencing in mammalian cells. Genes &amp; Development 2002;16:948–58. <a href="https://doi.org/10.1101/gad.981002">https://doi.org/10.1101/gad.981002.</a></span>
</li>
<li><span id="Guo2010">Guo H, Ingolia NT, Weissman JS, Bartel DP. Mammalian microRNAs predominantly act to decrease target mRNA levels. Nature 2010;466:835–40. <a href="https://doi.org/10.1038/nature09267">https://doi.org/10.1038/nature09267.</a></span>
</li>
<li><span id="Behm-Ansmant2006a">Behm-Ansmant I, Rehwinkel J, Doerks T, Stark A, Bork P, Izaurralde E. mRNA degradation by miRNAs and GW182 requires both CCR4:NOT deadenylase and DCP1:DCP2 decapping complexes. Genes &amp; Development 2006;20:1885–98. <a href="https://doi.org/10.1101/gad.1424106">https://doi.org/10.1101/gad.1424106.</a></span>
</li>
<li><span id="Giraldez2006">Giraldez AJ, Mishima Y, Rihel J, Grocock RJ, Dongen SV, Inoue K, et al. Zebrafish MiR-430 promotes deadenylation and clearance of maternal mRNAs. Science 2006;312:75–9. <a href="https://doi.org/10.1126/science.1122689">https://doi.org/10.1126/science.1122689.</a></span>
</li>
<li><span id="Wu2006">Wu L, Fan J, Belasco JG. MicroRNAs direct rapid deadenylation of mRNA. Proceedings of the National Academy of Sciences 2006;103:4034–9. <a href="https://doi.org/10.1073/pnas.0510928103">https://doi.org/10.1073/pnas.0510928103.</a></span>
</li>
<li><span id="Jackson2003">Jackson AL, Bartz SR, Schelter J, Kobayashi SV, Burchard J, Mao M, et al. Expression profiling reveals off-target gene regulation by RNAi. Nature Biotechnology 2003;21:635–7. <a href="https://doi.org/10.1038/nbt831">https://doi.org/10.1038/nbt831.</a></span>
</li>
<li><span id="Friedman2009">Friedman RC, Farh KK-H, Burge CB, Bartel DP. Most mammalian mRNAs are conserved targets of microRNAs. Genome Research 2008;19:92–105. <a href="https://doi.org/10.1101/gr.082701.108">https://doi.org/10.1101/gr.082701.108.</a></span>
</li>
<li><span id="Stefani2008">Stefani G, Slack FJ. Small non-coding RNAs in animal development. Nature Reviews Molecular Cell Biology 2008;9:219–30. <a href="https://doi.org/10.1038/nrm2347">https://doi.org/10.1038/nrm2347.</a></span>
</li>
<li><span id="Alvarez-Garcia2005">Alvarez-Garcia I, Miska EA. MicroRNA functions in animal development and human disease. Development 2005;132:4653–62. <a href="https://doi.org/10.1242/dev.02073">https://doi.org/10.1242/dev.02073.</a></span>
</li>
<li><span id="He2005">He L, Thomson JM, Hemann MT, Hernando-Monge E, Mu D, Goodson S, et al. A microRNA polycistron as a potential human oncogene. Nature 2005;435:828–33. <a href="https://doi.org/10.1038/nature03552">https://doi.org/10.1038/nature03552.</a></span>
</li>
<li><span id="Lu2005">Lu J, Getz G, Miska EA, Alvarez-Saavedra E, Lamb J, Peck D, et al. MicroRNA expression profiles classify human cancers. Nature 2005;435:834–8. <a href="https://doi.org/10.1038/nature03702">https://doi.org/10.1038/nature03702.</a></span>
</li>
<li><span id="Chen2008">Chen J-F, Murchison EP, Tang R, Callis TE, Tatsuguchi M, Deng Z, et al. Targeted deletion of Dicer in the heart leads to dilated cardiomyopathy and heart failure. Proceedings of the National Academy of Sciences 2008;105:2111–6. <a href="https://doi.org/10.1073/pnas.0710228105">https://doi.org/10.1073/pnas.0710228105.</a></span>
</li>
<li><span id="Zhao2007">Zhao Y, Ransom JF, Li A, Vedantham V, Drehle M von, Muth AN, et al. Dysregulation of cardiogenesis, cardiac conduction, and cell cycle in mice lacking miRNA-1-2. Cell 2007;129:303–17. <a href="https://doi.org/10.1016/j.cell.2007.03.030">https://doi.org/10.1016/j.cell.2007.03.030.</a></span>
</li>
<li><span id="Maes2009">Maes O, Chertkow H, Wang E, Schipper H. MicroRNA: implications for alzheimer disease and other human CNS disorders. Current Genomics 2009;10:154–68. <a href="https://doi.org/10.2174/138920209788185252">https://doi.org/10.2174/138920209788185252.</a></span>
</li>
<li><span id="Bartel2009">Bartel DP. MicroRNAs: target recognition and regulatory functions. Cell 2009;136:215–33. <a href="https://doi.org/10.1016/j.cell.2009.01.002">https://doi.org/10.1016/j.cell.2009.01.002.</a></span>
</li>
<li><span id="Lewis2003">Lewis BP, Shih I-hung, Jones-Rhoades MW, Bartel DP, Burge CB. Prediction of mammalian MicroRNA targets. Cell 2003;115:787–98. <a href="https://doi.org/10.1016/s0092-8674(03)01018-3">https://doi.org/10.1016/s0092-8674(03)01018-3.</a></span>
</li>
<li><span id="Brennecke2005">Brennecke J, Stark A, Russell RB, Cohen SM. Principles of MicroRNA-target recognition. PLoS Biology 2005;3:e85. <a href="https://doi.org/10.1371/journal.pbio.0030085">https://doi.org/10.1371/journal.pbio.0030085.</a></span>
</li>
<li><span id="Lewis2005">Lewis BP, Burge CB, Bartel DP. Conserved seed pairing, often flanked by adenosines, indicates that thousands of human genes are MicroRNA targets. Cell 2005;120:15–20. <a href="https://doi.org/10.1016/j.cell.2004.12.035">https://doi.org/10.1016/j.cell.2004.12.035.</a></span>
</li>
<li><span id="Gaidatzis2007">Gaidatzis D, Nimwegen E van, Hausser J, Zavolan M. Inference of miRNA targets using evolutionary conservation and pathway analysis. BMC Bioinformatics 2007;8:69. <a href="https://doi.org/10.1186/1471-2105-8-69">https://doi.org/10.1186/1471-2105-8-69.</a></span>
</li>
<li><span id="Ellwanger2011">Ellwanger DC, Büttner FA, Mewes H-W, Stümpflen V. The sufficient minimal set of miRNA seed types. Bioinformatics 2011;27:1346–50. <a href="https://doi.org/10.1093/bioinformatics/btr149">https://doi.org/10.1093/bioinformatics/btr149.</a></span>
</li>
<li><span id="Grimson2007">Grimson A, Farh KK-H, Johnston WK, Garrett-Engele P, Lim LP, Bartel DP. MicroRNA targeting specificity in mammals: determinants beyond seed pairing. Molecular Cell 2007;27:91–105. <a href="https://doi.org/10.1016/j.molcel.2007.06.017">https://doi.org/10.1016/j.molcel.2007.06.017.</a></span>
</li>
<li><span id="Krek2005">Krek A, Grün D, Poy MN, Wolf R, Rosenberg L, Epstein EJ, et al. Combinatorial microRNA target predictions. Nature Genetics 2005;37:495–500. <a href="https://doi.org/10.1038/ng1536">https://doi.org/10.1038/ng1536.</a></span>
</li>
<li><span id="Enright2003">Enright AJ, John B, Gaul U, Tuschl T, Sander C, Marks DS. MicroRNA targets in Drosophila. Genome Biology 2003;5:R1. <a href="https://doi.org/10.1186/gb-2003-5-1-r1">https://doi.org/10.1186/gb-2003-5-1-r1.</a></span>
</li>
<li><span id="Lall2006">Lall S, Grün D, Krek A, Chen K, Wang Y-L, Dewey CN, et al. A genome-wide map of conserved MicroRNA targets in C. Elegans. Current Biology 2006;16:460–71. <a href="https://doi.org/10.1016/j.cub.2006.01.050">https://doi.org/10.1016/j.cub.2006.01.050.</a></span>
</li>
<li><span id="Betel2010">Betel D, Koppal A, Agius P, Sander C, Leslie C. Comprehensive modeling of microRNA targets predicts functional non-conserved and non-canonical sites. Genome Biology 2010;11:R90. <a href="https://doi.org/10.1186/gb-2010-11-8-r90">https://doi.org/10.1186/gb-2010-11-8-r90.</a></span>
</li>
<li><span id="Kertesz2007">Kertesz M, Iovino N, Unnerstall U, Gaul U, Segal E. The role of site accessibility in microRNA target recognition. Nature Genetics 2007;39:1278–84. <a href="https://doi.org/10.1038/ng2135">https://doi.org/10.1038/ng2135.</a></span>
</li>
<li><span id="Sethupathy2006">Sethupathy P, Megraw M, Hatzigeorgiou AG. A guide through present computational approaches for the identification of mammalian microRNA targets. Nature Methods 2006;3:881–6. <a href="https://doi.org/10.1038/nmeth954">https://doi.org/10.1038/nmeth954.</a></span>
</li>
<li><span id="Long2007">Long D, Lee R, Williams P, Chan CY, Ambros V, Ding Y. Potent effect of target structure on microRNA function. Nature Structural &amp; Molecular Biology 2007;14:287–94. <a href="https://doi.org/10.1038/nsmb1226">https://doi.org/10.1038/nsmb1226.</a></span>
</li>
<li><span id="John2004">John B, Enright AJ, Aravin A, Tuschl T, Sander C, Marks DS. Human MicroRNA targets. PLoS Biology 2004;2:e363. <a href="https://doi.org/10.1371/journal.pbio.0020363">https://doi.org/10.1371/journal.pbio.0020363.</a></span>
</li>
<li><span id="Saetrom2007">Saetrom P, Heale BSE, Snøve O, Aagaard L, Alluin J, Rossi JJ. Distance constraints between microRNA target sites dictate efficacy and cooperativity. Nucleic Acids Res 2007;35:2333–42. <a href="https://doi.org/10.1093/nar/gkm133">https://doi.org/10.1093/nar/gkm133.</a></span>
</li>
<li><span id="Hafner2010">Hafner M, Landthaler M, Burger L, Khorshid M, Hausser J, Berninger P, et al. Transcriptome-wide identification of RNA-binding protein and MicroRNA target sites by PAR-CLIP. Cell 2010;141:129–41. <a href="https://doi.org/10.1016/j.cell.2010.03.009">https://doi.org/10.1016/j.cell.2010.03.009.</a></span>
</li>
<li><span id="Lee2009">Lee I, Ajay SS, Yook JI, Kim HS, Hong SH, Kim NH, et al. New class of microRNA targets containing simultaneous 5’-UTR and 3’-UTR interaction sites. Genome Research 2009;19:1175–83. <a href="https://doi.org/10.1101/gr.089367.108">https://doi.org/10.1101/gr.089367.108.</a></span>
</li>
<li><span id="Kim2008">Kim DH, Saetrom P, Snove O, Rossi JJ. MicroRNA-directed transcriptional gene silencing in mammalian cells. Proceedings of the National Academy of Sciences 2008;105:16230–5. <a href="https://doi.org/10.1073/pnas.0808830105">https://doi.org/10.1073/pnas.0808830105.</a></span>
</li>
<li><span id="Place2008">Place RF, Li L-C, Pookot D, Noonan EJ, Dahiya R. MicroRNA-373 induces expression of genes with complementary promoter sequences. Proceedings of the National Academy of Sciences 2008;105:1608–13. <a href="https://doi.org/10.1073/pnas.0707594105">https://doi.org/10.1073/pnas.0707594105.</a></span>
</li>
<li><span id="Gu2009">Gu S, Jin L, Zhang F, Sarnow P, Kay MA. Biological basis for restriction of microRNA targets to the 3′ untranslated region in mammalian mRNAs. Nature Structural &amp; Molecular Biology 2009;16:144–50. <a href="https://doi.org/10.1038/nsmb.1552">https://doi.org/10.1038/nsmb.1552.</a></span>
</li>
<li><span id="Duursma2008">Duursma AM, Kedde M, Schrier M, Sage C le, Agami R. miR-148 targets human DNMT3b protein coding region. RNA 2008;14:872–7. <a href="https://doi.org/10.1261/rna.972008">https://doi.org/10.1261/rna.972008.</a></span>
</li>
<li><span id="Elcheva2009">Elcheva I, Goswami S, Noubissi FK, Spiegelman VS. CRD-BP protects the coding region of βTrCP1 mRNA from miR-183-mediated degradation. Molecular Cell 2009;35:240–6. <a href="https://doi.org/10.1016/j.molcel.2009.06.007">https://doi.org/10.1016/j.molcel.2009.06.007.</a></span>
</li>
<li><span id="Forman2008">Forman JJ, Legesse-Miller A, Coller HA. A search for conserved sequences in coding regions reveals that the let-7 microRNA targets Dicer within its coding sequence. Proceedings of the National Academy of Sciences 2008;105:14879–84. <a href="https://doi.org/10.1073/pnas.0803230105">https://doi.org/10.1073/pnas.0803230105.</a></span>
</li>
<li><span id="Tay2008">Tay Y, Zhang J, Thomson AM, Lim B, Rigoutsos I. MicroRNAs to Nanog, Oct4 and Sox2 coding regions modulate embryonic stem cell differentiation. Nature 2008;455:1124–8. <a href="https://doi.org/10.1038/nature07299">https://doi.org/10.1038/nature07299.</a></span>
</li>
<li><span id="Hwang2007">Hwang H-W, Wentzel EA, Mendell JT. A hexanucleotide element directs MicroRNA nuclear import. Science 2007;315:97–100. <a href="https://doi.org/10.1126/science.1136235">https://doi.org/10.1126/science.1136235.</a></span>
</li>
<li><span id="Politz2006">Politz JCR, Zhang F, Pederson T. MicroRNA-206 colocalizes with ribosome-rich regions in both the nucleolus and cytoplasm of rat myogenic cells. Proceedings of the National Academy of Sciences 2006;103:18957–62. <a href="https://doi.org/10.1073/pnas.0609466103">https://doi.org/10.1073/pnas.0609466103.</a></span>
</li>
<li><span id="Engstrom2006">Engström PG, Suzuki H, Ninomiya N, Akalin A, Sessa L, Lavorgna G, et al. Complex loci in human and mouse genomes. PLoS Genetics 2006;2:e47. <a href="https://doi.org/10.1371/journal.pgen.0020047">https://doi.org/10.1371/journal.pgen.0020047.</a></span>
</li>
<li><span id="Duhig1998">Duhig T, Ruhrberg C, Mor O, Fried M. The human surfeit locus. Genomics 1998;52:72–8. <a href="https://doi.org/10.1006/geno.1998.5372">https://doi.org/10.1006/geno.1998.5372.</a></span>
</li>
<li><span id="Holmes2003">Holmes R, Williamson C, Peters J, Denny P, Group RIKENGER, Members GSL, et al. A comprehensive transcript map of the mouse Gnas imprinted complex. Genome Research 2003;13:1410–5. <a href="https://doi.org/10.1101/gr.955503">https://doi.org/10.1101/gr.955503.</a></span>
</li>
<li><span id="Trinklein2004a">Trinklein ND, Aldred SF, Hartman SJ, Schroeder DI, Otillar RP, Myers RM. An abundance of bidirectional promoters in the human genome. Genome Research 2003;14:62–6. <a href="https://doi.org/10.1101/gr.1982804">https://doi.org/10.1101/gr.1982804.</a></span>
</li>
<li><span id="Wang2005">Wang X-J, Gaasterland T, Chua N-H. Genome-wide prediction and identification of cis-natural antisense transcripts in Arabidopsis thaliana. Genome Biology 2005;6:R30. <a href="https://doi.org/10.1186/gb-2005-6-4-r30">https://doi.org/10.1186/gb-2005-6-4-r30.</a></span>
</li>
<li><span id="Chen2004">Chen J, Sun M, Kent WJ, Huang X, Xie H, Wang W, et al. Over 20% of human transcripts might form sense-antisense pairs. Nucleic Acids Research 2004;32:4812–20. <a href="https://doi.org/10.1093/nar/gkh818">https://doi.org/10.1093/nar/gkh818.</a></span>
</li>
<li><span id="Zhang2006">Zhang Y, Liu XS, Liu Q-R, Wei L. Genome-wide in silico identification and analysis of cis natural antisense transcripts (cis-NATs) in ten species. Nucleic Acids Research 2006;34:3465–75. <a href="https://doi.org/10.1093/nar/gkl473">https://doi.org/10.1093/nar/gkl473.</a></span>
</li>
<li><span id="Osato2007">Osato N, Suzuki Y, Ikeo K, Gojobori T. Transcriptional interferences in cis natural antisense transcripts of humans and mice. Genetics 2007;176:1299–306. <a href="https://doi.org/10.1534/genetics.106.069484">https://doi.org/10.1534/genetics.106.069484.</a></span>
</li>
<li><span id="Borsani2005">Borsani O, Zhu J, Verslues PE, Sunkar R, Zhu J-K. Endogenous siRNAs derived from a pair of natural cis-antisense transcripts regulate salt tolerance in arabidopsis. Cell 2005;123:1279–91. <a href="https://doi.org/10.1016/j.cell.2005.11.035">https://doi.org/10.1016/j.cell.2005.11.035.</a></span>
</li>
<li><span id="Tufarelli2003">Tufarelli C, Stanley JAS, Garrick D, Sharpe JA, Ayyub H, Wood WG, et al. Transcription of antisense RNA leading to gene silencing and methylation as a novel cause of human genetic disease. Nature Genetics 2003;34:157–65. <a href="https://doi.org/10.1038/ng1157">https://doi.org/10.1038/ng1157.</a></span>
</li>
<li><span id="Beisel2011">Beisel C, Paro R. Silencing chromatin: comparing modes and mechanisms. Nature Reviews Genetics 2011;12:123–35. <a href="https://doi.org/10.1038/nrg2932">https://doi.org/10.1038/nrg2932.</a></span>
</li>
<li><span id="Mondal2010">Mondal T, Rasmussen M, Pandey GK, Isaksson A, Kanduri C. Characterization of the RNA content of chromatin. Genome Research 2010;20:899–907. <a href="https://doi.org/10.1101/gr.103473.109">https://doi.org/10.1101/gr.103473.109.</a></span>
</li>
<li><span id="Mercer2009">Mercer TR, Dinger ME, Mattick JS. Long non-coding RNAs: insights into functions. Nature Reviews Genetics 2009;10:155–9. <a href="https://doi.org/10.1038/nrg2521">https://doi.org/10.1038/nrg2521.</a></span>
</li>
<li><span id="Whitehead2009">Whitehead J, Pandey GK, Kanduri C. Regulation of the mammalian epigenome by long noncoding RNAs. Biochimica Et Biophysica Acta (BBA) - General Subjects 2009;1790:936–47. <a href="https://doi.org/10.1016/j.bbagen.2008.10.007">https://doi.org/10.1016/j.bbagen.2008.10.007.</a></span>
</li></ol>

        

        

        
          <hr>
          <h2>Leave a comment</h2>
          <div id="disqus_thread"></div>
        

        
        
          <hr>
          <footer>
            
              <p><a href="#top" id="back-to-top">Back to top</a></p>
            

            <p class="text-small text-grey-dk-100 mb-0">Copyright &copy; 2021 Takaya Saito.</p>

            
              <div class="d-flex mt-2">
                
                
              </div>
            
          </footer>
        

      </div>
    </div>

    
      

      <div class="search-overlay"></div>
    
  </div>

  
     <script>addBackToTop({
       id: 'back-to-top-js-button'
     })</script>
  

  
    <script>
        /**
        *  RECOMMENDED CONFIGURATION VARIABLES: EDIT AND UNCOMMENT THE SECTION BELOW TO INSERT DYNAMIC VALUES FROM YOUR PLATFORM OR CMS.
        *  LEARN WHY DEFINING THESE VARIABLES IS IMPORTANT: https://disqus.com/admin/universalcode/#configuration-variables    */
        /*
        var disqus_config = function () {
        this.page.url = PAGE_URL;  // Replace PAGE_URL with your page's canonical URL variable
        this.page.identifier = PAGE_IDENTIFIER; // Replace PAGE_IDENTIFIER with your page's unique identifier variable
        };
        */
        (function() { // DON'T EDIT BELOW THIS LINE
        var d = document, s = d.createElement('script');
        s.src = 'https://mitkb.disqus.com/embed.js';
        s.setAttribute('data-timestamp', +new Date());
        (d.head || d.body).appendChild(s);
        })();
    </script>
    <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
  

</body>
</html>

