

<!DOCTYPE html>

<html lang="en-US">
<head>
  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=Edge">

  

  <link rel="shortcut icon" href="/mitkb/favicon.ico" type="image/x-icon">

  <link rel="stylesheet" href="/mitkb/assets/css/just-the-docs-default.css">

  
    <script async src="https://www.googletagmanager.com/gtag/js?id=G-84ETYTKDBN"></script>
    <script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'G-84ETYTKDBN', { 'anonymize_ip': true });
    </script>

  

  
    <script type="text/javascript" src="/mitkb/assets/js/vendor/lunr.min.js"></script>
  
  <script type="text/javascript" src="/mitkb/assets/js/just-the-docs.js"></script>

  <meta name="viewport" content="width=device-width, initial-scale=1">

  <!-- Begin Jekyll SEO tag v2.7.1 -->
<title>Review: miRNA target | mitkb</title>
<meta name="generator" content="Jekyll v4.2.0" />
<meta property="og:title" content="Review: miRNA target" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="MicroRNAs: targeting and target prediction" />
<meta property="og:description" content="MicroRNAs: targeting and target prediction" />
<link rel="canonical" href="https://ncrnas.github.io/mitkb/mirna-target-prediction/micrornas-targeting-and-target-prediction/" />
<meta property="og:url" content="https://ncrnas.github.io/mitkb/mirna-target-prediction/micrornas-targeting-and-target-prediction/" />
<meta property="og:site_name" content="mitkb" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="Review: miRNA target" />
<script type="application/ld+json">
{"description":"MicroRNAs: targeting and target prediction","url":"https://ncrnas.github.io/mitkb/mirna-target-prediction/micrornas-targeting-and-target-prediction/","@type":"WebPage","headline":"Review: miRNA target","publisher":{"@type":"Organization","logo":{"@type":"ImageObject","url":"https://ncrnas.github.io/mitkb/assets/images/logo.png"}},"@context":"https://schema.org"}</script>
<!-- End Jekyll SEO tag -->


  
<meta name="google-site-verification" content="vstkqjcoJe9vQcOOpsCKrxv6WfqZ2SKRNWSv7XoHFeQ" />

<link rel="apple-touch-icon" sizes="180x180" href="/mitkb/assets/images/favicon/apple-touch-icon.png">
<link rel="icon" type="image/png" sizes="32x32" href="/mitkb/assets/images/favicon/favicon-32x32.png">
<link rel="icon" type="image/png" sizes="16x16" href="/mitkb/assets/images/favicon/favicon-16x16.png">
<link rel="manifest" href="/mitkb/assets/images/favicon/site.webmanifest">
<link rel="mask-icon" href="/mitkb/assets/images/favicon/safari-pinned-tab.svg" color="#5bbad5">
<meta name="msapplication-TileColor" content="#2b5797">
<meta name="theme-color" content="#ffffff">

<!-- for mathjax support -->



  <script src="https://unpkg.com/vanilla-back-to-top@7.2.1/dist/vanilla-back-to-top.min.js"></script>


<script
  data-cookie-notice='{ "learnMoreLinkEnabled": true, "learnMoreLinkHref": "/mitkb/terms/" }'
  src="https://unpkg.com/cookie-notice@^1/dist/cookie.notice.min.js"
></script>


</head>

<body>
  <svg xmlns="http://www.w3.org/2000/svg" style="display: none;">
    <symbol id="svg-link" viewBox="0 0 24 24">
      <title>Link</title>
      <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-link">
        <path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path>
      </svg>
    </symbol>
    <symbol id="svg-search" viewBox="0 0 24 24">
      <title>Search</title>
      <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-search">
        <circle cx="11" cy="11" r="8"></circle><line x1="21" y1="21" x2="16.65" y2="16.65"></line>
      </svg>
    </symbol>
    <symbol id="svg-menu" viewBox="0 0 24 24">
      <title>Menu</title>
      <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-menu">
        <line x1="3" y1="12" x2="21" y2="12"></line><line x1="3" y1="6" x2="21" y2="6"></line><line x1="3" y1="18" x2="21" y2="18"></line>
      </svg>
    </symbol>
    <symbol id="svg-arrow-right" viewBox="0 0 24 24">
      <title>Expand</title>
      <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-chevron-right">
        <polyline points="9 18 15 12 9 6"></polyline>
      </svg>
    </symbol>
    <symbol id="svg-doc" viewBox="0 0 24 24">
      <title>Document</title>
      <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-file">
        <path d="M13 2H6a2 2 0 0 0-2 2v16a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V9z"></path><polyline points="13 2 13 9 20 9"></polyline>
      </svg>
    </symbol>
  </svg>

  <div class="side-bar">
    <div class="site-header">
      <a href="https://ncrnas.github.io/mitkb/" class="site-title lh-tight">
  <div class="site-logo"></div>

</a>
      <a href="#" id="menu-button" class="site-button">
        <svg viewBox="0 0 24 24" class="icon"><use xlink:href="#svg-menu"></use></svg>
      </a>
    </div>
    <nav role="navigation" aria-label="Main" id="site-nav" class="site-nav">
      
        <ul class="nav-list"><li class="nav-list-item"><a href="https://ncrnas.github.io/mitkb/" class="nav-list-link">Home</a></li><li class="nav-list-item"><a href="#" class="nav-list-expander"><svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg></a><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/" class="nav-list-link">Overview</a><ul class="nav-list "><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/0-front-matter/" class="nav-list-link">0 Front matter</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/1-introduction/" class="nav-list-link">1 Introduction</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/2-papers-and-their-corresponding-sub-goals/" class="nav-list-link">2 Papers and sub-goals</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/3-micrornas-and-other-non-coding-rnas/" class="nav-list-link">3 miRNAs and ncRNAs</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/4-high-throughput-biological-experiments/" class="nav-list-link">4 Biological experiments</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/5-statistical-tests-and-methods/" class="nav-list-link">5 Statistical methods</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/6-machine-learning-theory-and-support-vector-machine/" class="nav-list-link">6 Support vector machine</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/7-computational-implementation/" class="nav-list-link">7 Implementation</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/8-future-perspectives/" class="nav-list-link">8 Future perspectives</a></li></ul></li><li class="nav-list-item active"><a href="#" class="nav-list-expander"><svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg></a><a href="https://ncrnas.github.io/mitkb/mirna-target-prediction/" class="nav-list-link">Target prediction</a><ul class="nav-list "><li class="nav-list-item  active"><a href="https://ncrnas.github.io/mitkb/mirna-target-prediction/micrornas-targeting-and-target-prediction/" class="nav-list-link active">Review: miRNA target</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/mirna-target-prediction/two-step-svm-model/" class="nav-list-link">Research: Two-step SVM</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/mirna-target-prediction/two-step-svm-model-supplementary-information/" class="nav-list-link">Supplementary information</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander"><svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg></a><a href="https://ncrnas.github.io/mitkb/mirna-high-throughput-experiments/" class="nav-list-link">Biological experiments</a><ul class="nav-list "><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/mirna-high-throughput-experiments/confounding-factors-in-mirna-experiments/" class="nav-list-link">Research: Confounding</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/mirna-high-throughput-experiments/confounding-factors-in-mirna-experiments-supplementary-information/" class="nav-list-link">Supplementary information</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander"><svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg></a><a href="https://ncrnas.github.io/mitkb/mirna-and-other-ncrnas/" class="nav-list-link">Other ncRNAs</a><ul class="nav-list "></ul></li><li class="nav-list-item"><a href="https://ncrnas.github.io/mitkb/support-articles/" class="nav-list-link">Support articles</a></li><li class="nav-list-item"><a href="https://ncrnas.github.io/mitkb/contact" class="nav-list-link">Contact us</a></li></ul>

      
    </nav>
    <footer class="site-footer">
      <a href="https://ncrnas.github.io/mitkb/terms">Terms and Privacy Policy</a> <br /><br />
      This site uses <a href="https://github.com/pmarsceill/just-the-docs">Just the Docs</a>, a documentation theme for Jekyll.
    </footer>
  </div>
  <div class="main" id="top">
    <div id="main-header" class="main-header">
      
        <div class="search">
          <div class="search-input-wrap">
            <input type="text" id="search-input" class="search-input" tabindex="0" placeholder="Search mitkb" aria-label="Search mitkb" autocomplete="off">
            <label for="search-input" class="search-label"><svg viewBox="0 0 24 24" class="search-icon"><use xlink:href="#svg-search"></use></svg></label>
          </div>
          <div id="search-results" class="search-results"></div>
        </div>
      
      
      
        <nav aria-label="Auxiliary" class="aux-nav">
          <ul class="aux-nav-list">
            
              <li class="aux-nav-list-item">
                <a href="https://github.com/ncrnas/mitkb" class="site-button"
                  
                >
                  mitkb on github
                </a>
              </li>
            
          </ul>
        </nav>
      
    </div>
    <div id="main-content-wrap" class="main-content-wrap">
      
        <nav aria-label="Breadcrumb" class="breadcrumb-nav">
            <ol class="breadcrumb-nav-list">
              
                <li class="breadcrumb-nav-list-item"><a href="https://ncrnas.github.io/mitkb/mirna-target-prediction/">Target prediction</a></li>
              
              <li class="breadcrumb-nav-list-item"><span>Review: miRNA target</span></li>
            </ol>
          </nav>
        
      
      <div id="main-content" class="main-content" role="main">
        
          <h1 class="no_toc text-center" id="micrornas---targeting-and-target-prediction">
        
        
          <a href="#micrornas---targeting-and-target-prediction" class="anchor-heading" aria-labelledby="micrornas---targeting-and-target-prediction"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> MicroRNAs - targeting and target prediction
        
        
      </h1>
    

<p class="text-center">Takay Saito<sup>a</sup> and Pål Sætrom<sup>a,b</sup></p>

<p class="fs-3 text-center text-grey-dk-000"><strong><sup>a</sup></strong>Department of Cancer Research and Molecular Medicine,  <br />
Norwegian University of Science and Technology, NO-7489 Trondheim, Norway <br />
<strong><sup>b</sup></strong>Department of Computer and Information Science, <br />
Norwegian University of Science and Technology, NO-7491 Trondheim, Norway</p>

<p><br />
<a href="https://arxiv.org/pdf/1507.01804.pdf" class="btn">View PDF</a></p>
<div class="code-example bg-grey-lt-000 p-3">
<strong>Note:</strong> The web version of the article was created
based on the preprint version uploaded to
<a href="https://arxiv.org/abs/1507.01804">arXiv</a>. <br />
Preprint submitted to New Biotechnology: November 2009, <br />
Published: July 2010,
DOI: <a href="https://doi.org/10.1016/j.nbt.2010.02.016">https://doi.org/10.1016/j.nbt.2010.02.016</a>.
</div>

<div class="code-example bg-grey-lt-000 p-3">
<strong>Citation:</strong> <br />
Saito, T and Sætrom, P. (2010). MicroRNAs–targeting and target prediction.
New biotechnology 27 (3), 243-249.
<a href="https://doi.org/10.1016/j.nbt.2010.02.016">https://doi.org/10.1016/j.nbt.2010.02.016</a>.
</div>

<p><br /></p>

<details open="">
  <summary class="text-delta">
    Table of contents
  </summary>
<ol id="markdown-toc">
  <li><a href="#S:1" id="markdown-toc-S:1">Abstract</a></li>
  <li><a href="#S:2" id="markdown-toc-S:2">Introduction</a></li>
  <li><a href="#S:3" id="markdown-toc-S:3">miRNA target features</a>    <ol>
      <li><a href="#S:3a" id="markdown-toc-S:3a">miRNA:mRNA pairing: ‘Seed site’ is the most important feature for target recognition</a></li>
      <li><a href="#S:3b" id="markdown-toc-S:3b">Site location: most target sites reside within 3’ untranslated region (UTR) of target genes</a></li>
      <li><a href="#S:3c" id="markdown-toc-S:3c">Conservation: miRNAs and their targets are conserved among related species</a></li>
      <li><a href="#S:3d" id="markdown-toc-S:3d">Site accessibility: mRNA secondary structure affects site accessibility</a></li>
      <li><a href="#S:3e" id="markdown-toc-S:3e">Multiple sites: cooperativity enhances site efficacy</a></li>
      <li><a href="#S:3f" id="markdown-toc-S:3f">Expression profile: miRNA:mRNA pairs are negatively correlated in expression profiles</a></li>
    </ol>
  </li>
  <li><a href="#S:4" id="markdown-toc-S:4">Target prediction tools</a></li>
  <li><a href="#S:5" id="markdown-toc-S:5">Summary</a></li>
  <li><a href="#S:6" id="markdown-toc-S:6">References</a></li>
</ol>

</details>
      <h2 id="S:1">
        
        
          <a href="#S:1" class="anchor-heading" aria-labelledby="S:1"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> Abstract
        
        
      </h2>
    
<p>MicroRNAs (miRNAs) are a class of small noncoding RNAs that can
regulate many genes by base pairing to sites in mRNAs. The
functionality of miRNAs overlaps that of short interfering RNAs
(siRNAs), and many features of miRNA targeting have been revealed
experimentally by studying miRNA-mimicking siRNAs. This review
outlines the features associated with animal miRNA targeting and
describes currently available prediction tools.</p>
      <h2 id="S:2">
        
        
          <a href="#S:2" class="anchor-heading" aria-labelledby="S:2"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> Introduction
        
        
      </h2>
    
<p>MicroRNAs (miRNAs) were identified as a large sub-class of ncRNAs in 2001.
Since then, an increasing number of studies have firmly
established miRNAs’ importance in gene regulation in general and animal
development and disease in particular
<a class="citation" href="#Ambros2004">[1,2,3,4,5]</a>.
miRNAs regulate protein-coding genes post-transcriptionally by guiding a
protein complex known as the RNA-induced silencing complex (RISC) to
messenger RNAs (mRNAs) with partial complementarity to the miRNA
<a class="citation" href="#Carthew2009">[6]</a>. Through mechanisms not completely understood, RISC then
inhibits protein translation and causes mRNA degradation
<a class="citation" href="#Bartel2009">[7,8]</a>. Current estimates indicate that miRNAs
regulate at least 60% of the human protein-coding genes through this
post-transcriptional gene silencing (PTGS) <a class="citation" href="#Friedman2009">[9]</a>.</p>

<p>Incorporated into RISC, miRNAs are functionally equivalent to short
interfering RNAs (siRNAs) <a class="citation" href="#Hamilton1999">[10,11]</a>. The main
difference between these RNAs is that miRNAs are processed from
imperfect hairpin structures, whereas siRNAs are processed from long
double-stranded RNAs <a class="citation" href="#Kim2009">[12,13]</a>. Moreover, animal miRNAs
typically target imperfect sites, whereas siRNAs target sites with
near-perfect complementarity. SiRNAs do target imperfect sites as well,
however, and this miRNA-like targeting is the major source of siRNA
off-target effects <a class="citation" href="#Birmingham2006">[14,15,16]</a>.</p>

<p>The list of known miRNAs is large and increasing. Currently, the
official miRNA database miRBase lists 721 human miRNAs
(<a href="http://www.mirbase.org">http://www.mirbase.org</a>; Release 14) <a class="citation" href="#Griffiths-Jones2008">[17]</a>, but
estimates indicate that the human genome contains more than 1000 miRNAs.
As only a few regulatory targets are known, predicting and validating
miRNA targets is one of the major hurdles in understanding miRNA
biology. Here, we review the <a href="#S:3">features important for miRNA targeting</a> and
the <a href="#S:4">bioinformatics tools</a> available for predicting miRNA targets.</p>
      <h2 id="S:3">
        
        
          <a href="#S:3" class="anchor-heading" aria-labelledby="S:3"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> miRNA target features
        
        
      </h2>
    
<p>Identifying miRNA targets in animals has been very challenging. This is
mainly because the limited complementarity between miRNAs and their
targets, which might lead to the finding of hundreds of potential miRNA
targets per miRNA. Therefore, many studies have been conducted, both
experimentally and computationally, to reveal more efficient approaches
for miRNA target recognition. We have divided the miRNA target features
reported in these studies into six categories, <a href="#S:3a">miRNA:mRNA pairing</a>,
<a href="#S:3b">site location</a>, <a href="#S:3c">conservation</a>, <a href="#S:3d">site accessibility</a>,
<a href="#S:3e">multiple sites</a> and <a href="#S:3f">expression profile</a>.</p>
      <h3 id="S:3a">
        
        
          <a href="#S:3a" class="anchor-heading" aria-labelledby="S:3a"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> miRNA:mRNA pairing: ‘Seed site’ is the most important feature for target recognition
        
        
      </h3>
    
<p>miRNA targets commonly have at least one region that has Watson-Crick
pairing to the 5’ part of miRNA. This 5’ part, located at positions
2-7 from the 5’ end of miRNA, is known as the ‘seed’, as RISC uses
these positions as a nucleation signal for recognizing target mRNAs
<a class="citation" href="#Stark2003">[18,19,20]</a>. The corresponding sites in mRNA
are referred to as ‘seed sites’. A stringent-seed site has perfect
Watson-Crick pairing and can be divided into four ‘seed’ types – 8mer,
7mer-m8, 7mer-A1 and 6mer – depending on the combination of the
nucleotide of position 1 and pairing at position 8 (Fig. <a href="#F:1">1</a>a). 8mer has
both an adenine at position 1 of the target site and base pairing at
position 8. 7mer-A1 has an adenine at position 1, while 7mer-m8 has base
pairing at position 8. 6mer has neither an adenine at position 1 nor
base pairing at position 8 <a class="citation" href="#Grimson2007">[21]</a>. An adenine on the target site
corresponding to position 1 of miRNA is known to increase efficiency of
target recognition <a class="citation" href="#Lewis2005">[22]</a>.</p>
      <h4 id="F:1" class="no_toc">
        
        
          <a href="#F:1" class="anchor-heading" aria-labelledby="F:1"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a>  
        
        
      </h4>
    
<p><img src="/mitkb/assets/images/review/microrna-target-features.webp" alt="Figure 1" width="500" /></p>
      <h5 class="no_toc text-caption" id="figure-1-types-of-mirna-target-sites-and-multiple-sites">
        
        
          <a href="#figure-1-types-of-mirna-target-sites-and-multiple-sites" class="anchor-heading" aria-labelledby="figure-1-types-of-mirna-target-sites-and-multiple-sites"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> <strong>Figure 1</strong>. Types of miRNA target sites and multiple sites.
        
        
      </h5>
    
<p class="fs-2"><strong>(a)</strong> Stringent-seed site. 7mer-A1 is shown as an example.
Vertical lines indicate Watson-Crick paring. <br />
<strong>(b)</strong> Moderate-stringent-seed site. BM is shown as an example. <br />
<strong>(c)</strong> 3’-supplementary site. More than 3-4 nucleotide paring required. <br />
<strong>(d)</strong> Optimal distance of two miRNA target sites.</p><hr />

<p>In addition to this stringent-seed matching, moderate-stringent-seed
matching is also functional because RISC can tolerate small mismatches
or G:U wobble pairing within the seed region (Fig. <a href="#F:1">1</a>b). This
moderate-stringent-seed matching has five ‘seed’ types: GUM, GUT, BM, BT
and LP, defined regarding to the mismatch type. GUM has one G:U wobble
and the uracil on the seed site of miRNA, whereas GUT has the uracil on
the target site of mRNA. BM has one bulge and the mismatch is on the
seed site, whereas BT has the mismatch on the target site. LP has only
one loop <a class="citation" href="#Gaidatzis2007">[23]</a>. Furthermore, RISC can recognize offset sites
that are located at positions 3-10. Offset sites can be either stringent
or moderate-stringent-seed matching <a class="citation" href="#Hammell2008">[24]</a>.</p>

<p>Watson-Crick pairing in the 3’ part of miRNA is known to enhance the
site recognition efficacy in miRNA targets that have seed pairing
<a class="citation" href="#Grimson2007">[21]</a>. The preferable nucleotide number of matches in the 3’
part differs between the site that has stringent-seed pairing and the
one that has moderate-stringent-seed pairing. Stringent-seeds require
3-4 matches in the positions 13-16, whereas moderate-stringent-seeds
require 4-5 matches in the positions 13-19. Sites with this additional
3’ pairing are called 3’-supplementary (Fig. <a href="#F:1">1</a>c) and
3’-compensatory sites <a class="citation" href="#Bartel2009">[7]</a>.</p>

<p>It is difficult to measure the efficacy level of each seed type
precisely, but several microarray and conservation enrichment studies
have revealed hierarchies of relative efficacies. These hierarchies can
be described as ‘Stringent seed’ &gt; ‘Stringent seed in offset’ &gt;
‘Moderate-stringent seed’ &gt; ‘Moderate-stringent in offset’; 8mer &gt;
7mer-m8 &gt; 7mer-A1 &gt; 6mer in the stringent-seed types; and Bulge &gt;
G:U wobble &gt; Loop in the moderate-stringent-seed types
<a class="citation" href="#Bartel2009">[7,24]</a>. Moreover, multiple sites are more efficient
than single sites <a class="citation" href="#Saetrom2007">[25]</a>.</p>

<p>The advantages and disadvantages of using different set of seed types
are that considering only stringent-seed types increases specificity but
might miss many potential targets, whereas considering both stringent
and moderate-stringent-seed types increase sensitivity but might also
increase the number of false positives.</p>
      <h3 id="S:3b">
        
        
          <a href="#S:3b" class="anchor-heading" aria-labelledby="S:3b"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> Site location: most target sites reside within 3’ untranslated region (UTR) of target genes
        
        
      </h3>
    
<p>Several studies have reported that most target sites can be found in the
3’ UTR segment of the target genes, even though miRNA-loaded RISC in
theory can bind any segment of mRNA. Target genes tend to have longer
3’ UTR, whereas ubiquitously expressed genes, such as house-keeping
genes, have shorter 3’ UTR – potentially to avoid being regulated by
miRNAs <a class="citation" href="#Stark2005">[26]</a>. Target sites are not evenly distributed within 3’
UTR, but are located near both ends when the length of 3’ UTR is
≥2000 nucleotides. For shorter 3’ UTRs, sites
tend to be near the stop codon <a class="citation" href="#Gaidatzis2007">[23]</a>. Sites are not located
too close to the stop codon, however, but 15-20 nucleotides away from
the stop codon <a class="citation" href="#Grimson2007">[21]</a>. In addition, some genes have alternative
splicing in their 3’ UTR segments, especially genes with long 3’
UTRs. These genes might therefore have different potential target sites
for alternatively spliced 3’ UTRs <a class="citation" href="#Majoros2007">[27]</a>. Finally, alternative
polyadenylation sites can shorten 3’ UTRs and affect miRNA regulation
<a class="citation" href="#Sandberg2008">[28]</a>.</p>

<p>Although functional miRNA sites are preferentially located in 3’ UTR,
seed sites in the coding sequence (CDS) and 5’ UTR regions can also
give downregulation <a class="citation" href="#Lytle2007">[29,30]</a>. Why does RISC then
appear to prefer the 3’ UTR? The most probable explanation is that
RISC competes with other protein complexes, such as ribosomes in CDS and
translation initiation complexes in 5’ UTR; see discussion in the
following section
‘<a href="#S:3e">Multiple sites: cooperativity enhances site efficacy</a>’.
The 3’ UTR might simply be more accessible for long-term
binding than the two other mRNA regions <a class="citation" href="#Bartel2004">[5]</a>.</p>

<p>Despite this general trend for 3’ UTR targeting, there are some
notable exceptions. One recent study reported that many miRNAs
preferentially target 5’ UTR sites with high complementarity to the
miRNAs’ 3’ end in a species-specific manner <a class="citation" href="#Lee2009">[31]</a>. The targets
also showed signs of weaker interactions between the miRNA seed sequence
and the 3’ UTR. The authors proposed that these sites represented a
new miRNA target class called ‘miBridge’, in which one miRNA
simultaneously interacts with a seed pairing site in 3’ UTR and a 3’
pairing site in 5’ UTR. The molecular mechanisms behind and the
biological extent of these miBridge targets are still unknown, however.</p>

<p>Most miRNA target prediction studies only focus on the 3’ UTR, which
results in that all the available data are biased toward 3’ UTR.
Moreover, few studies consider alternative splicing or polyadenylation
because of shortcomings in current annotations. As transcript usage
often depends on cellular context – for example, whether the cell is
proliferating or terminally differentiated – future tools for miRNA
target analyses should probably use available information about cellular
state to increase prediction performance.</p>
      <h3 id="S:3c">
        
        
          <a href="#S:3c" class="anchor-heading" aria-labelledby="S:3c"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> Conservation: miRNAs and their targets are conserved among related species
        
        
      </h3>
    
<p>miRNA families are comprised of miRNAs that have the same seed site, and
are well conserved among related species. In addition, miRNA families
have targets that are conserved among related species <a class="citation" href="#Friedman2009">[9]</a>.
There are also species-specific miRNAs and targets, and one study showed
that about 30% of the experimentally validated target genes might not be
well conserved <a class="citation" href="#Sethupathy2006">[32]</a>.</p>

<p>siRNA off-target effects occur no matter whether the site is conserved
or not <a class="citation" href="#Burchard2009">[33]</a>, therefore searching for all potential target
sequences without considering evolutionally conservational might
increase siRNA off-target detection efficacy.</p>

<p>Applying a filter that requires predicted target sites to be conserved
can decrease the false-positive rate, but such a filter is effective
only for conserved miRNAs. It is important to identify targets both with
and without conservation – especially when species-specific miRNAs or
siRNA off-targets is of interest.</p>
      <h3 id="S:3d">
        
        
          <a href="#S:3d" class="anchor-heading" aria-labelledby="S:3d"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> Site accessibility: mRNA secondary structure affects site accessibility
        
        
      </h3>
    
<p>The mRNA secondary structure is very important for miRNA targeting. An
effective miRNA:mRNA interaction needs an open structure on the target
site to begin the hybridization reaction. After binding, RISC can
disrupt the secondary structure on the site to elongate hybridization
<a class="citation" href="#Long2007">[34,35]</a>. Minimum free energy is usually used to
estimate the secondary structure and RNA hybridization, but the amount
of A:Us surrounding the site can also be used to estimate the site
accessibility. Effective target sites often have A:U rich context in
approximately 30 nucleotides upstream and downstream from the seed
matching region of the target site <a class="citation" href="#Grimson2007">[21]</a>.</p>

<p>Calculating the minimum free energy of accessibility and hybridization
with the mRNA secondary structure requires analyzing different mRNA
folding patterns. This requires enormous amounts of computing power, as
finding the most stable RNA structure is a computational problem that
scales with the cube of the length of the RNA sequence <a class="citation" href="#Eddy2004">[36]</a>.
Hence, finding hybridization sites in long 3’ UTRs tends to be time
consuming. Moreover, the current thermodynamic models used in RNA
secondary structure prediction algorithms are only 90-95% accurate,
which results in that the algorithms tend to have only 50-70% of the
base pairs correct <a class="citation" href="#Eddy2004">[36]</a>. Thus, despite being theoretically sound,
calculating site accessibility has limited practical value when
predicting miRNA target sites; heuristics that are easy to compute, such
as local A:U context, have similar performance.</p>
      <h3 id="S:3e">
        
        
          <a href="#S:3e" class="anchor-heading" aria-labelledby="S:3e"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> Multiple sites: cooperativity enhances site efficacy
        
        
      </h3>
    
<p>Strong miRNA targets tend to have multiple target sites instead of one
single site <a class="citation" href="#John2004">[37]</a>. Considering the number of putative miRNA sites
per mRNA can therefore significantly enhance target prediction.</p>

<p>Although the general effect of multiple sites appears to be additive,
miRNA targeting can also be synergistic. Our previous study showed that
two target sites within optimal distance enhance target site efficacy.
The preferable optimal length is between 17 and 35 nucleotides, but the
length between 14 and 46 nucleotides also enhances the efficacy (Fig. <a href="#F:1">1</a>d).
This co-operability is functional between the same miRNAs as well as two
different miRNAs <a class="citation" href="#Saetrom2007">[25]</a>. Multiple sites involving more than two
sites can also contribute to the enhancement <a class="citation" href="#Doench2004">[38]</a>.</p>

<p>The exact mechanism underlying the synergism remains unknown. As
translational suppression is a relatively slow process compared with
RISC’s catalytic cleavage <a class="citation" href="#Haley2004">[39]</a>, however, multiple RISC complexes
bound at closely spaced target sites might cooperatively stabilize each
other at the sites or possibly accelerate the regulatory process. This
could explain why miRNAs prefer targets in 3’ UTRs, as ribosomes would
displace RISC from sites in CDS before RISC could effect translational
suppression. Indeed, a cluster of rare codons that stall the ribosome
can, when placed in front of a nonfunctional miRNA site in CDS, change
the site to a functional site <a class="citation" href="#Gu2009">[40]</a>. Moreover, the genes that
currently have verified miRNA target sites in CDS tend to have either
one very strong target site <a class="citation" href="#Duursma2008">[41,42]</a> or multiple,
closely spaced sites <a class="citation" href="#Tay2008">[43,44]</a>.</p>
      <h3 id="S:3f">
        
        
          <a href="#S:3f" class="anchor-heading" aria-labelledby="S:3f"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> Expression profile: miRNA:mRNA pairs are negatively correlated in expression profiles
        
        
      </h3>
    
<p>One miRNA can potentially regulate many genes; therefore, expression
profiles of mRNAs might vary substantially depending on the miRNA
expression levels. Many miRNAs are also expressed differently in
different tissues. Consequently, if negatively correlated expression
levels of a miRNA:mRNA pair are detected across different tissue
profiles, the mRNA of the pair is probably targeted by the miRNA
<a class="citation" href="#Rajewsky2006">[45,46]</a>. Filtering putative targets based on
expression profile correlations is an effective approach to reduce the
false-positive rate. Although the majority of miRNA targets appear to be
regulated both at the mRNA and protein level, some targets only show an
effect at the protein level, however <a class="citation" href="#Selbach2008">[47,48]</a>.
Researchers should therefore be aware that such filtering will exclude
potential targets.</p>
      <h2 id="S:4">
        
        
          <a href="#S:4" class="anchor-heading" aria-labelledby="S:4"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> Target prediction tools
        
        
      </h2>
    
<p>Many target prediction tools have been developed (Table <a href="#T:1">1</a>),
but the types of methods applied, the miRNA
and mRNA sequences used and the output prediction data and performance
evaluation vary widely between tools. Direct comparison of prediction
performance among tools is not straight forward because the set of
predicted target genes from different tools do not overlap well. What is
clear, however, is that conventional tools with simple stringent-seed
search are prone to high false-positive rates. Therefore, most tools are
designed to reduce the false-positive rate and maximize the accuracy at
the same time. We have compared the currently available tools based on
the target features the tools use in their predictions (Table <a href="#T:1">1</a>),
and the tools’ availability (Table <a href="#T:2">2</a>).
Availability is especially important for
researchers that are using their own miRNA or mRNA annotations, or are
working in a nonstandard species. In these cases, only tools that can be
downloaded or allows the user to input own miRNAs and mRNAs can be used.</p>

<p><br /></p>
      <h4 id="T:1" class="no_toc text-caption">
        
        
          <a href="#T:1" class="anchor-heading" aria-labelledby="T:1"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> <strong>Table 1</strong>. List of miRNA target prediction tools.
        
        
      </h4>
    

<div class="table-wrapper"><table>
  
    
      
      <thead> <tr>
        
          
            
            <th style="text-align: left">
          
          
            Tool
          
            </th>
        
          
            
            <th style="text-align: center">
          
          
            Pair<sup>a</sup>
          
            </th>
        
          
            
            <th style="text-align: center">
          
          
            Site<sup>b</sup>
          
            </th>
        
          
            
            <th style="text-align: center">
          
          
            Consv<sup>c</sup>
          
            </th>
        
          
            
            <th style="text-align: center">
          
          
            Access<sup>d</sup>
          
            </th>
        
          
            
            <th style="text-align: center">
          
          
            Multi<sup>e</sup>
          
            </th>
        
          
            
            <th style="text-align: center">
          
          
            Expr<sup>f</sup>
          
            </th>
        
          
            
            <th style="text-align: left">
          
          
            Refs
          
            </th>
        
      </tr> </thead>
      <tbody>
      
        
    
    <tr>
    
      
      
        <td style="text-align: left">
      
      
        TargetScan
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ○
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ○
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ○
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: left">
      
      
        
        
          <a class="citation" href="#Friedman2009">[9]</a>
        
          <a class="citation" href="#Grimson2007">[21]</a>
        
          <a class="citation" href="#Lewis2005">[22]</a>
        
      
        </td>
    
    </tr>
  
    
    <tr>
    
      
      
        <td style="text-align: left">
      
      
        PicTar
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ○
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: left">
      
      
        
        
          <a class="citation" href="#Chen2006">[49]</a>
        
          <a class="citation" href="#Grun2005">[50]</a>
        
          <a class="citation" href="#Krek2005">[51]</a>
        
          <a class="citation" href="#Lall2006">[52]</a>
        
      
        </td>
    
    </tr>
  
    
    <tr>
    
      
      
        <td style="text-align: left">
      
      
        miRanda
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ○
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ○
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: left">
      
      
        
        
          <a class="citation" href="#John2004">[37]</a>
        
          <a class="citation" href="#Betel2008">[53]</a>
        
      
        </td>
    
    </tr>
  
    
    <tr>
    
      
      
        <td style="text-align: left">
      
      
        MicroCosm Targets
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ○
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ○
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: left">
      
      
        
        
          <a class="citation" href="#Griffiths-Jones2008">[17]</a>
        
          <a class="citation" href="#Enright2003">[54]</a>
        
          <a class="citation" href="#Griffiths-Jones2006a">[55]</a>
        
      
        </td>
    
    </tr>
  
    
    <tr>
    
      
      
        <td style="text-align: left">
      
      
        RNAhybrid
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: left">
      
      
        
        
          <a class="citation" href="#Kruger2006">[56]</a>
        
          <a class="citation" href="#Rehmsmeier2004">[57]</a>
        
      
        </td>
    
    </tr>
  
    
    <tr>
    
      
      
        <td style="text-align: left">
      
      
        PITA
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ○
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: left">
      
      
        
        
          <a class="citation" href="#Kertesz2007">[35]</a>
        
      
        </td>
    
    </tr>
  
    
    <tr>
    
      
      
        <td style="text-align: left">
      
      
        STarMir
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: left">
      
      
        
        
          <a class="citation" href="#Long2007">[34]</a>
        
      
        </td>
    
    </tr>
  
    
    <tr>
    
      
      
        <td style="text-align: left">
      
      
        Rajewsky &amp; Socci
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: left">
      
      
        
        
          <a class="citation" href="#Rajewsky2004">[19]</a>
        
      
        </td>
    
    </tr>
  
    
    <tr>
    
      
      
        <td style="text-align: left">
      
      
        Robins
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ○
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: left">
      
      
        
        
          <a class="citation" href="#Robins2005">[58]</a>
        
      
        </td>
    
    </tr>
  
    
    <tr>
    
      
      
        <td style="text-align: left">
      
      
        mirWIP
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ○
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ○
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: left">
      
      
        
        
          <a class="citation" href="#Hammell2008">[24]</a>
        
      
        </td>
    
    </tr>
  
    
    <tr>
    
      
      
        <td style="text-align: left">
      
      
        MicroInspector
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: left">
      
      
        
        
          <a class="citation" href="#Rusinov2005">[59]</a>
        
      
        </td>
    
    </tr>
  
    
    <tr>
    
      
      
        <td style="text-align: left">
      
      
        MicroTar
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: left">
      
      
        
        
          <a class="citation" href="#Thadani2006">[60]</a>
        
      
        </td>
    
    </tr>
  
    
    <tr>
    
      
      
        <td style="text-align: left">
      
      
        MirTarget2
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ○
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: left">
      
      
        
        
          <a class="citation" href="#Wang2008">[61]</a>
        
      
        </td>
    
    </tr>
  
    
    <tr>
    
      
      
        <td style="text-align: left">
      
      
        miTarget
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: left">
      
      
        
        
          <a class="citation" href="#Kim2006">[62]</a>
        
      
        </td>
    
    </tr>
  
    
    <tr>
    
      
      
        <td style="text-align: left">
      
      
        TargetMiner
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ○
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: left">
      
      
        
        
          <a class="citation" href="#Bandyopadhyay2009">[63]</a>
        
      
        </td>
    
    </tr>
  
    
    <tr>
    
      
      
        <td style="text-align: left">
      
      
        EIMMo
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ○
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ○
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: left">
      
      
        
        
          <a class="citation" href="#Gaidatzis2007">[23]</a>
        
      
        </td>
    
    </tr>
  
    
    <tr>
    
      
      
        <td style="text-align: left">
      
      
        NbmiRTar
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ○
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: left">
      
      
        
        
          <a class="citation" href="#Yousef2007">[64]</a>
        
      
        </td>
    
    </tr>
  
    
    <tr>
    
      
      
        <td style="text-align: left">
      
      
        TargetBoost
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: left">
      
      
        
        
          <a class="citation" href="#Saetrom2005">[65]</a>
        
      
        </td>
    
    </tr>
  
    
    <tr>
    
      
      
        <td style="text-align: left">
      
      
        RNA22
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ○
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: left">
      
      
        
        
          <a class="citation" href="#Miranda2006">[66]</a>
        
      
        </td>
    
    </tr>
  
    
    <tr>
    
      
      
        <td style="text-align: left">
      
      
        TargetRank
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ○
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ○
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: left">
      
      
        
        
          <a class="citation" href="#Nielsen2007">[67]</a>
        
      
        </td>
    
    </tr>
  
    
    <tr>
    
      
      
        <td style="text-align: left">
      
      
        EMBL
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ○
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ○
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: left">
      
      
        
        
          <a class="citation" href="#Stark2003">[18]</a>
        
          <a class="citation" href="#Stark2005">[26]</a>
        
          <a class="citation" href="#Brennecke2005">[68]</a>
        
      
        </td>
    
    </tr>
  
    
    <tr>
    
      
      
        <td style="text-align: left">
      
      
        MovingTarget
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ○
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ○
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: left">
      
      
        
        
          <a class="citation" href="#Burgler2005">[69]</a>
        
      
        </td>
    
    </tr>
  
    
    <tr>
    
      
      
        <td style="text-align: left">
      
      
        DIANA-microT
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ○
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: left">
      
      
        
        
          <a class="citation" href="#Kiriakidou2004">[70]</a>
        
      
        </td>
    
    </tr>
  
    
    <tr>
    
      
      
        <td style="text-align: left">
      
      
        HOCTAR
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ○
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: left">
      
      
        
        
          <a class="citation" href="#Gennarino2009">[71]</a>
        
      
        </td>
    
    </tr>
  
    
    <tr>
    
      
      
        <td style="text-align: left">
      
      
        Stanhope
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: left">
      
      
        
        
          <a class="citation" href="#Stanhope2009">[72]</a>
        
      
        </td>
    
    </tr>
  
    
    <tr>
    
      
      
        <td style="text-align: left">
      
      
        GenMiR++
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ○
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ○
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: left">
      
      
        
        
          <a class="citation" href="#Huang2007">[73]</a>
        
      
        </td>
    
    </tr>
  
    
    <tr>
    
      
      
        <td style="text-align: left">
      
      
        HuMiTar
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: left">
      
      
        
        
          <a class="citation" href="#Ruan2008">[74]</a>
        
      
        </td>
    
    </tr>
  
    
    <tr>
    
      
      
        <td style="text-align: left">
      
      
        MirTif
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: left">
      
      
        
        
          <a class="citation" href="#Yang2008">[75]</a>
        
      
        </td>
    
    </tr>
  
    
    <tr>
    
      
      
        <td style="text-align: left">
      
      
        Yan et al.
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ○
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ●
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: left">
      
      
        
        
          <a class="citation" href="#Yan2007">[76]</a>
        
      
        </td>
    
    </tr>
  
    
    <tr>
    
      
      
        <td style="text-align: left">
      
      
        Xie et al.
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ○
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        ○
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: center">
      
      
        
      
        </td>
    
      
      
        <td style="text-align: left">
      
      
        
        
          <a class="citation" href="#Xie2005">[77]</a>
        
      
        </td>
    
    </tr>
  
  </tbody>
</table></div>

<p class="fs-2"><sup><strong>a</strong></sup>miRNA:mRNA pairing. ●: stringent seeds, ○:
moderately stringent seeds, Blank: seed sites not considered.<br />
<sup><strong>b</strong></sup>Site location. ●: target positions considered,
Blank: target positions not considered.<br />
<sup><strong>c</strong></sup>Conservation. ●: with/without conservation filter,
○: with conservation filter, Blank: conservation not considered.<br />
<sup><strong>d</strong></sup>Site accessibility. ●: site accessibility with
minimum free energy considered, ○: A:U rich flanking considered,
Blank: site accessibility not considered.<br />
<sup><strong>e</strong></sup>Multiple sites. ●: multiple sites considered,
○: the number of putative sites consided, Blank: multiple
co-operability not considered.<br />
<sup><strong>f</strong></sup>Expression profile. ●: expression profiles used,
Blank: expression profiles not used.</p><hr />
      <h4 id="T:2" class="no_toc text-caption">
        
        
          <a href="#T:2" class="anchor-heading" aria-labelledby="T:2"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> <strong>Table 2</strong>. Resource availability for miRNA target prediction tools.
        
        
      </h4>
    

<div class="table-wrapper"><table>
<thead>

<tr>

  
  
    <th rowspan="2">Tool</th>
  

  
  
    <th rowspan="2">Predicted species<sup>a</sup></th>
  

  
  
    <th colspan="3">Web access</th>
  

  
  
    <th rowspan="2">SW<sup>d</sup></th>
  

  
  
    <th rowspan="2">URL</th>
  

  
    
    </tr><tr>
  
  
    <th>Online tool</th>
  

  
  
    <th>Own miRNA<sup>b</sup></th>
  

  
  
    <th>Own mRNA<sup>c</sup></th>
  

</tr>
</thead>

<tbody>

  <tr>
  
    
    <td>TargetScan</td>
  
    
    <td>23 vertebrates,f,w</td>
  
    
    <td>Yes</td>
  
    
    <td>Yes</td>
  
    
    <td>Yes</td>
  
    
    <td>Yes</td>
  
    
      
        <td><a href="http://www.targetscan.org">http://www.targetscan.org</a></td>
        
    
      
  </tr>

  <tr>
  
    
    <td>PicTar</td>
  
    
    <td>v,m,f,w</td>
  
    
    <td>Yes</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
      
        <td><a href="https://pictar.mdc-berlin.de">https://pictar.mdc-berlin.de</a></td>
        
    
      
  </tr>

  <tr>
  
    
    <td>miRanda</td>
  
    
    <td>h,m,r</td>
  
    
    <td>Yes</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
    <td>Yes</td>
  
    
      
    
    <td>http://www.microrna.org</td>
  
    
      
  </tr>

  <tr>
  
    
    <td>MicroCosm Targets</td>
  
    
    <td>44 species</td>
  
    
    <td>Yes</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
      
    
    <td>http://www.ebi.ac.uk/enright-srv/microcosm/htdocs/targets/v5</td>
  
    
      
  </tr>

  <tr>
  
    
    <td>RNAhybrid</td>
  
    
    <td></td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
    <td>Yes</td>
  
    
      
        <td><a href="https://bibiserv.techfak.uni-bielefeld.de/rnahybrid">https://bibiserv.techfak.uni-bielefeld.de/rnahybrid</a></td>
        
    
      
  </tr>

  <tr>
  
    
    <td>PITA</td>
  
    
    <td>h,m,f,w</td>
  
    
    <td>Yes</td>
  
    
    <td>Yes</td>
  
    
    <td>Yes</td>
  
    
    <td>Yes</td>
  
    
      
        <td><a href="https://genie.weizmann.ac.il/pubs/mir07">https://genie.weizmann.ac.il/pubs/mir07</a></td>
        
    
      
  </tr>

  <tr>
  
    
    <td>STarMir</td>
  
    
    <td></td>
  
    
    <td>Yes</td>
  
    
    <td>Yes</td>
  
    
    <td>Yes</td>
  
    
    <td>No</td>
  
    
      
    
    <td>http://sfold.wadsworth.org/starmir.pl</td>
  
    
      
  </tr>

  <tr>
  
    
    <td>Rajewsky &amp; Socci</td>
  
    
    <td>f</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
      
    
    <td></td>
  
    
      
  </tr>

  <tr>
  
    
    <td>Robins</td>
  
    
    <td>f,w</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
      
    
    <td></td>
  
    
      
  </tr>

  <tr>
  
    
    <td>mirWIP</td>
  
    
    <td>w</td>
  
    
    <td>Yes</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
    <td>Yes</td>
  
    
      
    
    <td>http://146.189.76.171/query.php</td>
  
    
      
  </tr>

  <tr>
  
    
    <td>MicroInspector</td>
  
    
    <td></td>
  
    
    <td>Yes</td>
  
    
    <td>Yes</td>
  
    
    <td>Yes</td>
  
    
    <td>No</td>
  
    
      
    
    <td>http://mirna.imbb.forth.gr/microinspector</td>
  
    
      
  </tr>

  <tr>
  
    
    <td>MicroTar</td>
  
    
    <td></td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
    <td>Yes</td>
  
    
      
    
    <td>http://tiger.dbs.nus.edu.sg/microtar</td>
  
    
      
  </tr>

  <tr>
  
    
    <td>MirTarget2</td>
  
    
    <td>h,m,r,d,c</td>
  
    
    <td>Yes</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
      
        <td><a href="http://mirdb.org">http://mirdb.org</a></td>
        
    
      
  </tr>

  <tr>
  
    
    <td>miTarget</td>
  
    
    <td></td>
  
    
    <td>Yes</td>
  
    
    <td>Yes</td>
  
    
    <td>Yes</td>
  
    
    <td>No</td>
  
    
      
    
    <td>http://cbit.snu.ac.kr/∼miTarget</td>
  
    
      
  </tr>

  <tr>
  
    
    <td>TargetMiner</td>
  
    
    <td>h</td>
  
    
    <td>Yes</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
      
    
    <td>http://www.isical.ac.in/∼bioinfo_miu</td>
  
    
      
  </tr>

  <tr>
  
    
    <td>EIMMo</td>
  
    
    <td>h,m,f,z</td>
  
    
    <td>Yes</td>
  
    
    <td>No</td>
  
    
    <td>Yes</td>
  
    
    <td>No</td>
  
    
      
    
    <td>http://www.mirz.unibas.ch/ElMMo2</td>
  
    
      
  </tr>

  <tr>
  
    
    <td>NBmiRTar</td>
  
    
    <td></td>
  
    
    <td>Yes</td>
  
    
    <td>Yes</td>
  
    
    <td>Yes</td>
  
    
    <td>No</td>
  
    
      
    
    <td>http://wotan.wistar.upenn.edu/NBmiRTar</td>
  
    
      
  </tr>

  <tr>
  
    
    <td>TargetBoost</td>
  
    
    <td>w</td>
  
    
    <td>Yes</td>
  
    
    <td>Yes</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
      
    
    <td>https://demo1.interagon.com/targetboost</td>
  
    
      
  </tr>

  <tr>
  
    
    <td>RNA22</td>
  
    
    <td></td>
  
    
    <td>Yes</td>
  
    
    <td>Yes</td>
  
    
    <td>Yes</td>
  
    
    <td>No</td>
  
    
      
    
    <td>http://cbcsrv.watson.ibm.com/rna22.html</td>
  
    
      
  </tr>

  <tr>
  
    
    <td>TargetRank</td>
  
    
    <td>h,m</td>
  
    
    <td>Yes</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
      
        <td><a href="http://hollywood.mit.edu/targetrank">http://hollywood.mit.edu/targetrank</a></td>
        
    
      
  </tr>

  <tr>
  
    
    <td>EMBL</td>
  
    
    <td>f</td>
  
    
    <td>Yes</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
      
    
    <td>http://www.russell.embl-heidelberg.de/miRNAs</td>
  
    
      
  </tr>

  <tr>
  
    
    <td>MovingTarget</td>
  
    
    <td>f</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
      
    
    <td></td>
  
    
      
  </tr>

  <tr>
  
    
    <td>DIANA-microT</td>
  
    
    <td></td>
  
    
    <td>Yes</td>
  
    
    <td>Yes</td>
  
    
    <td>Yes</td>
  
    
    <td>No</td>
  
    
      
    
    <td>http://diana.pcbi.upenn.edu/cgi-bin/micro_t.cgi</td>
  
    
      
  </tr>

  <tr>
  
    
    <td>HOCTAR</td>
  
    
    <td>h</td>
  
    
    <td>Yes</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
      
        <td><a href="http://hoctar.tigem.it">http://hoctar.tigem.it</a></td>
        
    
      
  </tr>

  <tr>
  
    
    <td>Stanhope</td>
  
    
    <td>h</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
      
    
    <td></td>
  
    
      
  </tr>

  <tr>
  
    
    <td>GenMiR++</td>
  
    
    <td>h</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
    <td>Yes</td>
  
    
      
    
    <td>http://www.psi.toronto.edu/genmir/</td>
  
    
      
  </tr>

  <tr>
  
    
    <td>HuMiTar</td>
  
    
    <td>h</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
      
    
    <td></td>
  
    
      
  </tr>

  <tr>
  
    
    <td>MirTif</td>
  
    
    <td></td>
  
    
    <td>Yes</td>
  
    
    <td>Yes</td>
  
    
    <td>Yes</td>
  
    
    <td>No</td>
  
    
      
    
    <td>http://bsal.ym.edu.tw/mirtif</td>
  
    
      
  </tr>

  <tr>
  
    
    <td>Yan et al.</td>
  
    
    <td>h</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
      
    
    <td></td>
  
    
      
  </tr>

  <tr>
  
    
    <td>Xie et al.</td>
  
    
    <td>h,m,r,d</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
    <td>No</td>
  
    
      
    
    <td></td>
  
    
      
  </tr>

</tbody>

</table></div>

<p class="fs-2"><sup><strong>a</strong></sup>Both species of pre-computed prediction and the species
available on the web tool are listed. Letters indicate the species: fly
(f), worm (w), human (h), mouse (m), rat (r), chicken (c), zebra fish
(z), and dog (d). Cells are left empty when no information is
available.<br />
<sup><strong>b</strong></sup>Yes/No indicate whether own miRNA sequences can be used on
the web interface or not.<br />
<sup><strong>c</strong></sup>Yes/No indicate whether own mRNA sequences can be used on
the web interface or not.<br />
<sup><strong>d</strong></sup>SW: Software availability (executable or source code).</p><hr />

<p>Most tools rely on either one or a combination of seed matching, site
accessibility and evolutionary conservation features, although some
recently developed tools use expression profiles. No tools have
successfully incorporated some of the important features, such as
optimal distances of multiple miRNA sites or supplementary sites in CDS
and 5’ UTR.</p>

<p>TargetScan <a class="citation" href="#Friedman2009">[9,21,22]</a>, PicTar
<a class="citation" href="#Chen2006">[49,50,51,52]</a> , miRanda
<a class="citation" href="#John2004">[37,53]</a>, RNAhybrid <a class="citation" href="#Kruger2006">[56,57]</a> and
PITA <a class="citation" href="#Kertesz2007">[35]</a> have been frequently used for performance
comparisons or as preprocessors for other tools to obtain initial
putative target sites. Of the five, TargetScan often shows the best
performance in comparisons. TargetScan considers only stringent seeds,
however, and therefore ignores many potential targets.</p>
      <h2 id="S:5">
        
        
          <a href="#S:5" class="anchor-heading" aria-labelledby="S:5"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> Summary
        
        
      </h2>
    
<p>Finding true functional miRNA targets is still challenging even though
many biological features of miRNA targeting have been revealed
experimentally and computationally. Building a model with more features
might achieve higher accuracy and enhance site recognition efficacy, but
its implementation might also become more complex. None of the existing
prediction tools has been able to incorporate all currently known
features. We expect that a new approach that can combine the features
from the six categories we have shown will significantly improve
computational miRNA target prediction.</p>

<p>Another important problem that has hardly been addressed is predicting
target interactions between different miRNAs. Different miRNAs can
cooperatively regulate individual targets, but miRNA expression
signatures differ between cell types and cellular conditions.
Determining how varying miRNA expression affects target regulation in
cancerous versus normal cells, for example, will therefore be a major
problem in the coming years.</p>
      <h2 id="S:6">
        
        
          <a href="#S:6" class="anchor-heading" aria-labelledby="S:6"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> References
        
        
      </h2>
    
<ol class="bibliography"><li><span id="Ambros2004">Ambros V. The functions of animal microRNAs. Nature 2004;431:350–5. <a href="https://doi.org/10.1038/nature02871">https://doi.org/10.1038/nature02871.</a></span>
</li>
<li><span id="Soifer2007">Soifer HS, Rossi JJ, Sætrom P. MicroRNAs in disease and potential therapeutic applications. Molecular Therapy 2007;15:2070–9. <a href="https://doi.org/10.1038/sj.mt.6300311">https://doi.org/10.1038/sj.mt.6300311.</a></span>
</li>
<li><span id="Croce2009">Croce CM. Causes and consequences of microRNA dysregulation in cancer. Nature Reviews Genetics 2009;10:704–14. <a href="https://doi.org/10.1038/nrg2634">https://doi.org/10.1038/nrg2634.</a></span>
</li>
<li><span id="Stefani2008">Stefani G, Slack FJ. Small non-coding RNAs in animal development. Nature Reviews Molecular Cell Biology 2008;9:219–30. <a href="https://doi.org/10.1038/nrm2347">https://doi.org/10.1038/nrm2347.</a></span>
</li>
<li><span id="Bartel2004">Bartel DP. MicroRNAs: genomics, biogenesis, mechanism, and function. Cell 2004;116:281–97. <a href="https://doi.org/10.1016/s0092-8674(04)00045-5">https://doi.org/10.1016/s0092-8674(04)00045-5.</a></span>
</li>
<li><span id="Carthew2009">Carthew RW, Sontheimer EJ. Origins and mechanisms of miRNAs and siRNAs. Cell 2009;136:642–55. <a href="https://doi.org/10.1016/j.cell.2009.01.035">https://doi.org/10.1016/j.cell.2009.01.035.</a></span>
</li>
<li><span id="Bartel2009">Bartel DP. MicroRNAs: target recognition and regulatory functions. Cell 2009;136:215–33. <a href="https://doi.org/10.1016/j.cell.2009.01.002">https://doi.org/10.1016/j.cell.2009.01.002.</a></span>
</li>
<li><span id="Filipowicz2008">Filipowicz W, Bhattacharyya SN, Sonenberg N. Mechanisms of post-transcriptional regulation by microRNAs: are the answers in sight? Nature Reviews Genetics 2008;9:102–14. <a href="https://doi.org/10.1038/nrg2290">https://doi.org/10.1038/nrg2290.</a></span>
</li>
<li><span id="Friedman2009">Friedman RC, Farh KK-H, Burge CB, Bartel DP. Most mammalian mRNAs are conserved targets of microRNAs. Genome Research 2008;19:92–105. <a href="https://doi.org/10.1101/gr.082701.108">https://doi.org/10.1101/gr.082701.108.</a></span>
</li>
<li><span id="Hamilton1999">Hamilton AJ, Baulcombe DC. A species of small antisense RNA in posttranscriptional gene silencing in plants. Science 1999;286:950–2. <a href="https://doi.org/10.1126/science.286.5441.950">https://doi.org/10.1126/science.286.5441.950.</a></span>
</li>
<li><span id="Zamore2000">Zamore PD, Tuschl T, Sharp PA, Bartel DP. RNAi: double-stranded RNA directs the ATP-dependent cleavage of mRNA at 21 to 23 nucleotide intervals. Cell 2000;101:25–33. <a href="https://doi.org/10.1016/s0092-8674(00)80620-0">https://doi.org/10.1016/s0092-8674(00)80620-0.</a></span>
</li>
<li><span id="Kim2009">Kim VN, Han J, Siomi MC. Biogenesis of small RNAs in animals. Nature Reviews Molecular Cell Biology 2009;10:126–39. <a href="https://doi.org/10.1038/nrm2632">https://doi.org/10.1038/nrm2632.</a></span>
</li>
<li><span id="Winter2009">Winter J, Jung S, Keller S, Gregory RI, Diederichs S. Many roads to maturity: microRNA biogenesis pathways and their regulation. Nature Cell Biology 2009;11:228–34. <a href="https://doi.org/10.1038/ncb0309-228">https://doi.org/10.1038/ncb0309-228.</a></span>
</li>
<li><span id="Birmingham2006">Birmingham A, Anderson EM, Reynolds A, Ilsley-Tyree D, Leake D, Fedorov Y, et al. 3\prime UTR seed matches, but not overall identity, are associated with RNAi off-targets. Nature Methods 2006;3:199–204. <a href="https://doi.org/10.1038/nmeth854">https://doi.org/10.1038/nmeth854.</a></span>
</li>
<li><span id="Jackson2006">Jackson AL, Burchard J, Schelter J, Chau BN, Cleary M, Lim L, et al. Widespread siRNA "off-target" transcript silencing mediated by seed region sequence complementarity. RNA 2006;12:1179–87. <a href="https://doi.org/10.1261/rna.25706">https://doi.org/10.1261/rna.25706.</a></span>
</li>
<li><span id="Jackson2003">Jackson AL, Bartz SR, Schelter J, Kobayashi SV, Burchard J, Mao M, et al. Expression profiling reveals off-target gene regulation by RNAi. Nature Biotechnology 2003;21:635–7. <a href="https://doi.org/10.1038/nbt831">https://doi.org/10.1038/nbt831.</a></span>
</li>
<li><span id="Griffiths-Jones2008">Griffiths-Jones S, Saini HK, Dongen S van, Enright AJ. miRBase: tools for microRNA genomics. Nucleic Acids Research 2007;36:D154–D158. <a href="https://doi.org/10.1093/nar/gkm952">https://doi.org/10.1093/nar/gkm952.</a></span>
</li>
<li><span id="Stark2003">Stark A, Brennecke J, Russell RB, Cohen SM. Identification of Drosophila MicroRNA targets. PLoS Biology 2003;1:e60. <a href="https://doi.org/10.1371/journal.pbio.0000060">https://doi.org/10.1371/journal.pbio.0000060.</a></span>
</li>
<li><span id="Rajewsky2004">Rajewsky N, Socci ND. Computational identification of microRNA targets. Developmental Biology 2004;267:529–35. <a href="https://doi.org/10.1016/j.ydbio.2003.12.003">https://doi.org/10.1016/j.ydbio.2003.12.003.</a></span>
</li>
<li><span id="Lewis2003">Lewis BP, Shih I-hung, Jones-Rhoades MW, Bartel DP, Burge CB. Prediction of mammalian MicroRNA targets. Cell 2003;115:787–98. <a href="https://doi.org/10.1016/s0092-8674(03)01018-3">https://doi.org/10.1016/s0092-8674(03)01018-3.</a></span>
</li>
<li><span id="Grimson2007">Grimson A, Farh KK-H, Johnston WK, Garrett-Engele P, Lim LP, Bartel DP. MicroRNA targeting specificity in mammals: determinants beyond seed pairing. Molecular Cell 2007;27:91–105. <a href="https://doi.org/10.1016/j.molcel.2007.06.017">https://doi.org/10.1016/j.molcel.2007.06.017.</a></span>
</li>
<li><span id="Lewis2005">Lewis BP, Burge CB, Bartel DP. Conserved seed pairing, often flanked by adenosines, indicates that thousands of human genes are MicroRNA targets. Cell 2005;120:15–20. <a href="https://doi.org/10.1016/j.cell.2004.12.035">https://doi.org/10.1016/j.cell.2004.12.035.</a></span>
</li>
<li><span id="Gaidatzis2007">Gaidatzis D, Nimwegen E van, Hausser J, Zavolan M. Inference of miRNA targets using evolutionary conservation and pathway analysis. BMC Bioinformatics 2007;8:69. <a href="https://doi.org/10.1186/1471-2105-8-69">https://doi.org/10.1186/1471-2105-8-69.</a></span>
</li>
<li><span id="Hammell2008">Hammell M, Long D, Zhang L, Lee A, Carmack CS, Han M, et al. mirWIP: microRNA target prediction based on microRNA-containing ribonucleoprotein-enriched transcripts. Nature Methods 2008;5:813–9. <a href="https://doi.org/10.1038/nmeth.1247">https://doi.org/10.1038/nmeth.1247.</a></span>
</li>
<li><span id="Saetrom2007">Saetrom P, Heale BSE, Snøve O, Aagaard L, Alluin J, Rossi JJ. Distance constraints between microRNA target sites dictate efficacy and cooperativity. Nucleic Acids Res 2007;35:2333–42. <a href="https://doi.org/10.1093/nar/gkm133">https://doi.org/10.1093/nar/gkm133.</a></span>
</li>
<li><span id="Stark2005">Stark A, Brennecke J, Bushati N, Russell RB, Cohen SM. Animal MicroRNAs confer robustness to gene expression and have a significant impact on 3′UTR evolution. Cell 2005;123:1133–46. <a href="https://doi.org/10.1016/j.cell.2005.11.023">https://doi.org/10.1016/j.cell.2005.11.023.</a></span>
</li>
<li><span id="Majoros2007">Majoros WH, Ohler U. Spatial preferences of microRNA targets in 3’ untranslated regions. BMC Genomics 2007;8:152. <a href="https://doi.org/10.1186/1471-2164-8-152">https://doi.org/10.1186/1471-2164-8-152.</a></span>
</li>
<li><span id="Sandberg2008">Sandberg R, Neilson JR, Sarma A, Sharp PA, Burge CB. Proliferating cells express mRNAs with shortened 3’ untranslated regions and fewer MicroRNA target sites. Science 2008;320:1643–7. <a href="https://doi.org/10.1126/science.1155390">https://doi.org/10.1126/science.1155390.</a></span>
</li>
<li><span id="Lytle2007">Lytle JR, Yario TA, Steitz JA. Target mRNAs are repressed as efficiently by microRNA-binding sites in the 5’ UTR as in the 3’ UTR. Proceedings of the National Academy of Sciences 2007;104:9667–72. <a href="https://doi.org/10.1073/pnas.0703820104">https://doi.org/10.1073/pnas.0703820104.</a></span>
</li>
<li><span id="Kloosterman2004">Kloosterman WP, Wienholds E, Ketting RF, Plasterk RHA. Substrate requirements for let-7 function in the developing zebrafish embryo. Nucleic Acids Research 2004;32:6284–91. <a href="https://doi.org/10.1093/nar/gkh968">https://doi.org/10.1093/nar/gkh968.</a></span>
</li>
<li><span id="Lee2009">Lee I, Ajay SS, Yook JI, Kim HS, Hong SH, Kim NH, et al. New class of microRNA targets containing simultaneous 5’-UTR and 3’-UTR interaction sites. Genome Research 2009;19:1175–83. <a href="https://doi.org/10.1101/gr.089367.108">https://doi.org/10.1101/gr.089367.108.</a></span>
</li>
<li><span id="Sethupathy2006">Sethupathy P, Megraw M, Hatzigeorgiou AG. A guide through present computational approaches for the identification of mammalian microRNA targets. Nature Methods 2006;3:881–6. <a href="https://doi.org/10.1038/nmeth954">https://doi.org/10.1038/nmeth954.</a></span>
</li>
<li><span id="Burchard2009">Burchard J, Jackson AL, Malkov V, Needham RHV, Tan Y, Bartz SR, et al. MicroRNA-like off-target transcript regulation by siRNAs is species specific. RNA 2009;15:308–15. <a href="https://doi.org/10.1261/rna.1326809">https://doi.org/10.1261/rna.1326809.</a></span>
</li>
<li><span id="Long2007">Long D, Lee R, Williams P, Chan CY, Ambros V, Ding Y. Potent effect of target structure on microRNA function. Nature Structural &amp; Molecular Biology 2007;14:287–94. <a href="https://doi.org/10.1038/nsmb1226">https://doi.org/10.1038/nsmb1226.</a></span>
</li>
<li><span id="Kertesz2007">Kertesz M, Iovino N, Unnerstall U, Gaul U, Segal E. The role of site accessibility in microRNA target recognition. Nature Genetics 2007;39:1278–84. <a href="https://doi.org/10.1038/ng2135">https://doi.org/10.1038/ng2135.</a></span>
</li>
<li><span id="Eddy2004">Eddy SR. How do RNA folding algorithms work? Nature Biotechnology 2004;22:1457–8. <a href="https://doi.org/10.1038/nbt1104-1457">https://doi.org/10.1038/nbt1104-1457.</a></span>
</li>
<li><span id="John2004">John B, Enright AJ, Aravin A, Tuschl T, Sander C, Marks DS. Human MicroRNA targets. PLoS Biology 2004;2:e363. <a href="https://doi.org/10.1371/journal.pbio.0020363">https://doi.org/10.1371/journal.pbio.0020363.</a></span>
</li>
<li><span id="Doench2004">Doench JG, Sharp PA. Specificity of microRNA target selection in translational repression. Genes &amp; Development 2004;18:504–11. <a href="https://doi.org/10.1101/gad.1184404">https://doi.org/10.1101/gad.1184404.</a></span>
</li>
<li><span id="Haley2004">Haley B, Zamore PD. Kinetic analysis of the RNAi enzyme complex. Nature Structural &amp; Molecular Biology 2004;11:599–606. <a href="https://doi.org/10.1038/nsmb780">https://doi.org/10.1038/nsmb780.</a></span>
</li>
<li><span id="Gu2009">Gu S, Jin L, Zhang F, Sarnow P, Kay MA. Biological basis for restriction of microRNA targets to the 3′ untranslated region in mammalian mRNAs. Nature Structural &amp; Molecular Biology 2009;16:144–50. <a href="https://doi.org/10.1038/nsmb.1552">https://doi.org/10.1038/nsmb.1552.</a></span>
</li>
<li><span id="Duursma2008">Duursma AM, Kedde M, Schrier M, Sage C le, Agami R. miR-148 targets human DNMT3b protein coding region. RNA 2008;14:872–7. <a href="https://doi.org/10.1261/rna.972008">https://doi.org/10.1261/rna.972008.</a></span>
</li>
<li><span id="Elcheva2009">Elcheva I, Goswami S, Noubissi FK, Spiegelman VS. CRD-BP protects the coding region of βTrCP1 mRNA from miR-183-mediated degradation. Molecular Cell 2009;35:240–6. <a href="https://doi.org/10.1016/j.molcel.2009.06.007">https://doi.org/10.1016/j.molcel.2009.06.007.</a></span>
</li>
<li><span id="Tay2008">Tay Y, Zhang J, Thomson AM, Lim B, Rigoutsos I. MicroRNAs to Nanog, Oct4 and Sox2 coding regions modulate embryonic stem cell differentiation. Nature 2008;455:1124–8. <a href="https://doi.org/10.1038/nature07299">https://doi.org/10.1038/nature07299.</a></span>
</li>
<li><span id="Forman2008">Forman JJ, Legesse-Miller A, Coller HA. A search for conserved sequences in coding regions reveals that the let-7 microRNA targets Dicer within its coding sequence. Proceedings of the National Academy of Sciences 2008;105:14879–84. <a href="https://doi.org/10.1073/pnas.0803230105">https://doi.org/10.1073/pnas.0803230105.</a></span>
</li>
<li><span id="Rajewsky2006">Rajewsky N. microRNA target predictions in animals. Nature Genetics 2006;38:S8–S13. <a href="https://doi.org/10.1038/ng1798">https://doi.org/10.1038/ng1798.</a></span>
</li>
<li><span id="Lim2005">Lim LP, Lau NC, Garrett-Engele P, Grimson A, Schelter JM, Castle J, et al. Microarray analysis shows that some microRNAs downregulate large numbers of target mRNAs. Nature 2005;433:769–73. <a href="https://doi.org/10.1038/nature03315">https://doi.org/10.1038/nature03315.</a></span>
</li>
<li><span id="Selbach2008">Selbach M, Schwanhäusser B, Thierfelder N, Fang Z, Khanin R, Rajewsky N. Widespread changes in protein synthesis induced by microRNAs. Nature 2008;455:58–63. <a href="https://doi.org/10.1038/nature07228">https://doi.org/10.1038/nature07228.</a></span>
</li>
<li><span id="Baek2008">Baek D, Villén J, Shin C, Camargo FD, Gygi SP, Bartel DP. The impact of microRNAs on protein output. Nature 2008;455:64–71. <a href="https://doi.org/10.1038/nature07242">https://doi.org/10.1038/nature07242.</a></span>
</li>
<li><span id="Chen2006">Chen K, Rajewsky N. Natural selection on human microRNA binding sites inferred from SNP data. Nature Genetics 2006;38:1452–6. <a href="https://doi.org/10.1038/ng1910">https://doi.org/10.1038/ng1910.</a></span>
</li>
<li><span id="Grun2005">Grün D, Wang Y-L, Langenberger D, Gunsalus KC, Rajewsky N. microRNA target predictions across seven drosophila species and comparison to mammalian targets. PLoS Computational Biology 2005;1:e13. <a href="https://doi.org/10.1371/journal.pcbi.0010013">https://doi.org/10.1371/journal.pcbi.0010013.</a></span>
</li>
<li><span id="Krek2005">Krek A, Grün D, Poy MN, Wolf R, Rosenberg L, Epstein EJ, et al. Combinatorial microRNA target predictions. Nature Genetics 2005;37:495–500. <a href="https://doi.org/10.1038/ng1536">https://doi.org/10.1038/ng1536.</a></span>
</li>
<li><span id="Lall2006">Lall S, Grün D, Krek A, Chen K, Wang Y-L, Dewey CN, et al. A genome-wide map of conserved MicroRNA targets in C. Elegans. Current Biology 2006;16:460–71. <a href="https://doi.org/10.1016/j.cub.2006.01.050">https://doi.org/10.1016/j.cub.2006.01.050.</a></span>
</li>
<li><span id="Betel2008">Betel D, Wilson M, Gabow A, Marks DS, Sander C. The microRNA.org resource: targets and expression. Nucleic Acids Research 2008;36:D149–D153. <a href="https://doi.org/10.1093/nar/gkm995">https://doi.org/10.1093/nar/gkm995.</a></span>
</li>
<li><span id="Enright2003">Enright AJ, John B, Gaul U, Tuschl T, Sander C, Marks DS. MicroRNA targets in Drosophila. Genome Biology 2003;5:R1. <a href="https://doi.org/10.1186/gb-2003-5-1-r1">https://doi.org/10.1186/gb-2003-5-1-r1.</a></span>
</li>
<li><span id="Griffiths-Jones2006a">Griffiths-Jones S, Grocock RJ, Dongen S van, Bateman A, Enright AJ. miRBase: microRNA sequences, targets and gene nomenclature. Nucleic Acids Research 2006;34:D140–D144. <a href="https://doi.org/10.1093/nar/gkj112">https://doi.org/10.1093/nar/gkj112.</a></span>
</li>
<li><span id="Kruger2006">Kruger J, Rehmsmeier M. RNAhybrid: microRNA target prediction easy, fast and flexible. Nucleic Acids Research 2006;34:W451–W454. <a href="https://doi.org/10.1093/nar/gkl243">https://doi.org/10.1093/nar/gkl243.</a></span>
</li>
<li><span id="Rehmsmeier2004">Rehmsmeier M, Steffen P, Höchsmann M, Giegerich R. Fast and effective prediction of microRNA/target duplexes. RNA 2004;10:1507–17. <a href="https://doi.org/10.1261/rna.5248604">https://doi.org/10.1261/rna.5248604.</a></span>
</li>
<li><span id="Robins2005">Robins H, Li Y, Padgett RW. Incorporating structure to predict microRNA targets. Proceedings of the National Academy of Sciences 2005;102:4006–9. <a href="https://doi.org/10.1073/pnas.0500775102">https://doi.org/10.1073/pnas.0500775102.</a></span>
</li>
<li><span id="Rusinov2005">Rusinov V, Baev V, Minkov IN, Tabler M. MicroInspector: a web tool for detection of miRNA binding sites in an RNA sequence. Nucleic Acids Research 2005;33:W696–W700. <a href="https://doi.org/10.1093/nar/gki364">https://doi.org/10.1093/nar/gki364.</a></span>
</li>
<li><span id="Thadani2006">Thadani R, Tammi MT. MicroTar: predicting microRNA targets from RNA duplexes. BMC Bioinformatics 2006;7:S20–. <a href="https://doi.org/10.1186/1471-2105-7-s5-s20">https://doi.org/10.1186/1471-2105-7-s5-s20.</a></span>
</li>
<li><span id="Wang2008">Wang X, Naqa IME. Prediction of both conserved and nonconserved microRNA targets in animals. Bioinformatics 2007;24:325–32. <a href="https://doi.org/10.1093/bioinformatics/btm595">https://doi.org/10.1093/bioinformatics/btm595.</a></span>
</li>
<li><span id="Kim2006">Kim S-K, Nam J-W, Rhee J-K, Lee W-J, Zhang B-T. miTarget: microRNA target gene prediction using a support vector machine. BMC Bioinformatics 2006;7:411. <a href="https://doi.org/10.1186/1471-2105-7-411">https://doi.org/10.1186/1471-2105-7-411.</a></span>
</li>
<li><span id="Bandyopadhyay2009">Bandyopadhyay S, Mitra R. TargetMiner: microRNA target prediction with systematic identification of tissue-specific negative examples. Bioinformatics 2009;25:2625–31. <a href="https://doi.org/10.1093/bioinformatics/btp503">https://doi.org/10.1093/bioinformatics/btp503.</a></span>
</li>
<li><span id="Yousef2007">Yousef M, Jung S, Kossenkov AV, Showe LC, Showe MK. Naïve Bayes for microRNA target predictions—machine learning for microRNA targets. Bioinformatics 2007;23:2987–92. <a href="https://doi.org/10.1093/bioinformatics/btm484">https://doi.org/10.1093/bioinformatics/btm484.</a></span>
</li>
<li><span id="Saetrom2005">Saetrom O, Snove J O., Saetrom P. Weighted sequence motifs as an improved seeding step in microRNA target prediction algorithms. RNA 2005;11:995–1003. <a href="https://doi.org/10.1261/rna.7290705">https://doi.org/10.1261/rna.7290705.</a></span>
</li>
<li><span id="Miranda2006">Miranda KC, Huynh T, Tay Y, Ang Y-S, Tam W-L, Thomson AM, et al. A pattern-based method for the identification of MicroRNA binding sites and their corresponding heteroduplexes. Cell 2006;126:1203–17. <a href="https://doi.org/10.1016/j.cell.2006.07.031">https://doi.org/10.1016/j.cell.2006.07.031.</a></span>
</li>
<li><span id="Nielsen2007">Nielsen CB, Shomron N, Sandberg R, Hornstein E, Kitzman J, Burge CB. Determinants of targeting by endogenous and exogenous microRNAs and siRNAs. RNA 2007;13:1894–910. <a href="https://doi.org/10.1261/rna.768207">https://doi.org/10.1261/rna.768207.</a></span>
</li>
<li><span id="Brennecke2005">Brennecke J, Stark A, Russell RB, Cohen SM. Principles of MicroRNA-target recognition. PLoS Biology 2005;3:e85. <a href="https://doi.org/10.1371/journal.pbio.0030085">https://doi.org/10.1371/journal.pbio.0030085.</a></span>
</li>
<li><span id="Burgler2005">Burgler C, Macdonald PM. Prediction and verification of microRNA targets by MovingTargets, a highly adaptable prediction method. BMC Genomics 2005;6:88–. <a href="https://doi.org/10.1186/1471-2164-6-88">https://doi.org/10.1186/1471-2164-6-88.</a></span>
</li>
<li><span id="Kiriakidou2004">Kiriakidou M, Nelson PT, Kouranov A, Fitziev P, Bouyioukos C, Mourelatos Z, et al. A combined computational-experimental approach predicts human microRNA targets. Genes &amp; Development 2004;18:1165–78. <a href="https://doi.org/10.1101/gad.1184704">https://doi.org/10.1101/gad.1184704.</a></span>
</li>
<li><span id="Gennarino2009">Gennarino VA, Sardiello M, Avellino R, Meola N, Maselli V, Anand S, et al. MicroRNA target prediction by expression analysis of host genes. Genome Research 2008;19:481–90. <a href="https://doi.org/10.1101/gr.084129.108">https://doi.org/10.1101/gr.084129.108.</a></span>
</li>
<li><span id="Stanhope2009">Stanhope SA, Sengupta S, Boon J den, Ahlquist P, Newton MA. Statistical use of argonaute expression and RISC assembly in microRNA target identification. PLoS Computational Biology 2009;5:e1000516. <a href="https://doi.org/10.1371/journal.pcbi.1000516">https://doi.org/10.1371/journal.pcbi.1000516.</a></span>
</li>
<li><span id="Huang2007">Huang JC, Babak T, Corson TW, Chua G, Khan S, Gallie BL, et al. Using expression profiling data to identify human microRNA targets. Nature Methods 2007;4:1045–9. <a href="https://doi.org/10.1038/nmeth1130">https://doi.org/10.1038/nmeth1130.</a></span>
</li>
<li><span id="Ruan2008">Ruan J, Chen H, Kurgan L, Chen K, Kang C, Pu P. HuMiTar: A sequence-based method for prediction of human microRNA targets. Algorithms for Molecular Biology 2008;3:16. <a href="https://doi.org/10.1186/1748-7188-3-16">https://doi.org/10.1186/1748-7188-3-16.</a></span>
</li>
<li><span id="Yang2008">Yang Y, Wang Y-P, Li K-B. MiRTif: a support vector machine-based microRNA target interaction filter. BMC Bioinformatics 2008;9:S4. <a href="https://doi.org/10.1186/1471-2105-9-s12-s4">https://doi.org/10.1186/1471-2105-9-s12-s4.</a></span>
</li>
<li><span id="Yan2007">Yan X, Chao T, Tu K, Zhang Y, Xie L, Gong Y, et al. Improving the prediction of human microRNA target genes by using ensemble algorithm. FEBS Letters 2007;581:1587–93. <a href="https://doi.org/10.1016/j.febslet.2007.03.022">https://doi.org/10.1016/j.febslet.2007.03.022.</a></span>
</li>
<li><span id="Xie2005">Xie X, Lu J, Kulbokas EJ, Golub TR, Mootha V, Lindblad-Toh K, et al. Systematic discovery of regulatory motifs in human promoters and 3′ UTRs by comparison of several mammals. Nature 2005;434:338–45. <a href="https://doi.org/10.1038/nature03441">https://doi.org/10.1038/nature03441.</a></span>
</li></ol>

        

        

        
          <hr>
          <h2>Leave a comment</h2>
          <div id="disqus_thread"></div>
        

        
        
          <hr>
          <footer>
            
              <p><a href="#top" id="back-to-top">Back to top</a></p>
            

            <p class="text-small text-grey-dk-100 mb-0">Copyright &copy; 2021 Takaya Saito.</p>

            
              <div class="d-flex mt-2">
                
                
              </div>
            
          </footer>
        

      </div>
    </div>

    
      

      <div class="search-overlay"></div>
    
  </div>

  
     <script>addBackToTop({
       id: 'back-to-top-js-button'
     })</script>
  

  
    <script>
        /**
        *  RECOMMENDED CONFIGURATION VARIABLES: EDIT AND UNCOMMENT THE SECTION BELOW TO INSERT DYNAMIC VALUES FROM YOUR PLATFORM OR CMS.
        *  LEARN WHY DEFINING THESE VARIABLES IS IMPORTANT: https://disqus.com/admin/universalcode/#configuration-variables    */
        /*
        var disqus_config = function () {
        this.page.url = PAGE_URL;  // Replace PAGE_URL with your page's canonical URL variable
        this.page.identifier = PAGE_IDENTIFIER; // Replace PAGE_IDENTIFIER with your page's unique identifier variable
        };
        */
        (function() { // DON'T EDIT BELOW THIS LINE
        var d = document, s = d.createElement('script');
        s.src = 'https://mitkb.disqus.com/embed.js';
        s.setAttribute('data-timestamp', +new Date());
        (d.head || d.body).appendChild(s);
        })();
    </script>
    <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
  

</body>
</html>

