

<!DOCTYPE html>

<html lang="en-US">
<head>
  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=Edge">

  

  <link rel="shortcut icon" href="/mitkb/favicon.ico" type="image/x-icon">

  <link rel="stylesheet" href="/mitkb/assets/css/just-the-docs-default.css">

  
    <script async src="https://www.googletagmanager.com/gtag/js?id=G-84ETYTKDBN"></script>
    <script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());

      gtag('config', 'G-84ETYTKDBN', { 'anonymize_ip': true });
    </script>

  

  
    <script type="text/javascript" src="/mitkb/assets/js/vendor/lunr.min.js"></script>
  
  <script type="text/javascript" src="/mitkb/assets/js/just-the-docs.js"></script>

  <meta name="viewport" content="width=device-width, initial-scale=1">

  <!-- Begin Jekyll SEO tag v2.7.1 -->
<title>1 Introduction | mitkb</title>
<meta name="generator" content="Jekyll v4.2.0" />
<meta property="og:title" content="1 Introduction" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Introduction" />
<meta property="og:description" content="Introduction" />
<link rel="canonical" href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/1-introduction/" />
<meta property="og:url" content="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/1-introduction/" />
<meta property="og:site_name" content="mitkb" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="1 Introduction" />
<script type="application/ld+json">
{"description":"Introduction","url":"https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/1-introduction/","@type":"WebPage","headline":"1 Introduction","publisher":{"@type":"Organization","logo":{"@type":"ImageObject","url":"https://ncrnas.github.io/mitkb/assets/images/logo.png"}},"@context":"https://schema.org"}</script>
<!-- End Jekyll SEO tag -->


  
<meta name="google-site-verification" content="vstkqjcoJe9vQcOOpsCKrxv6WfqZ2SKRNWSv7XoHFeQ" />

<link rel="apple-touch-icon" sizes="180x180" href="/mitkb/assets/images/favicon/apple-touch-icon.png">
<link rel="icon" type="image/png" sizes="32x32" href="/mitkb/assets/images/favicon/favicon-32x32.png">
<link rel="icon" type="image/png" sizes="16x16" href="/mitkb/assets/images/favicon/favicon-16x16.png">
<link rel="manifest" href="/mitkb/assets/images/favicon/site.webmanifest">
<link rel="mask-icon" href="/mitkb/assets/images/favicon/safari-pinned-tab.svg" color="#5bbad5">
<meta name="msapplication-TileColor" content="#2b5797">
<meta name="theme-color" content="#ffffff">

<!-- for mathjax support -->



  <script src="https://unpkg.com/vanilla-back-to-top@7.2.1/dist/vanilla-back-to-top.min.js"></script>


<script
  data-cookie-notice='{ "learnMoreLinkEnabled": true, "learnMoreLinkHref": "/mitkb/terms/" }'
  src="https://unpkg.com/cookie-notice@^1/dist/cookie.notice.min.js"
></script>


</head>

<body>
  <svg xmlns="http://www.w3.org/2000/svg" style="display: none;">
    <symbol id="svg-link" viewBox="0 0 24 24">
      <title>Link</title>
      <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-link">
        <path d="M10 13a5 5 0 0 0 7.54.54l3-3a5 5 0 0 0-7.07-7.07l-1.72 1.71"></path><path d="M14 11a5 5 0 0 0-7.54-.54l-3 3a5 5 0 0 0 7.07 7.07l1.71-1.71"></path>
      </svg>
    </symbol>
    <symbol id="svg-search" viewBox="0 0 24 24">
      <title>Search</title>
      <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-search">
        <circle cx="11" cy="11" r="8"></circle><line x1="21" y1="21" x2="16.65" y2="16.65"></line>
      </svg>
    </symbol>
    <symbol id="svg-menu" viewBox="0 0 24 24">
      <title>Menu</title>
      <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-menu">
        <line x1="3" y1="12" x2="21" y2="12"></line><line x1="3" y1="6" x2="21" y2="6"></line><line x1="3" y1="18" x2="21" y2="18"></line>
      </svg>
    </symbol>
    <symbol id="svg-arrow-right" viewBox="0 0 24 24">
      <title>Expand</title>
      <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-chevron-right">
        <polyline points="9 18 15 12 9 6"></polyline>
      </svg>
    </symbol>
    <symbol id="svg-doc" viewBox="0 0 24 24">
      <title>Document</title>
      <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" class="feather feather-file">
        <path d="M13 2H6a2 2 0 0 0-2 2v16a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V9z"></path><polyline points="13 2 13 9 20 9"></polyline>
      </svg>
    </symbol>
  </svg>

  <div class="side-bar">
    <div class="site-header">
      <a href="https://ncrnas.github.io/mitkb/" class="site-title lh-tight">
  <div class="site-logo"></div>

</a>
      <a href="#" id="menu-button" class="site-button">
        <svg viewBox="0 0 24 24" class="icon"><use xlink:href="#svg-menu"></use></svg>
      </a>
    </div>
    <nav role="navigation" aria-label="Main" id="site-nav" class="site-nav">
      
        <ul class="nav-list"><li class="nav-list-item"><a href="https://ncrnas.github.io/mitkb/" class="nav-list-link">Home</a></li><li class="nav-list-item active"><a href="#" class="nav-list-expander"><svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg></a><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/" class="nav-list-link">Overview</a><ul class="nav-list "><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/0-front-matter/" class="nav-list-link">0 Front matter</a></li><li class="nav-list-item  active"><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/1-introduction/" class="nav-list-link active">1 Introduction</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/2-papers-and-their-corresponding-sub-goals/" class="nav-list-link">2 Papers and sub-goals</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/3-micrornas-and-other-non-coding-rnas/" class="nav-list-link">3 miRNAs and ncRNAs</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/4-high-throughput-biological-experiments/" class="nav-list-link">4 Biological experiments</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/5-statistical-tests-and-methods/" class="nav-list-link">5 Statistical methods</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/6-machine-learning-theory-and-support-vector-machine/" class="nav-list-link">6 Support vector machine</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/7-computational-implementation/" class="nav-list-link">7 Implementation</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/8-future-perspectives/" class="nav-list-link">8 Future perspectives</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander"><svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg></a><a href="https://ncrnas.github.io/mitkb/mirna-target-prediction/" class="nav-list-link">Target prediction</a><ul class="nav-list "><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/mirna-target-prediction/micrornas-targeting-and-target-prediction/" class="nav-list-link">Review: miRNA target</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/mirna-target-prediction/two-step-svm-model/" class="nav-list-link">Research: Two-step SVM</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/mirna-target-prediction/two-step-svm-model-supplementary-information/" class="nav-list-link">Supplementary information</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander"><svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg></a><a href="https://ncrnas.github.io/mitkb/mirna-high-throughput-experiments/" class="nav-list-link">Biological experiments</a><ul class="nav-list "><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/mirna-high-throughput-experiments/confounding-factors-in-mirna-experiments/" class="nav-list-link">Research: Confounding</a></li><li class="nav-list-item "><a href="https://ncrnas.github.io/mitkb/mirna-high-throughput-experiments/confounding-factors-in-mirna-experiments-supplementary-information/" class="nav-list-link">Supplementary information</a></li></ul></li><li class="nav-list-item"><a href="#" class="nav-list-expander"><svg viewBox="0 0 24 24"><use xlink:href="#svg-arrow-right"></use></svg></a><a href="https://ncrnas.github.io/mitkb/mirna-and-other-ncrnas/" class="nav-list-link">Other ncRNAs</a><ul class="nav-list "></ul></li><li class="nav-list-item"><a href="https://ncrnas.github.io/mitkb/support-articles/" class="nav-list-link">Support articles</a></li><li class="nav-list-item"><a href="https://ncrnas.github.io/mitkb/contact" class="nav-list-link">Contact us</a></li></ul>

      
    </nav>
    <footer class="site-footer">
      <a href="https://ncrnas.github.io/mitkb/terms">Terms and Privacy Policy</a> <br /><br />
      This site uses <a href="https://github.com/pmarsceill/just-the-docs">Just the Docs</a>, a documentation theme for Jekyll.
    </footer>
  </div>
  <div class="main" id="top">
    <div id="main-header" class="main-header">
      
        <div class="search">
          <div class="search-input-wrap">
            <input type="text" id="search-input" class="search-input" tabindex="0" placeholder="Search mitkb" aria-label="Search mitkb" autocomplete="off">
            <label for="search-input" class="search-label"><svg viewBox="0 0 24 24" class="search-icon"><use xlink:href="#svg-search"></use></svg></label>
          </div>
          <div id="search-results" class="search-results"></div>
        </div>
      
      
      
        <nav aria-label="Auxiliary" class="aux-nav">
          <ul class="aux-nav-list">
            
              <li class="aux-nav-list-item">
                <a href="https://github.com/ncrnas/mitkb" class="site-button"
                  
                >
                  mitkb on github
                </a>
              </li>
            
          </ul>
        </nav>
      
    </div>
    <div id="main-content-wrap" class="main-content-wrap">
      
        <nav aria-label="Breadcrumb" class="breadcrumb-nav">
            <ol class="breadcrumb-nav-list">
              
                <li class="breadcrumb-nav-list-item"><a href="https://ncrnas.github.io/mitkb/regulatory-mechanism-and-interactions-of-micrornas/">Overview</a></li>
              
              <li class="breadcrumb-nav-list-item"><span>1 Introduction</span></li>
            </ol>
          </nav>
        
      
      <div id="main-content" class="main-content" role="main">
        
          <h1 class="no_toc" id="1-introduction">
        
        
          <a href="#1-introduction" class="anchor-heading" aria-labelledby="1-introduction"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 1 Introduction
        
        
      </h1>
    <hr />

<details open="">
  <summary class="text-delta">
    Table of contents
  </summary>
<ol id="markdown-toc">
  <li><a href="#S:1.1" id="markdown-toc-S:1.1">1.1 Challenges</a></li>
  <li><a href="#S:1.2" id="markdown-toc-S:1.2">1.2 Goals</a></li>
  <li><a href="#S:1.3" id="markdown-toc-S:1.3">1.3 Thesis structure</a></li>
  <li><a href="#S:1.4" id="markdown-toc-S:1.4">References</a></li>
</ol>

</details><hr />
      <h2 id="S:1.1">
        
        
          <a href="#S:1.1" class="anchor-heading" aria-labelledby="S:1.1"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 1.1 Challenges
        
        
      </h2>
    

<p>In the 1980s, most non-protein-coding regions in the genome were thought
to be ‘junk’ DNA with no functional purpose <a class="citation" href="#Orgel1980">[1]</a>. Nonetheless,
during the last two decades, new classes of non-coding RNAs (ncRNAs)
that have gene regulatory roles have been discovered within these ‘junk’
regions <a class="citation" href="#Wright2011">[2]</a>. MicroRNAs (miRNAs) are one such new class of
ncRNAs that have many important regulatory roles on a genome-wide scale
<a class="citation" href="#Bartel2004">[3]</a>. Because of their importance, research on miRNAs has
gained popularity in recent years (Fig. <a href="#F:1.1">1.1</a>).
Although many research efforts have revealed basic miRNA characteristic
and regulation <a class="citation" href="#Bartel2004">[3,4]</a>, there are still many
challenges to identify the comprehensive characteristics and precise
regulatory mechanism of miRNAs. This thesis covers three such challenges
related to miRNA studies.</p>
      <h3 id="F:1.1" class="no_toc">
        
        
          <a href="#F:1.1" class="anchor-heading" aria-labelledby="F:1.1"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a>  
        
        
      </h3>
    
<p><img src="/mitkb/assets/images/thesis/pubmed-query-mirna-papers.webp" alt="Figure 1.1" width="546" /></p>
      <h4 class="no_toc text-caption" id="figure-11-pubmed-query">
        
        
          <a href="#figure-11-pubmed-query" class="anchor-heading" aria-labelledby="figure-11-pubmed-query"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> <strong>Figure 1.1</strong>. PubMed query.
        
        
      </h4>
    
<p class="fs-2">Two figures show the trend of miRNA related papers
as <strong>(A)</strong> the number of papers, and <strong>(B)</strong> the ratio to all papers in
PubMed.</p><hr />

<p>The first challenge is to identify accurate miRNA targets in animals. It
is important to understand miRNA contributions to the genome-wide gene
regulation, but there are several known obstacles related to
indentifying miRNA targeRts in animals. Firstly, miRNAs are quite
abundant <a class="citation" href="#Bartel2004">[3]</a>, and one miRNA can potentially regulate many
protein-coding genes <a class="citation" href="#Friedman2009">[5]</a>. In some cases, miRNAs bind their
target mRNAs by base-pairing with only six nucleotides <a class="citation" href="#Friedman2009">[5]</a>,
which results in thousands of potential candidate genes influenced by
one miRNA at a genome-wide level. Secondly, since miRNAs are expressed
in a cell- or tissue-specific manner <a class="citation" href="#Wang2009b">[6]</a>, one true positive
miRNA target can be a false positive in a different cell or tissue type.
Thirdly, the precise mechanism of miRNA binding process on its target
mRNA is unknown <a class="citation" href="#Bartel2009">[4]</a>. Therefore, combinations of miRNA features
are usually used to predict miRNA targets, but the combined effects of
these features on miRNA targeting are unclear.</p>

<p>The second challenge is to interpret miRNA high-throughput data
appropriately with high accuracy. Microarray, next generation
sequencing, and quantitative proteomics are three major high-throughput
technologies widely used for miRNA studies. Nonetheless, analyses of the
data from these high-throughput technologies often give different
interpretations regarding miRNA characteristics and regulation
<a class="citation" href="#Baek2008">[7,8,9]</a>. A major obstacle is that there are
many factors involved in these analyses, but the main factors that cause
these differences are unknown.</p>

<p>The third challenge is to indentify potential miRNA interactions with
other ncRNAs. Although most miRNAs regulate genes at the
post-transcriptional level, some miRNAs can also regulate transcription
itself <a class="citation" href="#Place2008">[10,11,12]</a>. This transcriptional
regulation seems to involve ncRNAs overlapping or interacting with the
target gene promoters
<a class="citation" href="#Han2007">[13,14,15,16,17]</a>. Many
aspects of this miRNA regulation at the transcription level are poorly
understood. Moreover, few experimental data are available for this miRNA
regulation at the transcription level.</p>
      <h2 id="S:1.2">
        
        
          <a href="#S:1.2" class="anchor-heading" aria-labelledby="S:1.2"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 1.2 Goals
        
        
      </h2>
    

<p>The main goal of this thesis is to reveal the characteristics and
regulations of miRNAs by analyzing several different types of
high-throughput data through bioinformatics approaches. To achieve this
goal, I defined three sub-goals to solve the three challenges of miRNA
studies described in the previous section.</p>

<p>The first sub-goal is to develop a miRNA target prediction algorithm
with high accuracy. Most existing prediction algorithms focus on
identifying individual target sites without considering multiple target
sites. They do not include multiple target sites that possibly
contribute to miRNA regulation. Moreover, most algorithms use strict
filtering, such as filtering with evolutional conservation. Filtering
can reduce false positive miRNA targets, but it potentially removes many
true positive targets at the same time. Therefore, the aim of this
sub-goal is to develop a model that can predict unbiased miRNA targets
by considering multiple targets without filtering.</p>

<p>The second sub-goal is to analyze several different types of miRNA
high-throughput technologies. The aim of this sub-goal is to reveal the
characteristics of each technology and identify strong factors that
cause inconsistent results between different types of experiments by
statistical approaches.</p>

<p>The third sub-goal is to infer potential miRNA regulations outside of
3’ untranslated regions (UTRs) in general and interactions
between miRNAs and ncRNAs in complex loci in particular. A complex locus
is a region of DNA that contains multiple genes that have interactions
between them or share common regulatory mechanisms <a class="citation" href="#Engstrom2006">[18]</a>. Our
hypothesis is that some miRNAs interact with ncRNA:mRNA pairs in complex
loci. The aim of this sub-goal is to investigate this hypothesis of
miRNA involvement in complex loci together with miRNA regulations
outside of 3’ UTRs by computationally analyzing the data from
high-throughput experiments.</p>

<p>In this thesis, these sub-goals are referred to in italic to clarify the
relationship between parts of the text and their corresponding sub-goals
if necessary.</p>

<ul>
  <li>
    <p>First sub-goal: <em><a href="/mitkb/mirna-target-prediction/">miRNA target prediction</a></em></p>
  </li>
  <li>
    <p>Second sub-goal: <em><a href="/mitkb/mirna-high-throughput-experiments/">miRNA high-throughput experiments</a></em></p>
  </li>
  <li>
    <p>Third sub-goal: <em><a href="/mitkb/mirna-and-other-ncrnas/">miRNA and other ncRNAs</a></em></p>
  </li>
</ul>
      <h2 id="S:1.3">
        
        
          <a href="#S:1.3" class="anchor-heading" aria-labelledby="S:1.3"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> 1.3 Thesis structure
        
        
      </h2>
    

<p>This thesis consists of eight chapters followed by five papers.</p>

<p><a href="/mitkb/regulatory-mechanism-and-interactions-of-micrornas/2-papers-and-their-corresponding-sub-goals/">Chapter Two: Papers and their corresponding sub-goals.</a>
This chapter summarizes the five papers included in this thesis. It also relates them
to each sub-goal.</p>

<p><a href="/mitkb/regulatory-mechanism-and-interactions-of-micrornas/3-micrornas-and-other-non-coding-rnas/">Chapter Three: MicroRNAs and other non-coding RNAs.</a>
This chapter introduces the history, characteristics, and biological functions of
miRNAs as well as some additional information about other ncRNAs.</p>

<p><a href="/mitkb/regulatory-mechanism-and-interactions-of-micrornas/4-high-throughput-biological-experiments/">Chapter Four: High-throughput biological experiments.</a>
This chapter focuses on three high-throughput technologies used in our research:
microarray, next generation sequencing, and quantitative proteomics.</p>

<p><a href="/mitkb/regulatory-mechanism-and-interactions-of-micrornas/5-statistical-tests-and-methods/">Chapter Five: Statistical tests and methods.</a>
This chapter starts with explaining basic statistical tests followed by applied statistical
approaches used throughout in our research, such as non-parametric
tests, resampling, and multiple comparison tests.</p>

<p><a href="/mitkb/regulatory-mechanism-and-interactions-of-micrornas/6-machine-learning-theory-and-support-vector-machine/">Chapter Six: Machine learning theory and Support vector machine.</a>
Support vector machine (SVM) is the main method used in the first
sub-goal: <em>miRNA target prediction</em>. This chapter explains the
theoretical background of SVM, data preparation and evaluation methods
for SVM, as well as some other machine learning methods for comparison.</p>

<p><a href="/mitkb/regulatory-mechanism-and-interactions-of-micrornas/7-computational-implementation/">Chapter Seven: Computational implementation.</a>
Any state-of-the-art model or algorithm is ineffective without appropriate computational
implementation. This chapter focuses on the computation implementations
used in our research.</p>

<p><a href="/mitkb/regulatory-mechanism-and-interactions-of-micrornas/8-future-perspectives/">Chapter Eight: Future perspective.</a>
This chapter describes potential improvements of each sub-goal as future perspectives.</p>
      <h2 id="S:1.4">
        
        
          <a href="#S:1.4" class="anchor-heading" aria-labelledby="S:1.4"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> References
        
        
      </h2>
    
<ol class="bibliography"><li><span id="Orgel1980">Orgel LE, Crick FHC. Selfish DNA: the ultimate parasite. Nature 1980;284:604–7. <a href="https://doi.org/10.1038/284604a0">https://doi.org/10.1038/284604a0.</a></span>
</li>
<li><span id="Wright2011">Wright MW, Bruford EA. Naming ’junk’: Human non-protein coding RNA (ncRNA) gene nomenclature. Human Genomics 2011;5:90. <a href="https://doi.org/10.1186/1479-7364-5-2-90">https://doi.org/10.1186/1479-7364-5-2-90.</a></span>
</li>
<li><span id="Bartel2004">Bartel DP. MicroRNAs: genomics, biogenesis, mechanism, and function. Cell 2004;116:281–97. <a href="https://doi.org/10.1016/s0092-8674(04)00045-5">https://doi.org/10.1016/s0092-8674(04)00045-5.</a></span>
</li>
<li><span id="Bartel2009">Bartel DP. MicroRNAs: target recognition and regulatory functions. Cell 2009;136:215–33. <a href="https://doi.org/10.1016/j.cell.2009.01.002">https://doi.org/10.1016/j.cell.2009.01.002.</a></span>
</li>
<li><span id="Friedman2009">Friedman RC, Farh KK-H, Burge CB, Bartel DP. Most mammalian mRNAs are conserved targets of microRNAs. Genome Research 2008;19:92–105. <a href="https://doi.org/10.1101/gr.082701.108">https://doi.org/10.1101/gr.082701.108.</a></span>
</li>
<li><span id="Wang2009b">Wang L, Oberg AL, Asmann YW, Sicotte H, McDonnell SK, Riska SM, et al. Genome-wide transcriptional profiling reveals MicroRNA-correlated genes and biological processes in human lymphoblastoid cell lines. PLoS ONE 2009;4:e5878. <a href="https://doi.org/10.1371/journal.pone.0005878">https://doi.org/10.1371/journal.pone.0005878.</a></span>
</li>
<li><span id="Baek2008">Baek D, Villén J, Shin C, Camargo FD, Gygi SP, Bartel DP. The impact of microRNAs on protein output. Nature 2008;455:64–71. <a href="https://doi.org/10.1038/nature07242">https://doi.org/10.1038/nature07242.</a></span>
</li>
<li><span id="Selbach2008">Selbach M, Schwanhäusser B, Thierfelder N, Fang Z, Khanin R, Rajewsky N. Widespread changes in protein synthesis induced by microRNAs. Nature 2008;455:58–63. <a href="https://doi.org/10.1038/nature07228">https://doi.org/10.1038/nature07228.</a></span>
</li>
<li><span id="Wen2011">Wen J, Parker BJ, Jacobsen A, Krogh A. MicroRNA transfection and AGO-bound CLIP-seq data sets reveal distinct determinants of miRNA action. RNA 2011;17:820–34. <a href="https://doi.org/10.1261/rna.2387911">https://doi.org/10.1261/rna.2387911.</a></span>
</li>
<li><span id="Place2008">Place RF, Li L-C, Pookot D, Noonan EJ, Dahiya R. MicroRNA-373 induces expression of genes with complementary promoter sequences. Proceedings of the National Academy of Sciences 2008;105:1608–13. <a href="https://doi.org/10.1073/pnas.0707594105">https://doi.org/10.1073/pnas.0707594105.</a></span>
</li>
<li><span id="Kim2008">Kim DH, Saetrom P, Snove O, Rossi JJ. MicroRNA-directed transcriptional gene silencing in mammalian cells. Proceedings of the National Academy of Sciences 2008;105:16230–5. <a href="https://doi.org/10.1073/pnas.0808830105">https://doi.org/10.1073/pnas.0808830105.</a></span>
</li>
<li><span id="Younger2011">Younger ST, Corey DR. Transcriptional gene silencing in mammalian cells by miRNA mimics that target gene promoters. Nucleic Acids Research 2011;39:5682–91. <a href="https://doi.org/10.1093/nar/gkr155">https://doi.org/10.1093/nar/gkr155.</a></span>
</li>
<li><span id="Han2007">Han J, Kim D, Morris KV. Promoter-associated RNA is required for RNA-directed transcriptional gene silencing in human cells. Proceedings of the National Academy of Sciences 2007;104:12422–7. <a href="https://doi.org/10.1073/pnas.0701635104">https://doi.org/10.1073/pnas.0701635104.</a></span>
</li>
<li><span id="Morris2011">Morris KV. The emerging role of RNA in the regulation of gene transcription in human cells. Seminars in Cell &amp; Developmental Biology 2011;22:351–8. <a href="https://doi.org/10.1016/j.semcdb.2011.02.017">https://doi.org/10.1016/j.semcdb.2011.02.017.</a></span>
</li>
<li><span id="Morris2008">Morris KV, Santoso S, Turner A-M, Pastori C, Hawkins PG. Bidirectional transcription directs both transcriptional gene activation and suppression in human cells. PLoS Genetics 2008;4:e1000258. <a href="https://doi.org/10.1371/journal.pgen.1000258">https://doi.org/10.1371/journal.pgen.1000258.</a></span>
</li>
<li><span id="Schwartz2008">Schwartz JC, Younger ST, Nguyen N-B, Hardy DB, Monia BP, Corey DR, et al. Antisense transcripts are targets for activating small RNAs. Nature Structural &amp; Molecular Biology 2008;15:842–8. <a href="https://doi.org/10.1038/nsmb.1444">https://doi.org/10.1038/nsmb.1444.</a></span>
</li>
<li><span id="Yue2010">Yue X, Schwartz JC, Chu Y, Younger ST, Gagnon KT, Elbashir S, et al. Transcriptional regulation by small RNAs at sequences downstream from 3′ gene termini. Nature Chemical Biology 2010;6:621–9. <a href="https://doi.org/10.1038/nchembio.400">https://doi.org/10.1038/nchembio.400.</a></span>
</li>
<li><span id="Engstrom2006">Engström PG, Suzuki H, Ninomiya N, Akalin A, Sessa L, Lavorgna G, et al. Complex loci in human and mouse genomes. PLoS Genetics 2006;2:e47. <a href="https://doi.org/10.1371/journal.pgen.0020047">https://doi.org/10.1371/journal.pgen.0020047.</a></span>
</li></ol>

        

        

        
          <hr>
          <h2>Leave a comment</h2>
          <div id="disqus_thread"></div>
        

        
        
          <hr>
          <footer>
            
              <p><a href="#top" id="back-to-top">Back to top</a></p>
            

            <p class="text-small text-grey-dk-100 mb-0">Copyright &copy; 2021 Takaya Saito.</p>

            
              <div class="d-flex mt-2">
                
                
              </div>
            
          </footer>
        

      </div>
    </div>

    
      

      <div class="search-overlay"></div>
    
  </div>

  
     <script>addBackToTop({
       id: 'back-to-top-js-button'
     })</script>
  

  
    <script>
        /**
        *  RECOMMENDED CONFIGURATION VARIABLES: EDIT AND UNCOMMENT THE SECTION BELOW TO INSERT DYNAMIC VALUES FROM YOUR PLATFORM OR CMS.
        *  LEARN WHY DEFINING THESE VARIABLES IS IMPORTANT: https://disqus.com/admin/universalcode/#configuration-variables    */
        /*
        var disqus_config = function () {
        this.page.url = PAGE_URL;  // Replace PAGE_URL with your page's canonical URL variable
        this.page.identifier = PAGE_IDENTIFIER; // Replace PAGE_IDENTIFIER with your page's unique identifier variable
        };
        */
        (function() { // DON'T EDIT BELOW THIS LINE
        var d = document, s = d.createElement('script');
        s.src = 'https://mitkb.disqus.com/embed.js';
        s.setAttribute('data-timestamp', +new Date());
        (d.head || d.body).appendChild(s);
        })();
    </script>
    <noscript>Please enable JavaScript to view the <a href="https://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
  

</body>
</html>

